{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "import urllib2\n",
    "import json\n",
    "import requests"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "# BigGIM example\n",
    "\n",
    "Big GIM (Gene Interaction Miner) is a Translator Knowledge Source that contains function interaction data for all pairs of genes. Functional interaction data are available from four different sources: \n",
    "\n",
    "1. tissue-specific gene expression correlations from healthy tissue samples (GTEx), \n",
    "2. tissue-specific gene expression correlations from cancer samples (TCGA), \n",
    "3. tissue-specific probabilities of function interaction (GIANT), and \n",
    "4. direct interactions (BioGRID). \n",
    "   \n",
    "The data is stored as a Google BigQuery table enabling fast access.\n",
    "\n",
    "## Swagger api specification\n",
    "\n",
    "http://biggim.ncats.io/api/\n",
    "\n",
    "## Sections\n",
    "\n",
    "* 1.0 [Metadata Examples](#metadata_examples)\n",
    "    * 1.1 [Get available studies](#metadata_studies)\n",
    "    * 1.2 [Study descriptions](#metadata_study_descriptions)\n",
    "    * 1.3 [Get a single study](#metadata_single_study)\n",
    "    * 1.4 [Get substudies](#metadata_substudies)\n",
    "    * 1.5 [Get all tables](#metadata_tables)\n",
    "    * 1.6 [Get one table](#metadata_table)\n",
    "    * 1.7 [Get a single column](#metadata_column)\n",
    "    * 1.8 [List all tissues](#metadata_tissues)\n",
    "    * 1.9 [Get single tissue](#metadata_tissue)   \n",
    "* 2.0 [Query Examples](#query_examples)\n",
    "    * 2.1 [Simple predefined query](#query_simple)\n",
    "    * 2.2 [Check the status of the simple predefined query](#query_status)\n",
    "    * 2.3 [Get the results as dataframe](#query_results)\n",
    "* 3.0 [Full Example](#full_example)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "base_url = 'http://biggim.ncats.io/api'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "#a couple of simple helper functions\n",
    "def post(endpoint, data={}, base_url=base_url):\n",
    "    req = requests.post('%s/%s' % (base_url,endpoint), data=data)\n",
    "    req.raise_for_status()\n",
    "    return req.json()\n",
    "\n",
    "def get(endpoint, data={}, base_url=base_url):\n",
    "    req = requests.get('%s/%s' % (base_url,endpoint), data=data)\n",
    "    req.raise_for_status()\n",
    "    return req.json()\n",
    "    \n",
    "\n",
    "def jprint(dct):\n",
    "    print json.dumps(dct, indent=2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_examples\"></a>\n",
    "## Example Metadata"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_studies\"></a>\n",
    "### Get available studies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[\n",
      "  {\n",
      "    \"description\": \" Biological General Repository for Interaction Datasets\\n        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\\n\\n        https://thebiogrid.org/\\n        \", \n",
      "    \"name\": \"BioGRID\", \n",
      "    \"substudies\": [\n",
      "      {\n",
      "        \"tissue_hierarchy\": null, \n",
      "        \"description\": \"Default\", \n",
      "        \"name\": \"Default\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"BOOLEAN\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Interaction\", \n",
      "            \"interactions_type\": \"Boolean Flag For Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System\", \n",
      "            \"interactions_type\": \"Experimental System Used\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Experimental_System_Type\", \n",
      "            \"interactions_type\": \"Experimental System Type\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"STRING\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"BioGRID_Throughput\", \n",
      "            \"interactions_type\": \"High or Low Throughput Measurement\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": null\n",
      "      }\n",
      "    ]\n",
      "  }, \n",
      "  {\n",
      "    \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \", \n",
      "    \"name\": \"TCGA\", \n",
      "    \"substudies\": [\n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian serous carcinoma cell|1 - ovary cancer cell|2 - ovary|3 - female reproductive gland|3 - internal female genital organ|4 - female reproductive system|4 - endocrine gland|5 - reproductive system|5 - gland|6 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Ovarian serous cystadenocarcinoma\", \n",
      "        \"name\": \"OV\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_OV_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"ovarian serous carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adrenocortical carcinoma cell|1 - adrenal cortex|2 - adrenal gland|3 - viscus|3 - endocrine gland|4 - whole body|4 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Adrenocortical carcinoma\", \n",
      "        \"name\": \"ACC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ACC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"adrenocortical carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - glioblastoma cell|1 - astrocytoma cell|2 - glioma cell|2 - astrocyte|3 - glia|3 - glial cell|3 - astroglial cell|3 - macroglia|3 - astroblast|4 - spinal cord|4 - brain|4 - astroglia|4 - neuroblast|5 - central nervous system|5 - head|5 - embryo|6 - nervous system|6 - whole body|6 - embryonic structure|7 - animal|7 - organism form|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Glioblastoma multiforme\", \n",
      "        \"name\": \"GBM\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_GBM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"glioblastoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - glioma cell|1 - glia|1 - glial cell|2 - spinal cord|2 - brain|3 - central nervous system|3 - head|4 - nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Brain Lower Grade Glioma\", \n",
      "        \"name\": \"LGG\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LGG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"glioma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uterine sarcoma cell|1 - sarcoma cell|1 - uterine cancer cell|2 - cartilage|2 - skeletal muscle cancer cell|2 - bone cancer cell|2 - uterus|3 - connective tissue|3 - skeletal system|3 - skeletal muscle cell|3 - bone|3 - internal female genital organ|4 - whole body|4 - skeletal muscle|4 - female reproductive system|5 - animal|5 - muscle|5 - vertebrate muscular system|5 - reproductive system|6 - tissues, cell types and enzyme sources|6 - muscular system|6 - urogenital system\", \n",
      "        \"description\": \"Uterine Carcinosarcoma\", \n",
      "        \"name\": \"UCS\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCS_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uterine sarcoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uveal melanoma cell|1 - melanoma cell|2 - skin cancer cell|2 - melanocyte|3 - skin|3 - epidermal cell|4 - integument|4 - epidermis|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Uveal Melanoma\", \n",
      "        \"name\": \"UVM\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UVM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uveal melanoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder cancer cell|1 - urinary bladder|2 - bladder|2 - urinary system|2 - urinary tract|3 - viscus|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Bladder Urothelial Carcinoma\", \n",
      "        \"name\": \"BLCA\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"urinary bladder cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - breast cancer cell|1 - breast|1 - adenocarcinoma cell|2 - thorax|2 - female reproductive gland|2 - excretory gland|2 - carcinoma cell|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|3 - epithelioma cell|4 - whole body|4 - reproductive system|4 - epithelial cell|5 - animal|5 - urogenital system|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"description\": \"Breast invasive carcinoma\", \n",
      "        \"name\": \"BRCA\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"breast cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cervical adenocarcinoma cell|0 - cervical squamous cell carcinoma cell|1 - cervical carcinoma cell|1 - adenocarcinoma cell|1 - squamous cell carcinoma cell|2 - uterine cervix|2 - carcinoma cell|2 - squamous epithelium|3 - uterus|3 - epithelioma cell|3 - epithelium|4 - internal female genital organ|4 - epithelial cell|4 - integument|5 - female reproductive system|5 - whole body|6 - reproductive system|6 - animal|7 - urogenital system|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Cervical squamous cell carcinoma and endocervical adenocarcinoma\", \n",
      "        \"name\": \"CESC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CESC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cervical adenocarcinoma cell,cervical squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cholangiocarcinoma cell|1 - bile duct epithelium|1 - bile duct|2 - biliary epithelium|2 - liver|3 - epithelium|3 - digestive gland|3 - viscus|4 - integument|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Cholangiocarcinoma\", \n",
      "        \"name\": \"CHOL\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cholangiocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - colorectal adenocarcinoma cell|1 - rectal cancer cell|1 - colonic cancer cell|1 - adenocarcinoma cell|2 - rectum|2 - gastrointestinal cancer cell|2 - colon|2 - carcinoma cell|3 - large intestine|3 - colorectum|3 - gastrointestinal tract|3 - epithelioma cell|4 - intestine|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Colon adenocarcinoma\", \n",
      "        \"name\": \"COAD\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_COAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"colorectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - diffuse large B-cell lymphoma cell|1 - B-lymphoma cell line|2 - lymphoma cell line|3 - lymphoid cell line|3 - lymphoma cell|4 - lymphoid cell|4 - blood cancer cell|4 - lymphoid tissue|5 - hematopoietic cell|5 - immune system|5 - hematopoietic system|6 - blood|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\", \n",
      "        \"name\": \"DLBC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"diffuse large B-cell lymphoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - esophageal cancer cell|1 - esophagus|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Esophageal carcinoma\", \n",
      "        \"name\": \"ESCA\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"esophageal cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - head and neck cancer cell|1 - head|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Head and Neck squamous cell carcinoma\", \n",
      "        \"name\": \"HNSC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"head and neck cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - chromophobe renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"description\": \"Kidney Chromophobe\", \n",
      "        \"name\": \"KICH\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KICH_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"chromophobe renal cell carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - renal clear cell|0 - renal cell carcinoma cell|1 - renal tubule|1 - kidney cancer cell|1 - adenocarcinoma cell|2 - nephron|2 - kidney|2 - carcinoma cell|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|3 - epithelioma cell|4 - gland|4 - whole body|4 - urogenital system|4 - epithelial cell|5 - animal|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"description\": \"Kidney renal clear cell carcinoma\", \n",
      "        \"name\": \"KIRC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"renal clear cell,renal cell carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - papillary renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"description\": \"Kidney renal papillary cell carcinoma\", \n",
      "        \"name\": \"KIRP\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"papillary renal cell carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - acute myeloid leukemia cell|1 - myeloid leukemia cell|2 - leukemia cell|3 - leukocyte|3 - blood cancer cell|4 - hematopoietic cell|5 - blood|6 - hematopoietic system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Acute Myeloid Leukemia\", \n",
      "        \"name\": \"LAML\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LAML_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"acute myeloid leukemia cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - liver cancer cell|0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Liver hepatocellular carcinoma\", \n",
      "        \"name\": \"LIHC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"liver cancer cell,hepatocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lung adenocarcinoma cell|1 - carcinoma cell|1 - non-small cell lung cancer cell|2 - epithelioma cell|2 - bronchogenic carcinoma cell|3 - epithelial cell|3 - lung cancer cell|4 - epithelium|4 - lung|5 - integument|5 - respiratory system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Lung adenocarcinoma\", \n",
      "        \"name\": \"LUAD\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lung adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lung squamous cell carcinoma cell|1 - lung cancer cell|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Lung squamous cell carcinoma\", \n",
      "        \"name\": \"LUSC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lung squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - mesothelioma cell|1 - mesothelium|2 - mesoderm|2 - squamous epithelium|2 - coelom|3 - germ layer|3 - epithelium|4 - embryo|4 - integument|5 - embryonic structure|5 - whole body|6 - organism form|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Mesothelioma\", \n",
      "        \"name\": \"MESO\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_MESO_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"mesothelioma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic ductal adenocarcinoma cell|1 - pancreatic ductal carcinoma cell|1 - adenocarcinoma cell|2 - pancreatic duct|2 - carcinoma cell|3 - exocrine pancreas|3 - epithelioma cell|4 - pancreas|4 - epithelial cell|5 - digestive gland|5 - viscus|5 - epithelium|6 - endocrine gland|6 - whole body|6 - integument|7 - gland|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Pancreatic adenocarcinoma\", \n",
      "        \"name\": \"PAAD\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"pancreatic ductal adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pheochromocytoma cell|0 - neural crest cell|0 - neuroendocrine tumor cell|1 - adrenal gland cancer cell|1 - neural crest|1 - neuroendocrine cell|1 - carcinoid cell|2 - adrenal gland|2 - neural plate|2 - endocrine gland|2 - gastrointestinal cancer cell|2 - intestinal mucosa|3 - viscus|3 - ectoderm|3 - gland|3 - gastrointestinal tract|3 - mucosa|3 - intestine|4 - whole body|4 - germ layer|4 - alimentary canal|4 - integument|5 - animal|5 - embryo|6 - tissues, cell types and enzyme sources|6 - embryonic structure|7 - organism form\", \n",
      "        \"description\": \"Pheochromocytoma and Paraganglioma\", \n",
      "        \"name\": \"PCPG\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"pheochromocytoma cell,neural crest cell,neuroendocrine tumor cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - prostate adenocarcinoma cell|1 - adenocarcinoma cell|2 - carcinoma cell|3 - epithelioma cell|4 - epithelial cell|5 - epithelium|6 - integument|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Prostate adenocarcinoma\", \n",
      "        \"name\": \"PRAD\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"prostate adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - rectal adenocarcinoma cell|1 - colorectal adenocarcinoma cell|2 - rectal cancer cell|2 - colonic cancer cell|2 - adenocarcinoma cell|3 - rectum|3 - gastrointestinal cancer cell|3 - colon|3 - carcinoma cell|4 - large intestine|4 - colorectum|4 - gastrointestinal tract|4 - epithelioma cell|5 - intestine|5 - alimentary canal|5 - epithelial cell|6 - viscus|6 - epithelium|7 - whole body|7 - integument|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Rectum adenocarcinoma\", \n",
      "        \"name\": \"READ\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_READ_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"rectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - sarcoma cell|1 - cartilage|1 - skeletal muscle cancer cell|1 - bone cancer cell|2 - connective tissue|2 - skeletal system|2 - skeletal muscle cell|2 - bone|3 - whole body|3 - skeletal muscle|4 - animal|4 - muscle|4 - vertebrate muscular system|5 - tissues, cell types and enzyme sources|5 - muscular system\", \n",
      "        \"description\": \"Sarcoma\", \n",
      "        \"name\": \"SARC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SARC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"sarcoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - skin cancer cell|0 - melanoma cell|1 - skin|1 - melanocyte|2 - integument|2 - epidermal cell|3 - whole body|3 - epidermis|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Skin Cutaneous Melanoma\", \n",
      "        \"name\": \"SKCM\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"skin cancer cell,melanoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - gastric adenocarcinoma cell|1 - gastric cancer cell|1 - adenocarcinoma cell|2 - stomach|2 - carcinoma cell|3 - gastrointestinal tract|3 - epithelioma cell|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Stomach adenocarcinoma\", \n",
      "        \"name\": \"STAD\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_STAD_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"gastric adenocarcinoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - testicular cancer cell|1 - testis|2 - internal male genital organ|2 - gonad|2 - male reproductive gland|3 - male reproductive system|3 - female reproductive gland|3 - endocrine gland|4 - reproductive system|4 - female reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Testicular Germ Cell Tumors\", \n",
      "        \"name\": \"TGCT\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"testicular cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid cancer cell|1 - thyroid gland|2 - throat|2 - endocrine gland|3 - neck|3 - gland|4 - trunk|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Thyroid carcinoma\", \n",
      "        \"name\": \"THCA\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THCA_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thyroid cancer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thymoma cell|1 - thymic cancer cell|2 - thymus|3 - immune system|3 - thorax|3 - endocrine gland|3 - lymphoid tissue|4 - whole body|4 - trunk|4 - gland|4 - hematopoietic system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Thymoma\", \n",
      "        \"name\": \"THYM\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_THYM_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thymoma cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - endometrioid carcinoma cell|1 - uterine endometrium|1 - ovary cancer cell|2 - mucosa|2 - uterus|2 - ovary|3 - integument|3 - internal female genital organ|3 - female reproductive gland|4 - whole body|4 - female reproductive system|4 - endocrine gland|5 - animal|5 - reproductive system|5 - gland|6 - tissues, cell types and enzyme sources|6 - urogenital system\", \n",
      "        \"description\": \"Uterine Corpus Endometrial Carcinoma\", \n",
      "        \"name\": \"UCEC\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"endometrioid carcinoma cell\"\n",
      "      }\n",
      "    ]\n",
      "  }, \n",
      "  {\n",
      "    \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \", \n",
      "    \"name\": \"GIANT\", \n",
      "    \"substudies\": [\n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - ear|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"ear\", \n",
      "        \"name\": \"ear\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"ear\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - eye|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"eye\", \n",
      "        \"name\": \"eye\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"eye\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - bone|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"bone\", \n",
      "        \"name\": \"bone\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"bone\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - glia|1 - spinal cord|1 - brain|2 - central nervous system|2 - head|3 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"glia\", \n",
      "        \"name\": \"glia\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"glia\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lens|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"lens\", \n",
      "        \"name\": \"lens\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lens\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"lung\", \n",
      "        \"name\": \"lung\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lung\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pons|1 - brain stem|1 - metencephalon|2 - brain|2 - hindbrain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"pons\", \n",
      "        \"name\": \"pons\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"pons\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"skin\", \n",
      "        \"name\": \"skin\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"skin\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - aorta|1 - artery|2 - blood vessel|3 - vascular system|4 - cardiovascular system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"aorta\", \n",
      "        \"name\": \"aorta\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"aorta\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"blood\", \n",
      "        \"name\": \"blood\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"blood\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"brain\", \n",
      "        \"name\": \"brain\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"brain\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cecum|1 - large intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cecum\", \n",
      "        \"name\": \"cecum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cecum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"colon\", \n",
      "        \"name\": \"colon\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"colon\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - fetus|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"fetus\", \n",
      "        \"name\": \"fetus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"fetus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"heart\", \n",
      "        \"name\": \"heart\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"heart\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - ileum|1 - small intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"ileum\", \n",
      "        \"name\": \"ileum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"ileum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"liver\", \n",
      "        \"name\": \"liver\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"liver\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - ovary|1 - female reproductive gland|1 - internal female genital organ|2 - female reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"ovary\", \n",
      "        \"name\": \"ovary\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"ovary\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - serum|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"serum\", \n",
      "        \"name\": \"serum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"serum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - tooth|1 - mouth|2 - head|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"tooth\", \n",
      "        \"name\": \"tooth\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"tooth\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - artery|1 - blood vessel|2 - vascular system|3 - cardiovascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"artery\", \n",
      "        \"name\": \"artery\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"artery\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cornea|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cornea\", \n",
      "        \"name\": \"cornea\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cornea\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - embryo|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"embryo\", \n",
      "        \"name\": \"embryo\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"embryo\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - kidney|1 - excretory gland|1 - viscus|1 - urinary system|1 - urinary tract|2 - gland|2 - whole body|2 - urogenital system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"kidney\", \n",
      "        \"name\": \"kidney\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"kidney\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"muscle\", \n",
      "        \"name\": \"muscle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"muscle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - neuron|1 - nervous system|1 - nerve|1 - neuroblast|2 - whole body|2 - embryo|3 - animal|3 - embryonic structure|4 - tissues, cell types and enzyme sources|4 - organism form\", \n",
      "        \"description\": \"neuron\", \n",
      "        \"name\": \"neuron\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"neuron\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - retina|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"retina\", \n",
      "        \"name\": \"retina\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"retina\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"spleen\", \n",
      "        \"name\": \"spleen\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"spleen\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"testis\", \n",
      "        \"name\": \"testis\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"testis\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - tonsil|1 - lymphoid tissue|2 - immune system|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"tonsil\", \n",
      "        \"name\": \"tonsil\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"tonsil\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uterus|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"uterus\", \n",
      "        \"name\": \"uterus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uterus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - choroid|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"choroid\", \n",
      "        \"name\": \"choroid\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"choroid\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cochlea|1 - vestibular system|2 - vestibular labyrinth|3 - inner ear|4 - ear|5 - head|5 - sense organ|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cochlea\", \n",
      "        \"name\": \"cochlea\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cochlea\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - jejunum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"jejunum\", \n",
      "        \"name\": \"jejunum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"jejunum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - nephron|1 - kidney|2 - excretory gland|2 - viscus|2 - urinary system|2 - urinary tract|3 - gland|3 - whole body|3 - urogenital system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"nephron\", \n",
      "        \"name\": \"nephron\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"nephron\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - oviduct|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"oviduct\", \n",
      "        \"name\": \"oviduct\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"oviduct\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"stomach\", \n",
      "        \"name\": \"stomach\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"stomach\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - trachea|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"trachea\", \n",
      "        \"name\": \"trachea\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"trachea\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - amygdala|1 - temporal lobe|1 - avian pallium|1 - basal ganglion|1 - limbic system|2 - cerebral lobe|2 - telencephalon|2 - ganglion|2 - cerebral cortex|2 - brain|3 - forebrain|3 - nervous system|3 - cerebral hemisphere|3 - head|3 - central nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"amygdala\", \n",
      "        \"name\": \"amygdala\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"amygdala\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - basophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"basophil\", \n",
      "        \"name\": \"basophil\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"basophil\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - bronchus|1 - bronchoalveolar system|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"bronchus\", \n",
      "        \"name\": \"bronchus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"bronchus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - duodenum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"duodenum\", \n",
      "        \"name\": \"duodenum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"duodenum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - midbrain|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"midbrain\", \n",
      "        \"name\": \"midbrain\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"midbrain\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - monocyte|1 - leukocyte|1 - mononuclear phagocyte|2 - hematopoietic cell|2 - mononuclear cell|2 - marrow cell|2 - phagocyte|3 - blood|3 - myeloid progenitor cell|4 - hematopoietic system|4 - hematopoietic stem cell|5 - whole body|5 - blast cell|5 - adult stem cell|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"monocyte\", \n",
      "        \"name\": \"monocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"monocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"pancreas\", \n",
      "        \"name\": \"pancreas\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"pancreas\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - placenta|1 - fetus|1 - internal female genital organ|2 - embryonic structure|2 - female reproductive system|3 - whole body|3 - organism form|3 - reproductive system|4 - animal|4 - tissues, cell types and enzyme sources|4 - urogenital system\", \n",
      "        \"description\": \"placenta\", \n",
      "        \"name\": \"placenta\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"placenta\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - podocyte|1 - glomerular epithelium|1 - capsular epithelium|1 - epithelial cell|2 - renal glomerular capsule|2 - epithelium|3 - renal corpuscle|3 - integument|4 - nephron|4 - whole body|5 - kidney|5 - animal|6 - excretory gland|6 - viscus|6 - urinary system|6 - urinary tract|6 - tissues, cell types and enzyme sources|7 - gland|7 - urogenital system\", \n",
      "        \"description\": \"podocyte\", \n",
      "        \"name\": \"podocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"podocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thalamus|1 - diencephalon|2 - forebrain|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"thalamus\", \n",
      "        \"name\": \"thalamus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thalamus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - astrocyte|1 - astroglial cell|1 - macroglia|1 - astroblast|2 - astroglia|2 - glial cell|2 - glia|2 - neuroblast|3 - spinal cord|3 - brain|3 - embryo|4 - central nervous system|4 - head|4 - embryonic structure|5 - nervous system|5 - whole body|5 - organism form|6 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"astrocyte\", \n",
      "        \"name\": \"astrocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"astrocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cartilage|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cartilage\", \n",
      "        \"name\": \"cartilage\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cartilage\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - epidermis|1 - skin|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"epidermis\", \n",
      "        \"name\": \"epidermis\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"epidermis\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"esophagus\", \n",
      "        \"name\": \"esophagus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"esophagus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - forebrain|1 - brain|2 - head|2 - central nervous system|3 - whole body|3 - nervous system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"forebrain\", \n",
      "        \"name\": \"forebrain\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"forebrain\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - intestine|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"intestine\", \n",
      "        \"name\": \"intestine\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"intestine\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - leukocyte|1 - hematopoietic cell|2 - blood|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"leukocyte\", \n",
      "        \"name\": \"leukocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"leukocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - mast cell|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"mast_cell\", \n",
      "        \"name\": \"mast_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"mast cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spermatid|1 - secondary spermatocyte|2 - primary spermatocyte|3 - spermatocyte|4 - spermatogonium|4 - semen|4 - germ cell|5 - male reproductive system|5 - reproductive system|5 - gametocyte|6 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"spermatid\", \n",
      "        \"name\": \"spermatid\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"spermatid\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thymocyte|1 - thymus|1 - pre-T-lymphocyte|2 - immune system|2 - thorax|2 - endocrine gland|2 - lymphoid tissue|2 - T-lymphocyte|3 - whole body|3 - trunk|3 - gland|3 - hematopoietic system|3 - lymphocyte|4 - animal|4 - leukocyte|4 - lymphoid cell|5 - tissues, cell types and enzyme sources|5 - hematopoietic cell|6 - blood\", \n",
      "        \"description\": \"thymocyte\", \n",
      "        \"name\": \"thymocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thymocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellum|1 - metencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cerebellum\", \n",
      "        \"name\": \"cerebellum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cerebellum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - eosinophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"eosinophil\", \n",
      "        \"name\": \"eosinophil\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"eosinophil\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"hepatocyte\", \n",
      "        \"name\": \"hepatocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hepatocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"description\": \"hypophysis\", \n",
      "        \"name\": \"hypophysis\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lymph node|1 - immune system|1 - endocrine gland|1 - lymphoid tissue|2 - whole body|2 - gland|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"lymph_node\", \n",
      "        \"name\": \"lymph_node\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lymph node\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lymphocyte|1 - leukocyte|1 - lymphoid cell|2 - hematopoietic cell|2 - lymphoid tissue|3 - blood|3 - immune system|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"lymphocyte\", \n",
      "        \"name\": \"lymphocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"lymphocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - macrophage|1 - immune system|1 - monocyte|2 - whole body|2 - leukocyte|2 - mononuclear phagocyte|3 - animal|3 - hematopoietic cell|3 - mononuclear cell|3 - marrow cell|3 - phagocyte|4 - tissues, cell types and enzyme sources|4 - blood|4 - myeloid progenitor cell|5 - hematopoietic system|5 - hematopoietic stem cell|6 - blast cell|6 - adult stem cell\", \n",
      "        \"description\": \"macrophage\", \n",
      "        \"name\": \"macrophage\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"macrophage\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - myometrium|1 - smooth muscle|1 - uterus|2 - muscle|2 - vertebrate muscular system|2 - internal female genital organ|3 - muscular system|3 - female reproductive system|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"myometrium\", \n",
      "        \"name\": \"myometrium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"myometrium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - neutrophil|1 - granulocyte|1 - phagocyte|2 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"neutrophil\", \n",
      "        \"name\": \"neutrophil\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"neutrophil\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - osteoblast|1 - bone|1 - osteogenic cell|1 - preosteoblast|2 - connective tissue|2 - skeletal system|2 - periosteum|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"osteoblast\", \n",
      "        \"name\": \"osteoblast\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"osteoblast\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - tear gland|1 - eye|1 - excretory gland|2 - head|2 - sense organ|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"tear_gland\", \n",
      "        \"name\": \"tear_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"tear gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - animal|1 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"all_tissues\", \n",
      "        \"name\": \"all_tissues\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"animal\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - bone marrow|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"bone_marrow\", \n",
      "        \"name\": \"bone_marrow\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"bone marrow\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - chondrocyte|1 - cartilage|2 - connective tissue|2 - skeletal system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"chondrocyte\", \n",
      "        \"name\": \"chondrocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"chondrocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - granulocyte|1 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"granulocyte\", \n",
      "        \"name\": \"granulocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"granulocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hippocampus|1 - temporal lobe|1 - limbic system|2 - cerebral lobe|2 - brain|3 - cerebral cortex|3 - head|3 - central nervous system|4 - cerebral hemisphere|4 - whole body|4 - nervous system|5 - telencephalon|5 - animal|6 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"hippocampus\", \n",
      "        \"name\": \"hippocampus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hippocampus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spinal cord|1 - central nervous system|2 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"spinal_cord\", \n",
      "        \"name\": \"spinal_cord\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"spinal cord\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - trophoblast|1 - blastula|2 - morula|3 - blastomere|4 - zygote|5 - embryonic structure|6 - whole body|6 - organism form|7 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"trophoblast\", \n",
      "        \"name\": \"trophoblast\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"trophoblast\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - B-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"b_lymphocyte\", \n",
      "        \"name\": \"b_lymphocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"B-lymphocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood plasma|1 - blood|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"blood_plasma\", \n",
      "        \"name\": \"blood_plasma\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"blood plasma\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"blood_vessel\", \n",
      "        \"name\": \"blood_vessel\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"blood vessel\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - diencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"diencephalon\", \n",
      "        \"name\": \"diencephalon\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"diencephalon\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - frontal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"frontal_lobe\", \n",
      "        \"name\": \"frontal_lobe\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"frontal lobe\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hypothalamus|1 - diencephalon|1 - endocrine gland|1 - limbic system|2 - forebrain|2 - gland|2 - brain|3 - whole body|3 - head|3 - central nervous system|4 - animal|4 - nervous system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"hypothalamus\", \n",
      "        \"name\": \"hypothalamus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hypothalamus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - keratinocyte|1 - stratum spinosum|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"keratinocyte\", \n",
      "        \"name\": \"keratinocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"keratinocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - renal tubule|1 - nephron|2 - kidney|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|4 - gland|4 - whole body|4 - urogenital system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"renal_tubule\", \n",
      "        \"name\": \"renal_tubule\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"renal tubule\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spermatocyte|1 - spermatogonium|1 - semen|1 - germ cell|2 - male reproductive system|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"spermatocyte\", \n",
      "        \"name\": \"spermatocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"spermatocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - T-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"t_lymphocyte\", \n",
      "        \"name\": \"t_lymphocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"T-lymphocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"adrenal_gland\", \n",
      "        \"name\": \"adrenal_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"adrenal gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - corpus luteum|1 - mature ovarian follicle|2 - ovarian follicle|3 - ovary|4 - female reproductive gland|4 - internal female genital organ|5 - female reproductive system|5 - endocrine gland|6 - reproductive system|6 - gland|7 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"corpus_luteum\", \n",
      "        \"name\": \"corpus_luteum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"corpus luteum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - dentate gyrus|1 - hippocampus|2 - temporal lobe|2 - limbic system|3 - cerebral lobe|3 - brain|4 - cerebral cortex|4 - head|4 - central nervous system|5 - cerebral hemisphere|5 - whole body|5 - nervous system|6 - telencephalon|6 - animal|7 - forebrain|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"dentate_gyrus\", \n",
      "        \"name\": \"dentate_gyrus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"dentate gyrus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hair follicle|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"hair_follicle\", \n",
      "        \"name\": \"hair_follicle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hair follicle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - mammary gland|1 - breast|2 - thorax|2 - female reproductive gland|2 - excretory gland|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"mammary_gland\", \n",
      "        \"name\": \"mammary_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"mammary gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - megakaryocyte|1 - megakaryoblast|1 - marrow cell|2 - bone marrow|2 - bone marrow cell|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - skeletal system|3 - blood|3 - hematopoietic stem cell|4 - whole body|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"megakaryocyte\", \n",
      "        \"name\": \"megakaryocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"megakaryocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - parietal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"parietal_lobe\", \n",
      "        \"name\": \"parietal_lobe\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"parietal lobe\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - smooth muscle|1 - muscle|1 - vertebrate muscular system|2 - muscular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"smooth_muscle\", \n",
      "        \"name\": \"smooth_muscle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"smooth muscle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - telencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"telencephalon\", \n",
      "        \"name\": \"telencephalon\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"telencephalon\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - temporal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"temporal_lobe\", \n",
      "        \"name\": \"temporal_lobe\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"temporal lobe\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"thyroid_gland\", \n",
      "        \"name\": \"thyroid_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uroepithelium|1 - epithelium|1 - urinary tract|2 - integument|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"uroepithelium\", \n",
      "        \"name\": \"uroepithelium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uroepithelium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"adipose_tissue\", \n",
      "        \"name\": \"adipose_tissue\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"adipose tissue\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal cortex|1 - adrenal gland|2 - viscus|2 - endocrine gland|3 - whole body|3 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"adrenal_cortex\", \n",
      "        \"name\": \"adrenal_cortex\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"adrenal cortex\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - basal ganglion|1 - ganglion|1 - cerebral cortex|2 - nervous system|2 - cerebral hemisphere|3 - whole body|3 - telencephalon|4 - animal|4 - forebrain|5 - tissues, cell types and enzyme sources|5 - brain|6 - head|6 - central nervous system\", \n",
      "        \"description\": \"basal_ganglion\", \n",
      "        \"name\": \"basal_ganglion\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"basal ganglion\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood platelet|1 - megakaryocyte|1 - blood plasma|2 - megakaryoblast|2 - marrow cell|2 - blood|3 - bone marrow|3 - bone marrow cell|3 - hematopoietic cell|3 - myeloid progenitor cell|3 - hematopoietic system|4 - skeletal system|4 - hematopoietic stem cell|4 - whole body|5 - blast cell|5 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"blood_platelet\", \n",
      "        \"name\": \"blood_platelet\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"blood platelet\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cardiac muscle|1 - muscle|1 - vertebrate muscular system|1 - heart|2 - muscular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cardiac_muscle\", \n",
      "        \"name\": \"cardiac_muscle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cardiac muscle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - dendritic cell|1 - lymph node|1 - spleen|1 - peripheral blood|2 - immune system|2 - endocrine gland|2 - lymphoid tissue|2 - viscus|2 - blood|3 - whole body|3 - gland|3 - hematopoietic system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"dendritic_cell\", \n",
      "        \"name\": \"dendritic_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"dendritic cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - locus ceruleus|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"locus_ceruleus\", \n",
      "        \"name\": \"locus_ceruleus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"locus ceruleus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - nervous system|1 - whole body|2 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"nervous_system\", \n",
      "        \"name\": \"nervous_system\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"nervous system\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - occipital lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"occipital_lobe\", \n",
      "        \"name\": \"occipital_lobe\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"occipital lobe\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - occipital pole|1 - occipital lobe|2 - cerebral lobe|3 - cerebral cortex|4 - cerebral hemisphere|5 - telencephalon|6 - forebrain|7 - brain|8 - head|8 - central nervous system|9 - whole body|9 - nervous system|10 - animal|11 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"occipital_pole\", \n",
      "        \"name\": \"occipital_pole\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"occipital pole\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"prostate_gland\", \n",
      "        \"name\": \"prostate_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - salivary gland|1 - mouth|1 - exocrine gland|2 - head|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"salivary_gland\", \n",
      "        \"name\": \"salivary_gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"salivary gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spermatogonium|1 - male reproductive system|1 - germ cell|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"spermatogonium\", \n",
      "        \"name\": \"spermatogonium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"spermatogonium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical cord|1 - placenta|1 - connecting stalk|1 - fetus|2 - internal female genital organ|2 - mesoderm|2 - embryonic structure|3 - female reproductive system|3 - germ layer|3 - whole body|3 - organism form|4 - reproductive system|4 - embryo|4 - animal|4 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"description\": \"umbilical_cord\", \n",
      "        \"name\": \"umbilical_cord\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"umbilical cord\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uterine cervix|1 - uterus|2 - internal female genital organ|3 - female reproductive system|4 - reproductive system|5 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"uterine_cervix\", \n",
      "        \"name\": \"uterine_cervix\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uterine cervix\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - caudate nucleus|1 - corpus striatum|1 - neostriatum|2 - basal ganglion|3 - ganglion|3 - cerebral cortex|4 - nervous system|4 - cerebral hemisphere|5 - whole body|5 - telencephalon|6 - animal|6 - forebrain|7 - tissues, cell types and enzyme sources|7 - brain|8 - head|8 - central nervous system\", \n",
      "        \"description\": \"caudate_nucleus\", \n",
      "        \"name\": \"caudate_nucleus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"caudate nucleus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - caudate putamen|1 - nucleus lentiformis|1 - neostriatum|2 - corpus striatum|3 - basal ganglion|4 - ganglion|4 - cerebral cortex|5 - nervous system|5 - cerebral hemisphere|6 - whole body|6 - telencephalon|7 - animal|7 - forebrain|8 - tissues, cell types and enzyme sources|8 - brain|9 - head|9 - central nervous system\", \n",
      "        \"description\": \"caudate_putamen\", \n",
      "        \"name\": \"caudate_putamen\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"caudate putamen\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cerebral cortex|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cerebral_cortex\", \n",
      "        \"name\": \"cerebral_cortex\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cerebral cortex\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - corpus callosum|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"corpus_callosum\", \n",
      "        \"name\": \"corpus_callosum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"corpus callosum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - corpus striatum|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"description\": \"corpus_striatum\", \n",
      "        \"name\": \"corpus_striatum\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"corpus striatum\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - large intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"large_intestine\", \n",
      "        \"name\": \"large_intestine\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"large intestine\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - skeletal muscle|1 - muscle|1 - vertebrate muscular system|1 - skeletal system|2 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"skeletal_muscle\", \n",
      "        \"name\": \"skeletal_muscle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"skeletal muscle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - skin fibroblast|1 - fibroblast|1 - skin|2 - connective tissue|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"skin_fibroblast\", \n",
      "        \"name\": \"skin_fibroblast\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"skin fibroblast\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - small intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"small_intestine\", \n",
      "        \"name\": \"small_intestine\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"small intestine\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder|1 - bladder|1 - urinary system|1 - urinary tract|2 - viscus|2 - urogenital system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"urinary_bladder\", \n",
      "        \"name\": \"urinary_bladder\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"urinary bladder\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian follicle|1 - ovary|2 - female reproductive gland|2 - internal female genital organ|3 - female reproductive system|3 - endocrine gland|4 - reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"ovarian_follicle\", \n",
      "        \"name\": \"ovarian_follicle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"ovarian follicle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic islet|1 - endocrine pancreas|2 - pancreas|3 - digestive gland|3 - viscus|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"pancreatic_islet\", \n",
      "        \"name\": \"pancreatic_islet\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"pancreatic islet\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - renal glomerulus|1 - renal corpuscle|2 - nephron|3 - kidney|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|5 - gland|5 - whole body|5 - urogenital system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"renal_glomerulus\", \n",
      "        \"name\": \"renal_glomerulus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"renal glomerulus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - substantia nigra|1 - midbrain|1 - basal ganglion|2 - brain stem|2 - ganglion|2 - cerebral cortex|3 - brain|3 - nervous system|3 - cerebral hemisphere|4 - head|4 - central nervous system|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"substantia_nigra\", \n",
      "        \"name\": \"substantia_nigra\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"substantia nigra\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellar cortex|1 - cerebellum|2 - metencephalon|3 - hindbrain|4 - brain stem|5 - brain|6 - head|6 - central nervous system|7 - whole body|7 - nervous system|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"cerebellar_cortex\", \n",
      "        \"name\": \"cerebellar_cortex\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"cerebellar cortex\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - medulla oblongata|1 - myelencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"medulla_oblongata\", \n",
      "        \"name\": \"medulla_oblongata\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"medulla oblongata\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - nucleus accumbens|1 - lateral ventricle|2 - brain ventricle|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"nucleus_accumbens\", \n",
      "        \"name\": \"nucleus_accumbens\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"nucleus accumbens\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - mammary epithelium|1 - breast epithelium|2 - breast|2 - epithelium|3 - thorax|3 - female reproductive gland|3 - excretory gland|3 - integument|4 - trunk|4 - female reproductive system|4 - endocrine gland|4 - gland|4 - whole body|5 - reproductive system|5 - animal|6 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"mammary_epithelium\", \n",
      "        \"name\": \"mammary_epithelium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"mammary epithelium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - vermiform appendix|1 - cecum|2 - large intestine|2 - ileocecum|3 - intestine|4 - gastrointestinal tract|5 - alimentary canal|6 - viscus|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"vermiform_appendix\", \n",
      "        \"name\": \"vermiform_appendix\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"vermiform appendix\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - natural killer cell|1 - null cell|1 - large granular lymphocyte|2 - lymphocyte|3 - leukocyte|3 - lymphoid cell|4 - hematopoietic cell|4 - lymphoid tissue|5 - blood|5 - immune system|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"natural_killer_cell\", \n",
      "        \"name\": \"natural_killer_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"natural killer cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - subthalamic nucleus|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"description\": \"subthalamic_nucleus\", \n",
      "        \"name\": \"subthalamic_nucleus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"subthalamic nucleus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - uterine endometrium|1 - mucosa|1 - uterus|2 - integument|2 - internal female genital organ|3 - whole body|3 - female reproductive system|4 - animal|4 - reproductive system|5 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"description\": \"uterine_endometrium\", \n",
      "        \"name\": \"uterine_endometrium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"uterine endometrium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelium|1 - endothelium|2 - epithelium|3 - integument|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"vascular_endothelium\", \n",
      "        \"name\": \"vascular_endothelium\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"vascular endothelium\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - mononuclear phagocyte|1 - mononuclear cell|1 - marrow cell|1 - phagocyte|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - blood|3 - hematopoietic stem cell|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"mononuclear_phagocyte\", \n",
      "        \"name\": \"mononuclear_phagocyte\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"mononuclear phagocyte\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - central nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"central_nervous_system\", \n",
      "        \"name\": \"central_nervous_system\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"central nervous system\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - gastrointestinal tract|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"gastrointestinal_tract\", \n",
      "        \"name\": \"gastrointestinal_tract\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"gastrointestinal tract\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hematopoietic stem cell|1 - blast cell|1 - adult stem cell|2 - blood|2 - whole body|3 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"hematopoietic_stem_cell\", \n",
      "        \"name\": \"hematopoietic_stem_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hematopoietic stem cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - bronchial epithelial cell|1 - bronchial epithelium|2 - respiratory epithelium|2 - bronchus|3 - bronchoalveolar system|3 - epithelium|4 - lung|4 - integument|5 - respiratory system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"bronchial_epithelial_cell\", \n",
      "        \"name\": \"bronchial_epithelial_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"bronchial epithelial cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - peripheral nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"peripheral_nervous_system\", \n",
      "        \"name\": \"peripheral_nervous_system\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"peripheral nervous system\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelial cell|1 - vascular endothelium|1 - endothelial cell|2 - endothelium|2 - blood vessel endothelium|2 - heart|3 - epithelium|3 - blood vessel|3 - cardiovascular system|4 - integument|4 - vascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"vascular_endothelial_cell\", \n",
      "        \"name\": \"vascular_endothelial_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"vascular endothelial cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - culture condition:CD8+ cell|1 - culture condition:antigen-presenting cell|2 - culture condition|3 - cell culture|4 - other source|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"culture_condition_cd8_cell\", \n",
      "        \"name\": \"culture_condition_cd8_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"culture condition:CD8+ cell\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical vein endothelial cell|1 - umbilical vein endothelium|1 - vascular endothelial cell|2 - umbilical vein|2 - endothelium|2 - epithelium|2 - vascular endothelium|2 - endothelial cell|3 - umbilical cord|3 - integument|3 - blood vessel endothelium|3 - heart|4 - placenta|4 - connecting stalk|4 - fetus|4 - whole body|4 - blood vessel|4 - cardiovascular system|5 - internal female genital organ|5 - mesoderm|5 - embryonic structure|5 - animal|5 - vascular system|6 - female reproductive system|6 - germ layer|6 - organism form|6 - tissues, cell types and enzyme sources|7 - reproductive system|7 - embryo|8 - urogenital system\", \n",
      "        \"description\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"name\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\", \n",
      "            \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"umbilical vein endothelial cell\"\n",
      "      }\n",
      "    ]\n",
      "  }, \n",
      "  {\n",
      "    \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \", \n",
      "    \"name\": \"GTEx\", \n",
      "    \"substudies\": [\n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Lung\", \n",
      "        \"name\": \"Lung\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Lung_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Lung\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Skin\", \n",
      "        \"name\": \"Skin\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Skin_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Skin\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Blood\", \n",
      "        \"name\": \"Blood\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Blood\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Brain\", \n",
      "        \"name\": \"Brain\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Brain_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Brain\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Colon\", \n",
      "        \"name\": \"Colon\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Colon_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Colon\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Heart\", \n",
      "        \"name\": \"Heart\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Heart_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Heart\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Liver\", \n",
      "        \"name\": \"Liver\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Liver_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Liver\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - nerve|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Nerve\", \n",
      "        \"name\": \"Nerve\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Nerve_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Nerve\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - breast|1 - thorax|1 - female reproductive gland|1 - excretory gland|2 - trunk|2 - female reproductive system|2 - endocrine gland|2 - gland|3 - whole body|3 - reproductive system|4 - animal|4 - urogenital system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Breast\", \n",
      "        \"name\": \"Breast\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Breast_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Breast\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Muscle\", \n",
      "        \"name\": \"Muscle\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Muscle_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Muscle\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Spleen\", \n",
      "        \"name\": \"Spleen\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Spleen_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Spleen\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Testis\", \n",
      "        \"name\": \"Testis\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Testis_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Testis\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Stomach\", \n",
      "        \"name\": \"Stomach\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Stomach_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Stomach\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Thyroid\", \n",
      "        \"name\": \"Thyroid\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Thyroid_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Pancreas\", \n",
      "        \"name\": \"Pancreas\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pancreas_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Pancreas\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Prostate\", \n",
      "        \"name\": \"Prostate\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Prostate_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Esophagus\", \n",
      "        \"name\": \"Esophagus\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Esophagus_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Esophagus\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"description\": \"Pituitary\", \n",
      "        \"name\": \"Pituitary\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Pituitary_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Blood_Vessel\", \n",
      "        \"name\": \"Blood_Vessel\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Blood_Vessel_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Blood Vessel\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Adrenal_Gland\", \n",
      "        \"name\": \"Adrenal_Gland\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adrenal_Gland_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Adrenal Gland\"\n",
      "      }, \n",
      "      {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"description\": \"Adipose_Tissue\", \n",
      "        \"name\": \"Adipose_Tissue\", \n",
      "        \"columns\": [\n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"BigGIM_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"CRC_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_70_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_80_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_90_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Correlation\", \n",
      "            \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "          }, \n",
      "          {\n",
      "            \"datatype\": \"FLOAT\", \n",
      "            \"table\": {\n",
      "              \"name\": \"FA_v1\"\n",
      "            }, \n",
      "            \"name\": \"GTEx_Adipose_Tissue_Pvalue\", \n",
      "            \"interactions_type\": \"P-value (-log10)\"\n",
      "          }\n",
      "        ], \n",
      "        \"cell_of_origin\": \"Adipose Tissue\"\n",
      "      }\n",
      "    ]\n",
      "  }\n",
      "]\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    studies = get('metadata/study')\n",
    "    jprint(studies)\n",
    "except requests.HTTPError as e:\n",
    "    print e"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_study_descriptions\"></a>\n",
    "### Study descriptions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Study Name: BioGRID\n",
      "Study Description:  Biological General Repository for Interaction Datasets\n",
      "        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\n",
      "\n",
      "        https://thebiogrid.org/\n",
      "        \n",
      "Study Name: TCGA\n",
      "Study Description:  The Cancer Genome Atlas\n",
      "        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\n",
      "\n",
      "        https://cancergenome.nih.gov/\n",
      "        \n",
      "Study Name: GIANT\n",
      "Study Description: Genetic Investigation of ANthropometric Traits\n",
      "        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\n",
      "\n",
      "        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\n",
      "        \n",
      "Study Name: GTEx\n",
      "Study Description: Genotype-Tissue Expression (GTEx) program\n",
      "\n",
      "        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \n",
      "        \n",
      "        https://www.gtexportal.org/home/\n",
      "        \n"
     ]
    }
   ],
   "source": [
    "for study in studies:\n",
    "    print(\"Study Name: %s\" % study['name'])\n",
    "    print(\"Study Description: %s\" % study['description'])\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_single_study\"></a>\n",
    "### Get a single study"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \", \n",
      "  \"name\": \"TCGA\", \n",
      "  \"substudies\": [\n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - ovarian serous carcinoma cell|1 - ovary cancer cell|2 - ovary|3 - female reproductive gland|3 - internal female genital organ|4 - female reproductive system|4 - endocrine gland|5 - reproductive system|5 - gland|6 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Ovarian serous cystadenocarcinoma\", \n",
      "      \"name\": \"OV\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_OV_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"ovarian serous carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - adrenocortical carcinoma cell|1 - adrenal cortex|2 - adrenal gland|3 - viscus|3 - endocrine gland|4 - whole body|4 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Adrenocortical carcinoma\", \n",
      "      \"name\": \"ACC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ACC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"adrenocortical carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - glioblastoma cell|1 - astrocytoma cell|2 - glioma cell|2 - astrocyte|3 - glia|3 - glial cell|3 - astroglial cell|3 - macroglia|3 - astroblast|4 - spinal cord|4 - brain|4 - astroglia|4 - neuroblast|5 - central nervous system|5 - head|5 - embryo|6 - nervous system|6 - whole body|6 - embryonic structure|7 - animal|7 - organism form|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Glioblastoma multiforme\", \n",
      "      \"name\": \"GBM\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_GBM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"glioblastoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - glioma cell|1 - glia|1 - glial cell|2 - spinal cord|2 - brain|3 - central nervous system|3 - head|4 - nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Brain Lower Grade Glioma\", \n",
      "      \"name\": \"LGG\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LGG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"glioma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - uterine sarcoma cell|1 - sarcoma cell|1 - uterine cancer cell|2 - cartilage|2 - skeletal muscle cancer cell|2 - bone cancer cell|2 - uterus|3 - connective tissue|3 - skeletal system|3 - skeletal muscle cell|3 - bone|3 - internal female genital organ|4 - whole body|4 - skeletal muscle|4 - female reproductive system|5 - animal|5 - muscle|5 - vertebrate muscular system|5 - reproductive system|6 - tissues, cell types and enzyme sources|6 - muscular system|6 - urogenital system\", \n",
      "      \"description\": \"Uterine Carcinosarcoma\", \n",
      "      \"name\": \"UCS\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCS_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"uterine sarcoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - uveal melanoma cell|1 - melanoma cell|2 - skin cancer cell|2 - melanocyte|3 - skin|3 - epidermal cell|4 - integument|4 - epidermis|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Uveal Melanoma\", \n",
      "      \"name\": \"UVM\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UVM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"uveal melanoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - urinary bladder cancer cell|1 - urinary bladder|2 - bladder|2 - urinary system|2 - urinary tract|3 - viscus|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Bladder Urothelial Carcinoma\", \n",
      "      \"name\": \"BLCA\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BLCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"urinary bladder cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - breast cancer cell|1 - breast|1 - adenocarcinoma cell|2 - thorax|2 - female reproductive gland|2 - excretory gland|2 - carcinoma cell|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|3 - epithelioma cell|4 - whole body|4 - reproductive system|4 - epithelial cell|5 - animal|5 - urogenital system|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "      \"description\": \"Breast invasive carcinoma\", \n",
      "      \"name\": \"BRCA\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_BRCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"breast cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - cervical adenocarcinoma cell|0 - cervical squamous cell carcinoma cell|1 - cervical carcinoma cell|1 - adenocarcinoma cell|1 - squamous cell carcinoma cell|2 - uterine cervix|2 - carcinoma cell|2 - squamous epithelium|3 - uterus|3 - epithelioma cell|3 - epithelium|4 - internal female genital organ|4 - epithelial cell|4 - integument|5 - female reproductive system|5 - whole body|6 - reproductive system|6 - animal|7 - urogenital system|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Cervical squamous cell carcinoma and endocervical adenocarcinoma\", \n",
      "      \"name\": \"CESC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CESC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"cervical adenocarcinoma cell,cervical squamous cell carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - cholangiocarcinoma cell|1 - bile duct epithelium|1 - bile duct|2 - biliary epithelium|2 - liver|3 - epithelium|3 - digestive gland|3 - viscus|4 - integument|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Cholangiocarcinoma\", \n",
      "      \"name\": \"CHOL\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_CHOL_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"cholangiocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - colorectal adenocarcinoma cell|1 - rectal cancer cell|1 - colonic cancer cell|1 - adenocarcinoma cell|2 - rectum|2 - gastrointestinal cancer cell|2 - colon|2 - carcinoma cell|3 - large intestine|3 - colorectum|3 - gastrointestinal tract|3 - epithelioma cell|4 - intestine|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Colon adenocarcinoma\", \n",
      "      \"name\": \"COAD\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_COAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"colorectal adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - diffuse large B-cell lymphoma cell|1 - B-lymphoma cell line|2 - lymphoma cell line|3 - lymphoid cell line|3 - lymphoma cell|4 - lymphoid cell|4 - blood cancer cell|4 - lymphoid tissue|5 - hematopoietic cell|5 - immune system|5 - hematopoietic system|6 - blood|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\", \n",
      "      \"name\": \"DLBC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"diffuse large B-cell lymphoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - esophageal cancer cell|1 - esophagus|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Esophageal carcinoma\", \n",
      "      \"name\": \"ESCA\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_ESCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"esophageal cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - head and neck cancer cell|1 - head|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Head and Neck squamous cell carcinoma\", \n",
      "      \"name\": \"HNSC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_HNSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"head and neck cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - chromophobe renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "      \"description\": \"Kidney Chromophobe\", \n",
      "      \"name\": \"KICH\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KICH_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"chromophobe renal cell carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - renal clear cell|0 - renal cell carcinoma cell|1 - renal tubule|1 - kidney cancer cell|1 - adenocarcinoma cell|2 - nephron|2 - kidney|2 - carcinoma cell|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|3 - epithelioma cell|4 - gland|4 - whole body|4 - urogenital system|4 - epithelial cell|5 - animal|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "      \"description\": \"Kidney renal clear cell carcinoma\", \n",
      "      \"name\": \"KIRC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"renal clear cell,renal cell carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - papillary renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "      \"description\": \"Kidney renal papillary cell carcinoma\", \n",
      "      \"name\": \"KIRP\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_KIRP_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"papillary renal cell carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - acute myeloid leukemia cell|1 - myeloid leukemia cell|2 - leukemia cell|3 - leukocyte|3 - blood cancer cell|4 - hematopoietic cell|5 - blood|6 - hematopoietic system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Acute Myeloid Leukemia\", \n",
      "      \"name\": \"LAML\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LAML_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"acute myeloid leukemia cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - liver cancer cell|0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Liver hepatocellular carcinoma\", \n",
      "      \"name\": \"LIHC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LIHC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"liver cancer cell,hepatocyte\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - lung adenocarcinoma cell|1 - carcinoma cell|1 - non-small cell lung cancer cell|2 - epithelioma cell|2 - bronchogenic carcinoma cell|3 - epithelial cell|3 - lung cancer cell|4 - epithelium|4 - lung|5 - integument|5 - respiratory system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Lung adenocarcinoma\", \n",
      "      \"name\": \"LUAD\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"lung adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - lung squamous cell carcinoma cell|1 - lung cancer cell|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Lung squamous cell carcinoma\", \n",
      "      \"name\": \"LUSC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_LUSC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"lung squamous cell carcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - mesothelioma cell|1 - mesothelium|2 - mesoderm|2 - squamous epithelium|2 - coelom|3 - germ layer|3 - epithelium|4 - embryo|4 - integument|5 - embryonic structure|5 - whole body|6 - organism form|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Mesothelioma\", \n",
      "      \"name\": \"MESO\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_MESO_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"mesothelioma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - pancreatic ductal adenocarcinoma cell|1 - pancreatic ductal carcinoma cell|1 - adenocarcinoma cell|2 - pancreatic duct|2 - carcinoma cell|3 - exocrine pancreas|3 - epithelioma cell|4 - pancreas|4 - epithelial cell|5 - digestive gland|5 - viscus|5 - epithelium|6 - endocrine gland|6 - whole body|6 - integument|7 - gland|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Pancreatic adenocarcinoma\", \n",
      "      \"name\": \"PAAD\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PAAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"pancreatic ductal adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - pheochromocytoma cell|0 - neural crest cell|0 - neuroendocrine tumor cell|1 - adrenal gland cancer cell|1 - neural crest|1 - neuroendocrine cell|1 - carcinoid cell|2 - adrenal gland|2 - neural plate|2 - endocrine gland|2 - gastrointestinal cancer cell|2 - intestinal mucosa|3 - viscus|3 - ectoderm|3 - gland|3 - gastrointestinal tract|3 - mucosa|3 - intestine|4 - whole body|4 - germ layer|4 - alimentary canal|4 - integument|5 - animal|5 - embryo|6 - tissues, cell types and enzyme sources|6 - embryonic structure|7 - organism form\", \n",
      "      \"description\": \"Pheochromocytoma and Paraganglioma\", \n",
      "      \"name\": \"PCPG\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PCPG_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"pheochromocytoma cell,neural crest cell,neuroendocrine tumor cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - prostate adenocarcinoma cell|1 - adenocarcinoma cell|2 - carcinoma cell|3 - epithelioma cell|4 - epithelial cell|5 - epithelium|6 - integument|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Prostate adenocarcinoma\", \n",
      "      \"name\": \"PRAD\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_PRAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"prostate adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - rectal adenocarcinoma cell|1 - colorectal adenocarcinoma cell|2 - rectal cancer cell|2 - colonic cancer cell|2 - adenocarcinoma cell|3 - rectum|3 - gastrointestinal cancer cell|3 - colon|3 - carcinoma cell|4 - large intestine|4 - colorectum|4 - gastrointestinal tract|4 - epithelioma cell|5 - intestine|5 - alimentary canal|5 - epithelial cell|6 - viscus|6 - epithelium|7 - whole body|7 - integument|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Rectum adenocarcinoma\", \n",
      "      \"name\": \"READ\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_READ_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"rectal adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - sarcoma cell|1 - cartilage|1 - skeletal muscle cancer cell|1 - bone cancer cell|2 - connective tissue|2 - skeletal system|2 - skeletal muscle cell|2 - bone|3 - whole body|3 - skeletal muscle|4 - animal|4 - muscle|4 - vertebrate muscular system|5 - tissues, cell types and enzyme sources|5 - muscular system\", \n",
      "      \"description\": \"Sarcoma\", \n",
      "      \"name\": \"SARC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SARC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"sarcoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - skin cancer cell|0 - melanoma cell|1 - skin|1 - melanocyte|2 - integument|2 - epidermal cell|3 - whole body|3 - epidermis|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Skin Cutaneous Melanoma\", \n",
      "      \"name\": \"SKCM\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_SKCM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"skin cancer cell,melanoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - gastric adenocarcinoma cell|1 - gastric cancer cell|1 - adenocarcinoma cell|2 - stomach|2 - carcinoma cell|3 - gastrointestinal tract|3 - epithelioma cell|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Stomach adenocarcinoma\", \n",
      "      \"name\": \"STAD\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_STAD_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"gastric adenocarcinoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - testicular cancer cell|1 - testis|2 - internal male genital organ|2 - gonad|2 - male reproductive gland|3 - male reproductive system|3 - female reproductive gland|3 - endocrine gland|4 - reproductive system|4 - female reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Testicular Germ Cell Tumors\", \n",
      "      \"name\": \"TGCT\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_TGCT_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"testicular cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - thyroid cancer cell|1 - thyroid gland|2 - throat|2 - endocrine gland|3 - neck|3 - gland|4 - trunk|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Thyroid carcinoma\", \n",
      "      \"name\": \"THCA\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THCA_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"thyroid cancer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - thymoma cell|1 - thymic cancer cell|2 - thymus|3 - immune system|3 - thorax|3 - endocrine gland|3 - lymphoid tissue|4 - whole body|4 - trunk|4 - gland|4 - hematopoietic system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Thymoma\", \n",
      "      \"name\": \"THYM\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_THYM_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"thymoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - endometrioid carcinoma cell|1 - uterine endometrium|1 - ovary cancer cell|2 - mucosa|2 - uterus|2 - ovary|3 - integument|3 - internal female genital organ|3 - female reproductive gland|4 - whole body|4 - female reproductive system|4 - endocrine gland|5 - animal|5 - reproductive system|5 - gland|6 - tissues, cell types and enzyme sources|6 - urogenital system\", \n",
      "      \"description\": \"Uterine Corpus Endometrial Carcinoma\", \n",
      "      \"name\": \"UCEC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_UCEC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"endometrioid carcinoma cell\"\n",
      "    }\n",
      "  ]\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    tcga = get('metadata/study/%s' % ('TCGA'))\n",
    "    jprint(tcga)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_substudies\"></a>\n",
    "### Get a substudies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Substudy name: OV \n",
      "Substudy description: Ovarian serous cystadenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_OV_Correlation, BigGIM_70_v1.TCGA_OV_Pvalue, BigGIM_80_v1.TCGA_OV_Correlation, BigGIM_80_v1.TCGA_OV_Pvalue, BigGIM_90_v1.TCGA_OV_Correlation, BigGIM_90_v1.TCGA_OV_Pvalue, BigGIM_v1.TCGA_OV_Correlation, BigGIM_v1.TCGA_OV_Pvalue, CRC_70_v1.TCGA_OV_Correlation, CRC_70_v1.TCGA_OV_Pvalue, CRC_80_v1.TCGA_OV_Correlation, CRC_80_v1.TCGA_OV_Pvalue, CRC_90_v1.TCGA_OV_Correlation, CRC_90_v1.TCGA_OV_Pvalue, CRC_v1.TCGA_OV_Correlation, CRC_v1.TCGA_OV_Pvalue, FA_70_v1.TCGA_OV_Correlation, FA_70_v1.TCGA_OV_Pvalue, FA_80_v1.TCGA_OV_Correlation, FA_80_v1.TCGA_OV_Pvalue, FA_90_v1.TCGA_OV_Correlation, FA_90_v1.TCGA_OV_Pvalue, FA_v1.TCGA_OV_Correlation, FA_v1.TCGA_OV_Pvalue\n",
      "\n",
      "Substudy name: ACC \n",
      "Substudy description: Adrenocortical carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_ACC_Correlation, BigGIM_70_v1.TCGA_ACC_Pvalue, BigGIM_80_v1.TCGA_ACC_Correlation, BigGIM_80_v1.TCGA_ACC_Pvalue, BigGIM_90_v1.TCGA_ACC_Correlation, BigGIM_90_v1.TCGA_ACC_Pvalue, BigGIM_v1.TCGA_ACC_Correlation, BigGIM_v1.TCGA_ACC_Pvalue, CRC_70_v1.TCGA_ACC_Correlation, CRC_70_v1.TCGA_ACC_Pvalue, CRC_80_v1.TCGA_ACC_Correlation, CRC_80_v1.TCGA_ACC_Pvalue, CRC_90_v1.TCGA_ACC_Correlation, CRC_90_v1.TCGA_ACC_Pvalue, CRC_v1.TCGA_ACC_Correlation, CRC_v1.TCGA_ACC_Pvalue, FA_70_v1.TCGA_ACC_Correlation, FA_70_v1.TCGA_ACC_Pvalue, FA_80_v1.TCGA_ACC_Correlation, FA_80_v1.TCGA_ACC_Pvalue, FA_90_v1.TCGA_ACC_Correlation, FA_90_v1.TCGA_ACC_Pvalue, FA_v1.TCGA_ACC_Correlation, FA_v1.TCGA_ACC_Pvalue\n",
      "\n",
      "Substudy name: GBM \n",
      "Substudy description: Glioblastoma multiforme\n",
      "Table.column list: BigGIM_70_v1.TCGA_GBM_Correlation, BigGIM_70_v1.TCGA_GBM_Pvalue, BigGIM_80_v1.TCGA_GBM_Correlation, BigGIM_80_v1.TCGA_GBM_Pvalue, BigGIM_90_v1.TCGA_GBM_Correlation, BigGIM_90_v1.TCGA_GBM_Pvalue, BigGIM_v1.TCGA_GBM_Correlation, BigGIM_v1.TCGA_GBM_Pvalue, CRC_70_v1.TCGA_GBM_Correlation, CRC_70_v1.TCGA_GBM_Pvalue, CRC_80_v1.TCGA_GBM_Correlation, CRC_80_v1.TCGA_GBM_Pvalue, CRC_90_v1.TCGA_GBM_Correlation, CRC_90_v1.TCGA_GBM_Pvalue, CRC_v1.TCGA_GBM_Correlation, CRC_v1.TCGA_GBM_Pvalue, FA_70_v1.TCGA_GBM_Correlation, FA_70_v1.TCGA_GBM_Pvalue, FA_80_v1.TCGA_GBM_Correlation, FA_80_v1.TCGA_GBM_Pvalue, FA_90_v1.TCGA_GBM_Correlation, FA_90_v1.TCGA_GBM_Pvalue, FA_v1.TCGA_GBM_Correlation, FA_v1.TCGA_GBM_Pvalue\n",
      "\n",
      "Substudy name: LGG \n",
      "Substudy description: Brain Lower Grade Glioma\n",
      "Table.column list: BigGIM_70_v1.TCGA_LGG_Correlation, BigGIM_70_v1.TCGA_LGG_Pvalue, BigGIM_80_v1.TCGA_LGG_Correlation, BigGIM_80_v1.TCGA_LGG_Pvalue, BigGIM_90_v1.TCGA_LGG_Correlation, BigGIM_90_v1.TCGA_LGG_Pvalue, BigGIM_v1.TCGA_LGG_Correlation, BigGIM_v1.TCGA_LGG_Pvalue, CRC_70_v1.TCGA_LGG_Correlation, CRC_70_v1.TCGA_LGG_Pvalue, CRC_80_v1.TCGA_LGG_Correlation, CRC_80_v1.TCGA_LGG_Pvalue, CRC_90_v1.TCGA_LGG_Correlation, CRC_90_v1.TCGA_LGG_Pvalue, CRC_v1.TCGA_LGG_Correlation, CRC_v1.TCGA_LGG_Pvalue, FA_70_v1.TCGA_LGG_Correlation, FA_70_v1.TCGA_LGG_Pvalue, FA_80_v1.TCGA_LGG_Correlation, FA_80_v1.TCGA_LGG_Pvalue, FA_90_v1.TCGA_LGG_Correlation, FA_90_v1.TCGA_LGG_Pvalue, FA_v1.TCGA_LGG_Correlation, FA_v1.TCGA_LGG_Pvalue\n",
      "\n",
      "Substudy name: UCS \n",
      "Substudy description: Uterine Carcinosarcoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_UCS_Correlation, BigGIM_70_v1.TCGA_UCS_Pvalue, BigGIM_80_v1.TCGA_UCS_Correlation, BigGIM_80_v1.TCGA_UCS_Pvalue, BigGIM_90_v1.TCGA_UCS_Correlation, BigGIM_90_v1.TCGA_UCS_Pvalue, BigGIM_v1.TCGA_UCS_Correlation, BigGIM_v1.TCGA_UCS_Pvalue, CRC_70_v1.TCGA_UCS_Correlation, CRC_70_v1.TCGA_UCS_Pvalue, CRC_80_v1.TCGA_UCS_Correlation, CRC_80_v1.TCGA_UCS_Pvalue, CRC_90_v1.TCGA_UCS_Correlation, CRC_90_v1.TCGA_UCS_Pvalue, CRC_v1.TCGA_UCS_Correlation, CRC_v1.TCGA_UCS_Pvalue, FA_70_v1.TCGA_UCS_Correlation, FA_70_v1.TCGA_UCS_Pvalue, FA_80_v1.TCGA_UCS_Correlation, FA_80_v1.TCGA_UCS_Pvalue, FA_90_v1.TCGA_UCS_Correlation, FA_90_v1.TCGA_UCS_Pvalue, FA_v1.TCGA_UCS_Correlation, FA_v1.TCGA_UCS_Pvalue\n",
      "\n",
      "Substudy name: UVM \n",
      "Substudy description: Uveal Melanoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_UVM_Correlation, BigGIM_70_v1.TCGA_UVM_Pvalue, BigGIM_80_v1.TCGA_UVM_Correlation, BigGIM_80_v1.TCGA_UVM_Pvalue, BigGIM_90_v1.TCGA_UVM_Correlation, BigGIM_90_v1.TCGA_UVM_Pvalue, BigGIM_v1.TCGA_UVM_Correlation, BigGIM_v1.TCGA_UVM_Pvalue, CRC_70_v1.TCGA_UVM_Correlation, CRC_70_v1.TCGA_UVM_Pvalue, CRC_80_v1.TCGA_UVM_Correlation, CRC_80_v1.TCGA_UVM_Pvalue, CRC_90_v1.TCGA_UVM_Correlation, CRC_90_v1.TCGA_UVM_Pvalue, CRC_v1.TCGA_UVM_Correlation, CRC_v1.TCGA_UVM_Pvalue, FA_70_v1.TCGA_UVM_Correlation, FA_70_v1.TCGA_UVM_Pvalue, FA_80_v1.TCGA_UVM_Correlation, FA_80_v1.TCGA_UVM_Pvalue, FA_90_v1.TCGA_UVM_Correlation, FA_90_v1.TCGA_UVM_Pvalue, FA_v1.TCGA_UVM_Correlation, FA_v1.TCGA_UVM_Pvalue\n",
      "\n",
      "Substudy name: BLCA \n",
      "Substudy description: Bladder Urothelial Carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_BLCA_Correlation, BigGIM_70_v1.TCGA_BLCA_Pvalue, BigGIM_80_v1.TCGA_BLCA_Correlation, BigGIM_80_v1.TCGA_BLCA_Pvalue, BigGIM_90_v1.TCGA_BLCA_Correlation, BigGIM_90_v1.TCGA_BLCA_Pvalue, BigGIM_v1.TCGA_BLCA_Correlation, BigGIM_v1.TCGA_BLCA_Pvalue, CRC_70_v1.TCGA_BLCA_Correlation, CRC_70_v1.TCGA_BLCA_Pvalue, CRC_80_v1.TCGA_BLCA_Correlation, CRC_80_v1.TCGA_BLCA_Pvalue, CRC_90_v1.TCGA_BLCA_Correlation, CRC_90_v1.TCGA_BLCA_Pvalue, CRC_v1.TCGA_BLCA_Correlation, CRC_v1.TCGA_BLCA_Pvalue, FA_70_v1.TCGA_BLCA_Correlation, FA_70_v1.TCGA_BLCA_Pvalue, FA_80_v1.TCGA_BLCA_Correlation, FA_80_v1.TCGA_BLCA_Pvalue, FA_90_v1.TCGA_BLCA_Correlation, FA_90_v1.TCGA_BLCA_Pvalue, FA_v1.TCGA_BLCA_Correlation, FA_v1.TCGA_BLCA_Pvalue\n",
      "\n",
      "Substudy name: BRCA \n",
      "Substudy description: Breast invasive carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_BRCA_Correlation, BigGIM_70_v1.TCGA_BRCA_Pvalue, BigGIM_80_v1.TCGA_BRCA_Correlation, BigGIM_80_v1.TCGA_BRCA_Pvalue, BigGIM_90_v1.TCGA_BRCA_Correlation, BigGIM_90_v1.TCGA_BRCA_Pvalue, BigGIM_v1.TCGA_BRCA_Correlation, BigGIM_v1.TCGA_BRCA_Pvalue, CRC_70_v1.TCGA_BRCA_Correlation, CRC_70_v1.TCGA_BRCA_Pvalue, CRC_80_v1.TCGA_BRCA_Correlation, CRC_80_v1.TCGA_BRCA_Pvalue, CRC_90_v1.TCGA_BRCA_Correlation, CRC_90_v1.TCGA_BRCA_Pvalue, CRC_v1.TCGA_BRCA_Correlation, CRC_v1.TCGA_BRCA_Pvalue, FA_70_v1.TCGA_BRCA_Correlation, FA_70_v1.TCGA_BRCA_Pvalue, FA_80_v1.TCGA_BRCA_Correlation, FA_80_v1.TCGA_BRCA_Pvalue, FA_90_v1.TCGA_BRCA_Correlation, FA_90_v1.TCGA_BRCA_Pvalue, FA_v1.TCGA_BRCA_Correlation, FA_v1.TCGA_BRCA_Pvalue\n",
      "\n",
      "Substudy name: CESC \n",
      "Substudy description: Cervical squamous cell carcinoma and endocervical adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_CESC_Correlation, BigGIM_70_v1.TCGA_CESC_Pvalue, BigGIM_80_v1.TCGA_CESC_Correlation, BigGIM_80_v1.TCGA_CESC_Pvalue, BigGIM_90_v1.TCGA_CESC_Correlation, BigGIM_90_v1.TCGA_CESC_Pvalue, BigGIM_v1.TCGA_CESC_Correlation, BigGIM_v1.TCGA_CESC_Pvalue, CRC_70_v1.TCGA_CESC_Correlation, CRC_70_v1.TCGA_CESC_Pvalue, CRC_80_v1.TCGA_CESC_Correlation, CRC_80_v1.TCGA_CESC_Pvalue, CRC_90_v1.TCGA_CESC_Correlation, CRC_90_v1.TCGA_CESC_Pvalue, CRC_v1.TCGA_CESC_Correlation, CRC_v1.TCGA_CESC_Pvalue, FA_70_v1.TCGA_CESC_Correlation, FA_70_v1.TCGA_CESC_Pvalue, FA_80_v1.TCGA_CESC_Correlation, FA_80_v1.TCGA_CESC_Pvalue, FA_90_v1.TCGA_CESC_Correlation, FA_90_v1.TCGA_CESC_Pvalue, FA_v1.TCGA_CESC_Correlation, FA_v1.TCGA_CESC_Pvalue\n",
      "\n",
      "Substudy name: CHOL \n",
      "Substudy description: Cholangiocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_CHOL_Correlation, BigGIM_70_v1.TCGA_CHOL_Pvalue, BigGIM_80_v1.TCGA_CHOL_Correlation, BigGIM_80_v1.TCGA_CHOL_Pvalue, BigGIM_90_v1.TCGA_CHOL_Correlation, BigGIM_90_v1.TCGA_CHOL_Pvalue, BigGIM_v1.TCGA_CHOL_Correlation, BigGIM_v1.TCGA_CHOL_Pvalue, CRC_70_v1.TCGA_CHOL_Correlation, CRC_70_v1.TCGA_CHOL_Pvalue, CRC_80_v1.TCGA_CHOL_Correlation, CRC_80_v1.TCGA_CHOL_Pvalue, CRC_90_v1.TCGA_CHOL_Correlation, CRC_90_v1.TCGA_CHOL_Pvalue, CRC_v1.TCGA_CHOL_Correlation, CRC_v1.TCGA_CHOL_Pvalue, FA_70_v1.TCGA_CHOL_Correlation, FA_70_v1.TCGA_CHOL_Pvalue, FA_80_v1.TCGA_CHOL_Correlation, FA_80_v1.TCGA_CHOL_Pvalue, FA_90_v1.TCGA_CHOL_Correlation, FA_90_v1.TCGA_CHOL_Pvalue, FA_v1.TCGA_CHOL_Correlation, FA_v1.TCGA_CHOL_Pvalue\n",
      "\n",
      "Substudy name: COAD \n",
      "Substudy description: Colon adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_COAD_Correlation, BigGIM_70_v1.TCGA_COAD_Pvalue, BigGIM_80_v1.TCGA_COAD_Correlation, BigGIM_80_v1.TCGA_COAD_Pvalue, BigGIM_90_v1.TCGA_COAD_Correlation, BigGIM_90_v1.TCGA_COAD_Pvalue, BigGIM_v1.TCGA_COAD_Correlation, BigGIM_v1.TCGA_COAD_Pvalue, CRC_70_v1.TCGA_COAD_Correlation, CRC_70_v1.TCGA_COAD_Pvalue, CRC_80_v1.TCGA_COAD_Correlation, CRC_80_v1.TCGA_COAD_Pvalue, CRC_90_v1.TCGA_COAD_Correlation, CRC_90_v1.TCGA_COAD_Pvalue, CRC_v1.TCGA_COAD_Correlation, CRC_v1.TCGA_COAD_Pvalue, FA_70_v1.TCGA_COAD_Correlation, FA_70_v1.TCGA_COAD_Pvalue, FA_80_v1.TCGA_COAD_Correlation, FA_80_v1.TCGA_COAD_Pvalue, FA_90_v1.TCGA_COAD_Correlation, FA_90_v1.TCGA_COAD_Pvalue, FA_v1.TCGA_COAD_Correlation, FA_v1.TCGA_COAD_Pvalue\n",
      "\n",
      "Substudy name: DLBC \n",
      "Substudy description: Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_DLBC_Correlation, BigGIM_70_v1.TCGA_DLBC_Pvalue, BigGIM_80_v1.TCGA_DLBC_Correlation, BigGIM_80_v1.TCGA_DLBC_Pvalue, BigGIM_90_v1.TCGA_DLBC_Correlation, BigGIM_90_v1.TCGA_DLBC_Pvalue, BigGIM_v1.TCGA_DLBC_Correlation, BigGIM_v1.TCGA_DLBC_Pvalue, CRC_70_v1.TCGA_DLBC_Correlation, CRC_70_v1.TCGA_DLBC_Pvalue, CRC_80_v1.TCGA_DLBC_Correlation, CRC_80_v1.TCGA_DLBC_Pvalue, CRC_90_v1.TCGA_DLBC_Correlation, CRC_90_v1.TCGA_DLBC_Pvalue, CRC_v1.TCGA_DLBC_Correlation, CRC_v1.TCGA_DLBC_Pvalue, FA_70_v1.TCGA_DLBC_Correlation, FA_70_v1.TCGA_DLBC_Pvalue, FA_80_v1.TCGA_DLBC_Correlation, FA_80_v1.TCGA_DLBC_Pvalue, FA_90_v1.TCGA_DLBC_Correlation, FA_90_v1.TCGA_DLBC_Pvalue, FA_v1.TCGA_DLBC_Correlation, FA_v1.TCGA_DLBC_Pvalue\n",
      "\n",
      "Substudy name: ESCA \n",
      "Substudy description: Esophageal carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_ESCA_Correlation, BigGIM_70_v1.TCGA_ESCA_Pvalue, BigGIM_80_v1.TCGA_ESCA_Correlation, BigGIM_80_v1.TCGA_ESCA_Pvalue, BigGIM_90_v1.TCGA_ESCA_Correlation, BigGIM_90_v1.TCGA_ESCA_Pvalue, BigGIM_v1.TCGA_ESCA_Correlation, BigGIM_v1.TCGA_ESCA_Pvalue, CRC_70_v1.TCGA_ESCA_Correlation, CRC_70_v1.TCGA_ESCA_Pvalue, CRC_80_v1.TCGA_ESCA_Correlation, CRC_80_v1.TCGA_ESCA_Pvalue, CRC_90_v1.TCGA_ESCA_Correlation, CRC_90_v1.TCGA_ESCA_Pvalue, CRC_v1.TCGA_ESCA_Correlation, CRC_v1.TCGA_ESCA_Pvalue, FA_70_v1.TCGA_ESCA_Correlation, FA_70_v1.TCGA_ESCA_Pvalue, FA_80_v1.TCGA_ESCA_Correlation, FA_80_v1.TCGA_ESCA_Pvalue, FA_90_v1.TCGA_ESCA_Correlation, FA_90_v1.TCGA_ESCA_Pvalue, FA_v1.TCGA_ESCA_Correlation, FA_v1.TCGA_ESCA_Pvalue\n",
      "\n",
      "Substudy name: HNSC \n",
      "Substudy description: Head and Neck squamous cell carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_HNSC_Correlation, BigGIM_70_v1.TCGA_HNSC_Pvalue, BigGIM_80_v1.TCGA_HNSC_Correlation, BigGIM_80_v1.TCGA_HNSC_Pvalue, BigGIM_90_v1.TCGA_HNSC_Correlation, BigGIM_90_v1.TCGA_HNSC_Pvalue, BigGIM_v1.TCGA_HNSC_Correlation, BigGIM_v1.TCGA_HNSC_Pvalue, CRC_70_v1.TCGA_HNSC_Correlation, CRC_70_v1.TCGA_HNSC_Pvalue, CRC_80_v1.TCGA_HNSC_Correlation, CRC_80_v1.TCGA_HNSC_Pvalue, CRC_90_v1.TCGA_HNSC_Correlation, CRC_90_v1.TCGA_HNSC_Pvalue, CRC_v1.TCGA_HNSC_Correlation, CRC_v1.TCGA_HNSC_Pvalue, FA_70_v1.TCGA_HNSC_Correlation, FA_70_v1.TCGA_HNSC_Pvalue, FA_80_v1.TCGA_HNSC_Correlation, FA_80_v1.TCGA_HNSC_Pvalue, FA_90_v1.TCGA_HNSC_Correlation, FA_90_v1.TCGA_HNSC_Pvalue, FA_v1.TCGA_HNSC_Correlation, FA_v1.TCGA_HNSC_Pvalue\n",
      "\n",
      "Substudy name: KICH \n",
      "Substudy description: Kidney Chromophobe\n",
      "Table.column list: BigGIM_70_v1.TCGA_KICH_Correlation, BigGIM_70_v1.TCGA_KICH_Pvalue, BigGIM_80_v1.TCGA_KICH_Correlation, BigGIM_80_v1.TCGA_KICH_Pvalue, BigGIM_90_v1.TCGA_KICH_Correlation, BigGIM_90_v1.TCGA_KICH_Pvalue, BigGIM_v1.TCGA_KICH_Correlation, BigGIM_v1.TCGA_KICH_Pvalue, CRC_70_v1.TCGA_KICH_Correlation, CRC_70_v1.TCGA_KICH_Pvalue, CRC_80_v1.TCGA_KICH_Correlation, CRC_80_v1.TCGA_KICH_Pvalue, CRC_90_v1.TCGA_KICH_Correlation, CRC_90_v1.TCGA_KICH_Pvalue, CRC_v1.TCGA_KICH_Correlation, CRC_v1.TCGA_KICH_Pvalue, FA_70_v1.TCGA_KICH_Correlation, FA_70_v1.TCGA_KICH_Pvalue, FA_80_v1.TCGA_KICH_Correlation, FA_80_v1.TCGA_KICH_Pvalue, FA_90_v1.TCGA_KICH_Correlation, FA_90_v1.TCGA_KICH_Pvalue, FA_v1.TCGA_KICH_Correlation, FA_v1.TCGA_KICH_Pvalue\n",
      "\n",
      "Substudy name: KIRC \n",
      "Substudy description: Kidney renal clear cell carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_KIRC_Correlation, BigGIM_70_v1.TCGA_KIRC_Pvalue, BigGIM_80_v1.TCGA_KIRC_Correlation, BigGIM_80_v1.TCGA_KIRC_Pvalue, BigGIM_90_v1.TCGA_KIRC_Correlation, BigGIM_90_v1.TCGA_KIRC_Pvalue, BigGIM_v1.TCGA_KIRC_Correlation, BigGIM_v1.TCGA_KIRC_Pvalue, CRC_70_v1.TCGA_KIRC_Correlation, CRC_70_v1.TCGA_KIRC_Pvalue, CRC_80_v1.TCGA_KIRC_Correlation, CRC_80_v1.TCGA_KIRC_Pvalue, CRC_90_v1.TCGA_KIRC_Correlation, CRC_90_v1.TCGA_KIRC_Pvalue, CRC_v1.TCGA_KIRC_Correlation, CRC_v1.TCGA_KIRC_Pvalue, FA_70_v1.TCGA_KIRC_Correlation, FA_70_v1.TCGA_KIRC_Pvalue, FA_80_v1.TCGA_KIRC_Correlation, FA_80_v1.TCGA_KIRC_Pvalue, FA_90_v1.TCGA_KIRC_Correlation, FA_90_v1.TCGA_KIRC_Pvalue, FA_v1.TCGA_KIRC_Correlation, FA_v1.TCGA_KIRC_Pvalue\n",
      "\n",
      "Substudy name: KIRP \n",
      "Substudy description: Kidney renal papillary cell carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_KIRP_Correlation, BigGIM_70_v1.TCGA_KIRP_Pvalue, BigGIM_80_v1.TCGA_KIRP_Correlation, BigGIM_80_v1.TCGA_KIRP_Pvalue, BigGIM_90_v1.TCGA_KIRP_Correlation, BigGIM_90_v1.TCGA_KIRP_Pvalue, BigGIM_v1.TCGA_KIRP_Correlation, BigGIM_v1.TCGA_KIRP_Pvalue, CRC_70_v1.TCGA_KIRP_Correlation, CRC_70_v1.TCGA_KIRP_Pvalue, CRC_80_v1.TCGA_KIRP_Correlation, CRC_80_v1.TCGA_KIRP_Pvalue, CRC_90_v1.TCGA_KIRP_Correlation, CRC_90_v1.TCGA_KIRP_Pvalue, CRC_v1.TCGA_KIRP_Correlation, CRC_v1.TCGA_KIRP_Pvalue, FA_70_v1.TCGA_KIRP_Correlation, FA_70_v1.TCGA_KIRP_Pvalue, FA_80_v1.TCGA_KIRP_Correlation, FA_80_v1.TCGA_KIRP_Pvalue, FA_90_v1.TCGA_KIRP_Correlation, FA_90_v1.TCGA_KIRP_Pvalue, FA_v1.TCGA_KIRP_Correlation, FA_v1.TCGA_KIRP_Pvalue\n",
      "\n",
      "Substudy name: LAML \n",
      "Substudy description: Acute Myeloid Leukemia\n",
      "Table.column list: BigGIM_70_v1.TCGA_LAML_Correlation, BigGIM_70_v1.TCGA_LAML_Pvalue, BigGIM_80_v1.TCGA_LAML_Correlation, BigGIM_80_v1.TCGA_LAML_Pvalue, BigGIM_90_v1.TCGA_LAML_Correlation, BigGIM_90_v1.TCGA_LAML_Pvalue, BigGIM_v1.TCGA_LAML_Correlation, BigGIM_v1.TCGA_LAML_Pvalue, CRC_70_v1.TCGA_LAML_Correlation, CRC_70_v1.TCGA_LAML_Pvalue, CRC_80_v1.TCGA_LAML_Correlation, CRC_80_v1.TCGA_LAML_Pvalue, CRC_90_v1.TCGA_LAML_Correlation, CRC_90_v1.TCGA_LAML_Pvalue, CRC_v1.TCGA_LAML_Correlation, CRC_v1.TCGA_LAML_Pvalue, FA_70_v1.TCGA_LAML_Correlation, FA_70_v1.TCGA_LAML_Pvalue, FA_80_v1.TCGA_LAML_Correlation, FA_80_v1.TCGA_LAML_Pvalue, FA_90_v1.TCGA_LAML_Correlation, FA_90_v1.TCGA_LAML_Pvalue, FA_v1.TCGA_LAML_Correlation, FA_v1.TCGA_LAML_Pvalue\n",
      "\n",
      "Substudy name: LIHC \n",
      "Substudy description: Liver hepatocellular carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_LIHC_Correlation, BigGIM_70_v1.TCGA_LIHC_Pvalue, BigGIM_80_v1.TCGA_LIHC_Correlation, BigGIM_80_v1.TCGA_LIHC_Pvalue, BigGIM_90_v1.TCGA_LIHC_Correlation, BigGIM_90_v1.TCGA_LIHC_Pvalue, BigGIM_v1.TCGA_LIHC_Correlation, BigGIM_v1.TCGA_LIHC_Pvalue, CRC_70_v1.TCGA_LIHC_Correlation, CRC_70_v1.TCGA_LIHC_Pvalue, CRC_80_v1.TCGA_LIHC_Correlation, CRC_80_v1.TCGA_LIHC_Pvalue, CRC_90_v1.TCGA_LIHC_Correlation, CRC_90_v1.TCGA_LIHC_Pvalue, CRC_v1.TCGA_LIHC_Correlation, CRC_v1.TCGA_LIHC_Pvalue, FA_70_v1.TCGA_LIHC_Correlation, FA_70_v1.TCGA_LIHC_Pvalue, FA_80_v1.TCGA_LIHC_Correlation, FA_80_v1.TCGA_LIHC_Pvalue, FA_90_v1.TCGA_LIHC_Correlation, FA_90_v1.TCGA_LIHC_Pvalue, FA_v1.TCGA_LIHC_Correlation, FA_v1.TCGA_LIHC_Pvalue\n",
      "\n",
      "Substudy name: LUAD \n",
      "Substudy description: Lung adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_LUAD_Correlation, BigGIM_70_v1.TCGA_LUAD_Pvalue, BigGIM_80_v1.TCGA_LUAD_Correlation, BigGIM_80_v1.TCGA_LUAD_Pvalue, BigGIM_90_v1.TCGA_LUAD_Correlation, BigGIM_90_v1.TCGA_LUAD_Pvalue, BigGIM_v1.TCGA_LUAD_Correlation, BigGIM_v1.TCGA_LUAD_Pvalue, CRC_70_v1.TCGA_LUAD_Correlation, CRC_70_v1.TCGA_LUAD_Pvalue, CRC_80_v1.TCGA_LUAD_Correlation, CRC_80_v1.TCGA_LUAD_Pvalue, CRC_90_v1.TCGA_LUAD_Correlation, CRC_90_v1.TCGA_LUAD_Pvalue, CRC_v1.TCGA_LUAD_Correlation, CRC_v1.TCGA_LUAD_Pvalue, FA_70_v1.TCGA_LUAD_Correlation, FA_70_v1.TCGA_LUAD_Pvalue, FA_80_v1.TCGA_LUAD_Correlation, FA_80_v1.TCGA_LUAD_Pvalue, FA_90_v1.TCGA_LUAD_Correlation, FA_90_v1.TCGA_LUAD_Pvalue, FA_v1.TCGA_LUAD_Correlation, FA_v1.TCGA_LUAD_Pvalue\n",
      "\n",
      "Substudy name: LUSC \n",
      "Substudy description: Lung squamous cell carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_LUSC_Correlation, BigGIM_70_v1.TCGA_LUSC_Pvalue, BigGIM_80_v1.TCGA_LUSC_Correlation, BigGIM_80_v1.TCGA_LUSC_Pvalue, BigGIM_90_v1.TCGA_LUSC_Correlation, BigGIM_90_v1.TCGA_LUSC_Pvalue, BigGIM_v1.TCGA_LUSC_Correlation, BigGIM_v1.TCGA_LUSC_Pvalue, CRC_70_v1.TCGA_LUSC_Correlation, CRC_70_v1.TCGA_LUSC_Pvalue, CRC_80_v1.TCGA_LUSC_Correlation, CRC_80_v1.TCGA_LUSC_Pvalue, CRC_90_v1.TCGA_LUSC_Correlation, CRC_90_v1.TCGA_LUSC_Pvalue, CRC_v1.TCGA_LUSC_Correlation, CRC_v1.TCGA_LUSC_Pvalue, FA_70_v1.TCGA_LUSC_Correlation, FA_70_v1.TCGA_LUSC_Pvalue, FA_80_v1.TCGA_LUSC_Correlation, FA_80_v1.TCGA_LUSC_Pvalue, FA_90_v1.TCGA_LUSC_Correlation, FA_90_v1.TCGA_LUSC_Pvalue, FA_v1.TCGA_LUSC_Correlation, FA_v1.TCGA_LUSC_Pvalue\n",
      "\n",
      "Substudy name: MESO \n",
      "Substudy description: Mesothelioma\n",
      "Table.column list: BigGIM_70_v1.TCGA_MESO_Correlation, BigGIM_70_v1.TCGA_MESO_Pvalue, BigGIM_80_v1.TCGA_MESO_Correlation, BigGIM_80_v1.TCGA_MESO_Pvalue, BigGIM_90_v1.TCGA_MESO_Correlation, BigGIM_90_v1.TCGA_MESO_Pvalue, BigGIM_v1.TCGA_MESO_Correlation, BigGIM_v1.TCGA_MESO_Pvalue, CRC_70_v1.TCGA_MESO_Correlation, CRC_70_v1.TCGA_MESO_Pvalue, CRC_80_v1.TCGA_MESO_Correlation, CRC_80_v1.TCGA_MESO_Pvalue, CRC_90_v1.TCGA_MESO_Correlation, CRC_90_v1.TCGA_MESO_Pvalue, CRC_v1.TCGA_MESO_Correlation, CRC_v1.TCGA_MESO_Pvalue, FA_70_v1.TCGA_MESO_Correlation, FA_70_v1.TCGA_MESO_Pvalue, FA_80_v1.TCGA_MESO_Correlation, FA_80_v1.TCGA_MESO_Pvalue, FA_90_v1.TCGA_MESO_Correlation, FA_90_v1.TCGA_MESO_Pvalue, FA_v1.TCGA_MESO_Correlation, FA_v1.TCGA_MESO_Pvalue\n",
      "\n",
      "Substudy name: PAAD \n",
      "Substudy description: Pancreatic adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_PAAD_Correlation, BigGIM_70_v1.TCGA_PAAD_Pvalue, BigGIM_80_v1.TCGA_PAAD_Correlation, BigGIM_80_v1.TCGA_PAAD_Pvalue, BigGIM_90_v1.TCGA_PAAD_Correlation, BigGIM_90_v1.TCGA_PAAD_Pvalue, BigGIM_v1.TCGA_PAAD_Correlation, BigGIM_v1.TCGA_PAAD_Pvalue, CRC_70_v1.TCGA_PAAD_Correlation, CRC_70_v1.TCGA_PAAD_Pvalue, CRC_80_v1.TCGA_PAAD_Correlation, CRC_80_v1.TCGA_PAAD_Pvalue, CRC_90_v1.TCGA_PAAD_Correlation, CRC_90_v1.TCGA_PAAD_Pvalue, CRC_v1.TCGA_PAAD_Correlation, CRC_v1.TCGA_PAAD_Pvalue, FA_70_v1.TCGA_PAAD_Correlation, FA_70_v1.TCGA_PAAD_Pvalue, FA_80_v1.TCGA_PAAD_Correlation, FA_80_v1.TCGA_PAAD_Pvalue, FA_90_v1.TCGA_PAAD_Correlation, FA_90_v1.TCGA_PAAD_Pvalue, FA_v1.TCGA_PAAD_Correlation, FA_v1.TCGA_PAAD_Pvalue\n",
      "\n",
      "Substudy name: PCPG \n",
      "Substudy description: Pheochromocytoma and Paraganglioma\n",
      "Table.column list: BigGIM_70_v1.TCGA_PCPG_Correlation, BigGIM_70_v1.TCGA_PCPG_Pvalue, BigGIM_80_v1.TCGA_PCPG_Correlation, BigGIM_80_v1.TCGA_PCPG_Pvalue, BigGIM_90_v1.TCGA_PCPG_Correlation, BigGIM_90_v1.TCGA_PCPG_Pvalue, BigGIM_v1.TCGA_PCPG_Correlation, BigGIM_v1.TCGA_PCPG_Pvalue, CRC_70_v1.TCGA_PCPG_Correlation, CRC_70_v1.TCGA_PCPG_Pvalue, CRC_80_v1.TCGA_PCPG_Correlation, CRC_80_v1.TCGA_PCPG_Pvalue, CRC_90_v1.TCGA_PCPG_Correlation, CRC_90_v1.TCGA_PCPG_Pvalue, CRC_v1.TCGA_PCPG_Correlation, CRC_v1.TCGA_PCPG_Pvalue, FA_70_v1.TCGA_PCPG_Correlation, FA_70_v1.TCGA_PCPG_Pvalue, FA_80_v1.TCGA_PCPG_Correlation, FA_80_v1.TCGA_PCPG_Pvalue, FA_90_v1.TCGA_PCPG_Correlation, FA_90_v1.TCGA_PCPG_Pvalue, FA_v1.TCGA_PCPG_Correlation, FA_v1.TCGA_PCPG_Pvalue\n",
      "\n",
      "Substudy name: PRAD \n",
      "Substudy description: Prostate adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_PRAD_Correlation, BigGIM_70_v1.TCGA_PRAD_Pvalue, BigGIM_80_v1.TCGA_PRAD_Correlation, BigGIM_80_v1.TCGA_PRAD_Pvalue, BigGIM_90_v1.TCGA_PRAD_Correlation, BigGIM_90_v1.TCGA_PRAD_Pvalue, BigGIM_v1.TCGA_PRAD_Correlation, BigGIM_v1.TCGA_PRAD_Pvalue, CRC_70_v1.TCGA_PRAD_Correlation, CRC_70_v1.TCGA_PRAD_Pvalue, CRC_80_v1.TCGA_PRAD_Correlation, CRC_80_v1.TCGA_PRAD_Pvalue, CRC_90_v1.TCGA_PRAD_Correlation, CRC_90_v1.TCGA_PRAD_Pvalue, CRC_v1.TCGA_PRAD_Correlation, CRC_v1.TCGA_PRAD_Pvalue, FA_70_v1.TCGA_PRAD_Correlation, FA_70_v1.TCGA_PRAD_Pvalue, FA_80_v1.TCGA_PRAD_Correlation, FA_80_v1.TCGA_PRAD_Pvalue, FA_90_v1.TCGA_PRAD_Correlation, FA_90_v1.TCGA_PRAD_Pvalue, FA_v1.TCGA_PRAD_Correlation, FA_v1.TCGA_PRAD_Pvalue\n",
      "\n",
      "Substudy name: READ \n",
      "Substudy description: Rectum adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_READ_Correlation, BigGIM_70_v1.TCGA_READ_Pvalue, BigGIM_80_v1.TCGA_READ_Correlation, BigGIM_80_v1.TCGA_READ_Pvalue, BigGIM_90_v1.TCGA_READ_Correlation, BigGIM_90_v1.TCGA_READ_Pvalue, BigGIM_v1.TCGA_READ_Correlation, BigGIM_v1.TCGA_READ_Pvalue, CRC_70_v1.TCGA_READ_Correlation, CRC_70_v1.TCGA_READ_Pvalue, CRC_80_v1.TCGA_READ_Correlation, CRC_80_v1.TCGA_READ_Pvalue, CRC_90_v1.TCGA_READ_Correlation, CRC_90_v1.TCGA_READ_Pvalue, CRC_v1.TCGA_READ_Correlation, CRC_v1.TCGA_READ_Pvalue, FA_70_v1.TCGA_READ_Correlation, FA_70_v1.TCGA_READ_Pvalue, FA_80_v1.TCGA_READ_Correlation, FA_80_v1.TCGA_READ_Pvalue, FA_90_v1.TCGA_READ_Correlation, FA_90_v1.TCGA_READ_Pvalue, FA_v1.TCGA_READ_Correlation, FA_v1.TCGA_READ_Pvalue\n",
      "\n",
      "Substudy name: SARC \n",
      "Substudy description: Sarcoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_SARC_Correlation, BigGIM_70_v1.TCGA_SARC_Pvalue, BigGIM_80_v1.TCGA_SARC_Correlation, BigGIM_80_v1.TCGA_SARC_Pvalue, BigGIM_90_v1.TCGA_SARC_Correlation, BigGIM_90_v1.TCGA_SARC_Pvalue, BigGIM_v1.TCGA_SARC_Correlation, BigGIM_v1.TCGA_SARC_Pvalue, CRC_70_v1.TCGA_SARC_Correlation, CRC_70_v1.TCGA_SARC_Pvalue, CRC_80_v1.TCGA_SARC_Correlation, CRC_80_v1.TCGA_SARC_Pvalue, CRC_90_v1.TCGA_SARC_Correlation, CRC_90_v1.TCGA_SARC_Pvalue, CRC_v1.TCGA_SARC_Correlation, CRC_v1.TCGA_SARC_Pvalue, FA_70_v1.TCGA_SARC_Correlation, FA_70_v1.TCGA_SARC_Pvalue, FA_80_v1.TCGA_SARC_Correlation, FA_80_v1.TCGA_SARC_Pvalue, FA_90_v1.TCGA_SARC_Correlation, FA_90_v1.TCGA_SARC_Pvalue, FA_v1.TCGA_SARC_Correlation, FA_v1.TCGA_SARC_Pvalue\n",
      "\n",
      "Substudy name: SKCM \n",
      "Substudy description: Skin Cutaneous Melanoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_SKCM_Correlation, BigGIM_70_v1.TCGA_SKCM_Pvalue, BigGIM_80_v1.TCGA_SKCM_Correlation, BigGIM_80_v1.TCGA_SKCM_Pvalue, BigGIM_90_v1.TCGA_SKCM_Correlation, BigGIM_90_v1.TCGA_SKCM_Pvalue, BigGIM_v1.TCGA_SKCM_Correlation, BigGIM_v1.TCGA_SKCM_Pvalue, CRC_70_v1.TCGA_SKCM_Correlation, CRC_70_v1.TCGA_SKCM_Pvalue, CRC_80_v1.TCGA_SKCM_Correlation, CRC_80_v1.TCGA_SKCM_Pvalue, CRC_90_v1.TCGA_SKCM_Correlation, CRC_90_v1.TCGA_SKCM_Pvalue, CRC_v1.TCGA_SKCM_Correlation, CRC_v1.TCGA_SKCM_Pvalue, FA_70_v1.TCGA_SKCM_Correlation, FA_70_v1.TCGA_SKCM_Pvalue, FA_80_v1.TCGA_SKCM_Correlation, FA_80_v1.TCGA_SKCM_Pvalue, FA_90_v1.TCGA_SKCM_Correlation, FA_90_v1.TCGA_SKCM_Pvalue, FA_v1.TCGA_SKCM_Correlation, FA_v1.TCGA_SKCM_Pvalue\n",
      "\n",
      "Substudy name: STAD \n",
      "Substudy description: Stomach adenocarcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_STAD_Correlation, BigGIM_70_v1.TCGA_STAD_Pvalue, BigGIM_80_v1.TCGA_STAD_Correlation, BigGIM_80_v1.TCGA_STAD_Pvalue, BigGIM_90_v1.TCGA_STAD_Correlation, BigGIM_90_v1.TCGA_STAD_Pvalue, BigGIM_v1.TCGA_STAD_Correlation, BigGIM_v1.TCGA_STAD_Pvalue, CRC_70_v1.TCGA_STAD_Correlation, CRC_70_v1.TCGA_STAD_Pvalue, CRC_80_v1.TCGA_STAD_Correlation, CRC_80_v1.TCGA_STAD_Pvalue, CRC_90_v1.TCGA_STAD_Correlation, CRC_90_v1.TCGA_STAD_Pvalue, CRC_v1.TCGA_STAD_Correlation, CRC_v1.TCGA_STAD_Pvalue, FA_70_v1.TCGA_STAD_Correlation, FA_70_v1.TCGA_STAD_Pvalue, FA_80_v1.TCGA_STAD_Correlation, FA_80_v1.TCGA_STAD_Pvalue, FA_90_v1.TCGA_STAD_Correlation, FA_90_v1.TCGA_STAD_Pvalue, FA_v1.TCGA_STAD_Correlation, FA_v1.TCGA_STAD_Pvalue\n",
      "\n",
      "Substudy name: TGCT \n",
      "Substudy description: Testicular Germ Cell Tumors\n",
      "Table.column list: BigGIM_70_v1.TCGA_TGCT_Correlation, BigGIM_70_v1.TCGA_TGCT_Pvalue, BigGIM_80_v1.TCGA_TGCT_Correlation, BigGIM_80_v1.TCGA_TGCT_Pvalue, BigGIM_90_v1.TCGA_TGCT_Correlation, BigGIM_90_v1.TCGA_TGCT_Pvalue, BigGIM_v1.TCGA_TGCT_Correlation, BigGIM_v1.TCGA_TGCT_Pvalue, CRC_70_v1.TCGA_TGCT_Correlation, CRC_70_v1.TCGA_TGCT_Pvalue, CRC_80_v1.TCGA_TGCT_Correlation, CRC_80_v1.TCGA_TGCT_Pvalue, CRC_90_v1.TCGA_TGCT_Correlation, CRC_90_v1.TCGA_TGCT_Pvalue, CRC_v1.TCGA_TGCT_Correlation, CRC_v1.TCGA_TGCT_Pvalue, FA_70_v1.TCGA_TGCT_Correlation, FA_70_v1.TCGA_TGCT_Pvalue, FA_80_v1.TCGA_TGCT_Correlation, FA_80_v1.TCGA_TGCT_Pvalue, FA_90_v1.TCGA_TGCT_Correlation, FA_90_v1.TCGA_TGCT_Pvalue, FA_v1.TCGA_TGCT_Correlation, FA_v1.TCGA_TGCT_Pvalue\n",
      "\n",
      "Substudy name: THCA \n",
      "Substudy description: Thyroid carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_THCA_Correlation, BigGIM_70_v1.TCGA_THCA_Pvalue, BigGIM_80_v1.TCGA_THCA_Correlation, BigGIM_80_v1.TCGA_THCA_Pvalue, BigGIM_90_v1.TCGA_THCA_Correlation, BigGIM_90_v1.TCGA_THCA_Pvalue, BigGIM_v1.TCGA_THCA_Correlation, BigGIM_v1.TCGA_THCA_Pvalue, CRC_70_v1.TCGA_THCA_Correlation, CRC_70_v1.TCGA_THCA_Pvalue, CRC_80_v1.TCGA_THCA_Correlation, CRC_80_v1.TCGA_THCA_Pvalue, CRC_90_v1.TCGA_THCA_Correlation, CRC_90_v1.TCGA_THCA_Pvalue, CRC_v1.TCGA_THCA_Correlation, CRC_v1.TCGA_THCA_Pvalue, FA_70_v1.TCGA_THCA_Correlation, FA_70_v1.TCGA_THCA_Pvalue, FA_80_v1.TCGA_THCA_Correlation, FA_80_v1.TCGA_THCA_Pvalue, FA_90_v1.TCGA_THCA_Correlation, FA_90_v1.TCGA_THCA_Pvalue, FA_v1.TCGA_THCA_Correlation, FA_v1.TCGA_THCA_Pvalue\n",
      "\n",
      "Substudy name: THYM \n",
      "Substudy description: Thymoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_THYM_Correlation, BigGIM_70_v1.TCGA_THYM_Pvalue, BigGIM_80_v1.TCGA_THYM_Correlation, BigGIM_80_v1.TCGA_THYM_Pvalue, BigGIM_90_v1.TCGA_THYM_Correlation, BigGIM_90_v1.TCGA_THYM_Pvalue, BigGIM_v1.TCGA_THYM_Correlation, BigGIM_v1.TCGA_THYM_Pvalue, CRC_70_v1.TCGA_THYM_Correlation, CRC_70_v1.TCGA_THYM_Pvalue, CRC_80_v1.TCGA_THYM_Correlation, CRC_80_v1.TCGA_THYM_Pvalue, CRC_90_v1.TCGA_THYM_Correlation, CRC_90_v1.TCGA_THYM_Pvalue, CRC_v1.TCGA_THYM_Correlation, CRC_v1.TCGA_THYM_Pvalue, FA_70_v1.TCGA_THYM_Correlation, FA_70_v1.TCGA_THYM_Pvalue, FA_80_v1.TCGA_THYM_Correlation, FA_80_v1.TCGA_THYM_Pvalue, FA_90_v1.TCGA_THYM_Correlation, FA_90_v1.TCGA_THYM_Pvalue, FA_v1.TCGA_THYM_Correlation, FA_v1.TCGA_THYM_Pvalue\n",
      "\n",
      "Substudy name: UCEC \n",
      "Substudy description: Uterine Corpus Endometrial Carcinoma\n",
      "Table.column list: BigGIM_70_v1.TCGA_UCEC_Correlation, BigGIM_70_v1.TCGA_UCEC_Pvalue, BigGIM_80_v1.TCGA_UCEC_Correlation, BigGIM_80_v1.TCGA_UCEC_Pvalue, BigGIM_90_v1.TCGA_UCEC_Correlation, BigGIM_90_v1.TCGA_UCEC_Pvalue, BigGIM_v1.TCGA_UCEC_Correlation, BigGIM_v1.TCGA_UCEC_Pvalue, CRC_70_v1.TCGA_UCEC_Correlation, CRC_70_v1.TCGA_UCEC_Pvalue, CRC_80_v1.TCGA_UCEC_Correlation, CRC_80_v1.TCGA_UCEC_Pvalue, CRC_90_v1.TCGA_UCEC_Correlation, CRC_90_v1.TCGA_UCEC_Pvalue, CRC_v1.TCGA_UCEC_Correlation, CRC_v1.TCGA_UCEC_Pvalue, FA_70_v1.TCGA_UCEC_Correlation, FA_70_v1.TCGA_UCEC_Pvalue, FA_80_v1.TCGA_UCEC_Correlation, FA_80_v1.TCGA_UCEC_Pvalue, FA_90_v1.TCGA_UCEC_Correlation, FA_90_v1.TCGA_UCEC_Pvalue, FA_v1.TCGA_UCEC_Correlation, FA_v1.TCGA_UCEC_Pvalue\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for substudy in tcga['substudies']:\n",
    "    print(\"Substudy name: %s \" % substudy['name'])\n",
    "    print(\"Substudy description: %s\" % substudy['description'])\n",
    "    t_c_list = ', '.join(\"%s.%s\" % (column['table'][\"name\"], column['name']) for column in substudy['columns'])\n",
    "    print(\"Table.column list: %s\" % (t_c_list) )\n",
    "    print"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "### what an error looks like"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "404 Client Error: NOT FOUND for url: http://biggim.ncats.io/api/metadata/study/TCSG\n",
      "\n",
      "{\n",
      "  \"status\": \"error\", \n",
      "  \"message\": \"[TCSG] not a valid study\"\n",
      "}\n",
      "None\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    ta = get('metadata/study/%s' % ('TCSG'))\n",
    "    jprint(studies)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_tables\"></a>\n",
    "### Get all tables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[\n",
      "  {\n",
      "    \"num_rows\": 51090886, \n",
      "    \"default\": true, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\", \n",
      "    \"num_bytes\": 146013971730, \n",
      "    \"name\": \"BigGIM_70_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 14210893, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\", \n",
      "    \"num_bytes\": 41062787154, \n",
      "    \"name\": \"BigGIM_80_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 1173151, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\", \n",
      "    \"num_bytes\": 3389718425, \n",
      "    \"name\": \"BigGIM_90_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 411827180, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions.\", \n",
      "    \"num_bytes\": 931457211953, \n",
      "    \"name\": \"BigGIM_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 8604, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\", \n",
      "    \"num_bytes\": 26765474, \n",
      "    \"name\": \"CRC_70_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 2100, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\", \n",
      "    \"num_bytes\": 6536954, \n",
      "    \"name\": \"CRC_80_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 132, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\", \n",
      "    \"num_bytes\": 412721, \n",
      "    \"name\": \"CRC_90_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 27028, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020).\", \n",
      "    \"num_bytes\": 83725184, \n",
      "    \"name\": \"CRC_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 161445, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\", \n",
      "    \"num_bytes\": 482470699, \n",
      "    \"name\": \"FA_70_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 53392, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\", \n",
      "    \"num_bytes\": 160188930, \n",
      "    \"name\": \"FA_80_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 4151, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\", \n",
      "    \"num_bytes\": 12467222, \n",
      "    \"name\": \"FA_90_v1\"\n",
      "  }, \n",
      "  {\n",
      "    \"num_rows\": 587730, \n",
      "    \"default\": false, \n",
      "    \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes.\", \n",
      "    \"num_bytes\": 1519402925, \n",
      "    \"name\": \"FA_v1\"\n",
      "  }\n",
      "]\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    tables = get('/metadata/table')\n",
    "    jprint(tables)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table name: BigGIM_70_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\n",
      "\n",
      "Table name: BigGIM_80_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\n",
      "\n",
      "Table name: BigGIM_90_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\n",
      "\n",
      "Table name: BigGIM_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions.\n",
      "\n",
      "Table name: CRC_70_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\n",
      "\n",
      "Table name: CRC_80_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\n",
      "\n",
      "Table name: CRC_90_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020), and where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\n",
      "\n",
      "Table name: CRC_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where both genes in a pair are from the list of 233 core transcription factors (PMCID: PMC4772020).\n",
      "\n",
      "Table name: FA_70_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\n",
      "\n",
      "Table name: FA_80_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.8\n",
      "\n",
      "Table name: FA_90_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes, and where at least one of the TCGA or GTEx absolute correlations is higher than 0.9\n",
      "\n",
      "Table name: FA_v1\n",
      "Table description: Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of genes in a pair is one of the 20 FA genes.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for table in tables:\n",
    "    print(\"Table name: %s\" % table['name'])\n",
    "    print(\"Table description: %s\" % table['description'])\n",
    "    print"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_table\"></a>\n",
    "### Get one table\n",
    "\n",
    "The default table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Default table name: BigGIM_70_v1\n"
     ]
    }
   ],
   "source": [
    "default_table = [t for t in tables if t['default'] == True][0]['name']\n",
    "print(\"Default table name: %s\" % default_table)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"description\": \"Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\", \n",
      "  \"default\": true, \n",
      "  \"num_bytes\": 146013971730, \n",
      "  \"num_rows\": 51090886, \n",
      "  \"columns\": [\n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenocortical carcinoma cell|1 - adrenal cortex|2 - adrenal gland|3 - viscus|3 - endocrine gland|4 - whole body|4 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adrenocortical carcinoma\", \n",
      "        \"name\": \"ACC\", \n",
      "        \"cell_of_origin\": \"adrenocortical carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_ACC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenocortical carcinoma cell|1 - adrenal cortex|2 - adrenal gland|3 - viscus|3 - endocrine gland|4 - whole body|4 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adrenocortical carcinoma\", \n",
      "        \"name\": \"ACC\", \n",
      "        \"cell_of_origin\": \"adrenocortical carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_ACC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder cancer cell|1 - urinary bladder|2 - bladder|2 - urinary system|2 - urinary tract|3 - viscus|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Bladder Urothelial Carcinoma\", \n",
      "        \"name\": \"BLCA\", \n",
      "        \"cell_of_origin\": \"urinary bladder cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_BLCA_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder cancer cell|1 - urinary bladder|2 - bladder|2 - urinary system|2 - urinary tract|3 - viscus|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Bladder Urothelial Carcinoma\", \n",
      "        \"name\": \"BLCA\", \n",
      "        \"cell_of_origin\": \"urinary bladder cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_BLCA_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - breast cancer cell|1 - breast|1 - adenocarcinoma cell|2 - thorax|2 - female reproductive gland|2 - excretory gland|2 - carcinoma cell|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|3 - epithelioma cell|4 - whole body|4 - reproductive system|4 - epithelial cell|5 - animal|5 - urogenital system|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Breast invasive carcinoma\", \n",
      "        \"name\": \"BRCA\", \n",
      "        \"cell_of_origin\": \"breast cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_BRCA_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - breast cancer cell|1 - breast|1 - adenocarcinoma cell|2 - thorax|2 - female reproductive gland|2 - excretory gland|2 - carcinoma cell|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|3 - epithelioma cell|4 - whole body|4 - reproductive system|4 - epithelial cell|5 - animal|5 - urogenital system|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Breast invasive carcinoma\", \n",
      "        \"name\": \"BRCA\", \n",
      "        \"cell_of_origin\": \"breast cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_BRCA_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cervical adenocarcinoma cell|0 - cervical squamous cell carcinoma cell|1 - cervical carcinoma cell|1 - adenocarcinoma cell|1 - squamous cell carcinoma cell|2 - uterine cervix|2 - carcinoma cell|2 - squamous epithelium|3 - uterus|3 - epithelioma cell|3 - epithelium|4 - internal female genital organ|4 - epithelial cell|4 - integument|5 - female reproductive system|5 - whole body|6 - reproductive system|6 - animal|7 - urogenital system|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Cervical squamous cell carcinoma and endocervical adenocarcinoma\", \n",
      "        \"name\": \"CESC\", \n",
      "        \"cell_of_origin\": \"cervical adenocarcinoma cell,cervical squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_CESC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cervical adenocarcinoma cell|0 - cervical squamous cell carcinoma cell|1 - cervical carcinoma cell|1 - adenocarcinoma cell|1 - squamous cell carcinoma cell|2 - uterine cervix|2 - carcinoma cell|2 - squamous epithelium|3 - uterus|3 - epithelioma cell|3 - epithelium|4 - internal female genital organ|4 - epithelial cell|4 - integument|5 - female reproductive system|5 - whole body|6 - reproductive system|6 - animal|7 - urogenital system|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Cervical squamous cell carcinoma and endocervical adenocarcinoma\", \n",
      "        \"name\": \"CESC\", \n",
      "        \"cell_of_origin\": \"cervical adenocarcinoma cell,cervical squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_CESC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cholangiocarcinoma cell|1 - bile duct epithelium|1 - bile duct|2 - biliary epithelium|2 - liver|3 - epithelium|3 - digestive gland|3 - viscus|4 - integument|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Cholangiocarcinoma\", \n",
      "        \"name\": \"CHOL\", \n",
      "        \"cell_of_origin\": \"cholangiocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_CHOL_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cholangiocarcinoma cell|1 - bile duct epithelium|1 - bile duct|2 - biliary epithelium|2 - liver|3 - epithelium|3 - digestive gland|3 - viscus|4 - integument|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Cholangiocarcinoma\", \n",
      "        \"name\": \"CHOL\", \n",
      "        \"cell_of_origin\": \"cholangiocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_CHOL_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colorectal adenocarcinoma cell|1 - rectal cancer cell|1 - colonic cancer cell|1 - adenocarcinoma cell|2 - rectum|2 - gastrointestinal cancer cell|2 - colon|2 - carcinoma cell|3 - large intestine|3 - colorectum|3 - gastrointestinal tract|3 - epithelioma cell|4 - intestine|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Colon adenocarcinoma\", \n",
      "        \"name\": \"COAD\", \n",
      "        \"cell_of_origin\": \"colorectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_COAD_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colorectal adenocarcinoma cell|1 - rectal cancer cell|1 - colonic cancer cell|1 - adenocarcinoma cell|2 - rectum|2 - gastrointestinal cancer cell|2 - colon|2 - carcinoma cell|3 - large intestine|3 - colorectum|3 - gastrointestinal tract|3 - epithelioma cell|4 - intestine|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Colon adenocarcinoma\", \n",
      "        \"name\": \"COAD\", \n",
      "        \"cell_of_origin\": \"colorectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_COAD_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - diffuse large B-cell lymphoma cell|1 - B-lymphoma cell line|2 - lymphoma cell line|3 - lymphoid cell line|3 - lymphoma cell|4 - lymphoid cell|4 - blood cancer cell|4 - lymphoid tissue|5 - hematopoietic cell|5 - immune system|5 - hematopoietic system|6 - blood|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\", \n",
      "        \"name\": \"DLBC\", \n",
      "        \"cell_of_origin\": \"diffuse large B-cell lymphoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_DLBC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - diffuse large B-cell lymphoma cell|1 - B-lymphoma cell line|2 - lymphoma cell line|3 - lymphoid cell line|3 - lymphoma cell|4 - lymphoid cell|4 - blood cancer cell|4 - lymphoid tissue|5 - hematopoietic cell|5 - immune system|5 - hematopoietic system|6 - blood|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\", \n",
      "        \"name\": \"DLBC\", \n",
      "        \"cell_of_origin\": \"diffuse large B-cell lymphoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_DLBC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophageal cancer cell|1 - esophagus|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Esophageal carcinoma\", \n",
      "        \"name\": \"ESCA\", \n",
      "        \"cell_of_origin\": \"esophageal cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_ESCA_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophageal cancer cell|1 - esophagus|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Esophageal carcinoma\", \n",
      "        \"name\": \"ESCA\", \n",
      "        \"cell_of_origin\": \"esophageal cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_ESCA_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glioblastoma cell|1 - astrocytoma cell|2 - glioma cell|2 - astrocyte|3 - glia|3 - glial cell|3 - astroglial cell|3 - macroglia|3 - astroblast|4 - spinal cord|4 - brain|4 - astroglia|4 - neuroblast|5 - central nervous system|5 - head|5 - embryo|6 - nervous system|6 - whole body|6 - embryonic structure|7 - animal|7 - organism form|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Glioblastoma multiforme\", \n",
      "        \"name\": \"GBM\", \n",
      "        \"cell_of_origin\": \"glioblastoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_GBM_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glioblastoma cell|1 - astrocytoma cell|2 - glioma cell|2 - astrocyte|3 - glia|3 - glial cell|3 - astroglial cell|3 - macroglia|3 - astroblast|4 - spinal cord|4 - brain|4 - astroglia|4 - neuroblast|5 - central nervous system|5 - head|5 - embryo|6 - nervous system|6 - whole body|6 - embryonic structure|7 - animal|7 - organism form|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Glioblastoma multiforme\", \n",
      "        \"name\": \"GBM\", \n",
      "        \"cell_of_origin\": \"glioblastoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_GBM_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - head and neck cancer cell|1 - head|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Head and Neck squamous cell carcinoma\", \n",
      "        \"name\": \"HNSC\", \n",
      "        \"cell_of_origin\": \"head and neck cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_HNSC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - head and neck cancer cell|1 - head|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Head and Neck squamous cell carcinoma\", \n",
      "        \"name\": \"HNSC\", \n",
      "        \"cell_of_origin\": \"head and neck cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_HNSC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - chromophobe renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney Chromophobe\", \n",
      "        \"name\": \"KICH\", \n",
      "        \"cell_of_origin\": \"chromophobe renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KICH_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - chromophobe renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney Chromophobe\", \n",
      "        \"name\": \"KICH\", \n",
      "        \"cell_of_origin\": \"chromophobe renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KICH_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal clear cell|0 - renal cell carcinoma cell|1 - renal tubule|1 - kidney cancer cell|1 - adenocarcinoma cell|2 - nephron|2 - kidney|2 - carcinoma cell|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|3 - epithelioma cell|4 - gland|4 - whole body|4 - urogenital system|4 - epithelial cell|5 - animal|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney renal clear cell carcinoma\", \n",
      "        \"name\": \"KIRC\", \n",
      "        \"cell_of_origin\": \"renal clear cell,renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KIRC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal clear cell|0 - renal cell carcinoma cell|1 - renal tubule|1 - kidney cancer cell|1 - adenocarcinoma cell|2 - nephron|2 - kidney|2 - carcinoma cell|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|3 - epithelioma cell|4 - gland|4 - whole body|4 - urogenital system|4 - epithelial cell|5 - animal|5 - epithelium|6 - tissues, cell types and enzyme sources|6 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney renal clear cell carcinoma\", \n",
      "        \"name\": \"KIRC\", \n",
      "        \"cell_of_origin\": \"renal clear cell,renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KIRC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - papillary renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney renal papillary cell carcinoma\", \n",
      "        \"name\": \"KIRP\", \n",
      "        \"cell_of_origin\": \"papillary renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KIRP_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - papillary renal cell carcinoma cell|1 - renal cell carcinoma cell|2 - kidney cancer cell|2 - adenocarcinoma cell|3 - kidney|3 - carcinoma cell|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|4 - epithelioma cell|5 - gland|5 - whole body|5 - urogenital system|5 - epithelial cell|6 - animal|6 - epithelium|7 - tissues, cell types and enzyme sources|7 - integument\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Kidney renal papillary cell carcinoma\", \n",
      "        \"name\": \"KIRP\", \n",
      "        \"cell_of_origin\": \"papillary renal cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_KIRP_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - acute myeloid leukemia cell|1 - myeloid leukemia cell|2 - leukemia cell|3 - leukocyte|3 - blood cancer cell|4 - hematopoietic cell|5 - blood|6 - hematopoietic system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Acute Myeloid Leukemia\", \n",
      "        \"name\": \"LAML\", \n",
      "        \"cell_of_origin\": \"acute myeloid leukemia cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LAML_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - acute myeloid leukemia cell|1 - myeloid leukemia cell|2 - leukemia cell|3 - leukocyte|3 - blood cancer cell|4 - hematopoietic cell|5 - blood|6 - hematopoietic system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Acute Myeloid Leukemia\", \n",
      "        \"name\": \"LAML\", \n",
      "        \"cell_of_origin\": \"acute myeloid leukemia cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LAML_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glioma cell|1 - glia|1 - glial cell|2 - spinal cord|2 - brain|3 - central nervous system|3 - head|4 - nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Brain Lower Grade Glioma\", \n",
      "        \"name\": \"LGG\", \n",
      "        \"cell_of_origin\": \"glioma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LGG_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glioma cell|1 - glia|1 - glial cell|2 - spinal cord|2 - brain|3 - central nervous system|3 - head|4 - nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Brain Lower Grade Glioma\", \n",
      "        \"name\": \"LGG\", \n",
      "        \"cell_of_origin\": \"glioma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LGG_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver cancer cell|0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Liver hepatocellular carcinoma\", \n",
      "        \"name\": \"LIHC\", \n",
      "        \"cell_of_origin\": \"liver cancer cell,hepatocyte\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LIHC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver cancer cell|0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Liver hepatocellular carcinoma\", \n",
      "        \"name\": \"LIHC\", \n",
      "        \"cell_of_origin\": \"liver cancer cell,hepatocyte\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LIHC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung adenocarcinoma cell|1 - carcinoma cell|1 - non-small cell lung cancer cell|2 - epithelioma cell|2 - bronchogenic carcinoma cell|3 - epithelial cell|3 - lung cancer cell|4 - epithelium|4 - lung|5 - integument|5 - respiratory system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung adenocarcinoma\", \n",
      "        \"name\": \"LUAD\", \n",
      "        \"cell_of_origin\": \"lung adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LUAD_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung adenocarcinoma cell|1 - carcinoma cell|1 - non-small cell lung cancer cell|2 - epithelioma cell|2 - bronchogenic carcinoma cell|3 - epithelial cell|3 - lung cancer cell|4 - epithelium|4 - lung|5 - integument|5 - respiratory system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung adenocarcinoma\", \n",
      "        \"name\": \"LUAD\", \n",
      "        \"cell_of_origin\": \"lung adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LUAD_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung squamous cell carcinoma cell|1 - lung cancer cell|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung squamous cell carcinoma\", \n",
      "        \"name\": \"LUSC\", \n",
      "        \"cell_of_origin\": \"lung squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LUSC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung squamous cell carcinoma cell|1 - lung cancer cell|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung squamous cell carcinoma\", \n",
      "        \"name\": \"LUSC\", \n",
      "        \"cell_of_origin\": \"lung squamous cell carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_LUSC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mesothelioma cell|1 - mesothelium|2 - mesoderm|2 - squamous epithelium|2 - coelom|3 - germ layer|3 - epithelium|4 - embryo|4 - integument|5 - embryonic structure|5 - whole body|6 - organism form|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Mesothelioma\", \n",
      "        \"name\": \"MESO\", \n",
      "        \"cell_of_origin\": \"mesothelioma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_MESO_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mesothelioma cell|1 - mesothelium|2 - mesoderm|2 - squamous epithelium|2 - coelom|3 - germ layer|3 - epithelium|4 - embryo|4 - integument|5 - embryonic structure|5 - whole body|6 - organism form|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Mesothelioma\", \n",
      "        \"name\": \"MESO\", \n",
      "        \"cell_of_origin\": \"mesothelioma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_MESO_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian serous carcinoma cell|1 - ovary cancer cell|2 - ovary|3 - female reproductive gland|3 - internal female genital organ|4 - female reproductive system|4 - endocrine gland|5 - reproductive system|5 - gland|6 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Ovarian serous cystadenocarcinoma\", \n",
      "        \"name\": \"OV\", \n",
      "        \"cell_of_origin\": \"ovarian serous carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_OV_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian serous carcinoma cell|1 - ovary cancer cell|2 - ovary|3 - female reproductive gland|3 - internal female genital organ|4 - female reproductive system|4 - endocrine gland|5 - reproductive system|5 - gland|6 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Ovarian serous cystadenocarcinoma\", \n",
      "        \"name\": \"OV\", \n",
      "        \"cell_of_origin\": \"ovarian serous carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_OV_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic ductal adenocarcinoma cell|1 - pancreatic ductal carcinoma cell|1 - adenocarcinoma cell|2 - pancreatic duct|2 - carcinoma cell|3 - exocrine pancreas|3 - epithelioma cell|4 - pancreas|4 - epithelial cell|5 - digestive gland|5 - viscus|5 - epithelium|6 - endocrine gland|6 - whole body|6 - integument|7 - gland|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pancreatic adenocarcinoma\", \n",
      "        \"name\": \"PAAD\", \n",
      "        \"cell_of_origin\": \"pancreatic ductal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PAAD_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic ductal adenocarcinoma cell|1 - pancreatic ductal carcinoma cell|1 - adenocarcinoma cell|2 - pancreatic duct|2 - carcinoma cell|3 - exocrine pancreas|3 - epithelioma cell|4 - pancreas|4 - epithelial cell|5 - digestive gland|5 - viscus|5 - epithelium|6 - endocrine gland|6 - whole body|6 - integument|7 - gland|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pancreatic adenocarcinoma\", \n",
      "        \"name\": \"PAAD\", \n",
      "        \"cell_of_origin\": \"pancreatic ductal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PAAD_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pheochromocytoma cell|0 - neural crest cell|0 - neuroendocrine tumor cell|1 - adrenal gland cancer cell|1 - neural crest|1 - neuroendocrine cell|1 - carcinoid cell|2 - adrenal gland|2 - neural plate|2 - endocrine gland|2 - gastrointestinal cancer cell|2 - intestinal mucosa|3 - viscus|3 - ectoderm|3 - gland|3 - gastrointestinal tract|3 - mucosa|3 - intestine|4 - whole body|4 - germ layer|4 - alimentary canal|4 - integument|5 - animal|5 - embryo|6 - tissues, cell types and enzyme sources|6 - embryonic structure|7 - organism form\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pheochromocytoma and Paraganglioma\", \n",
      "        \"name\": \"PCPG\", \n",
      "        \"cell_of_origin\": \"pheochromocytoma cell,neural crest cell,neuroendocrine tumor cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PCPG_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pheochromocytoma cell|0 - neural crest cell|0 - neuroendocrine tumor cell|1 - adrenal gland cancer cell|1 - neural crest|1 - neuroendocrine cell|1 - carcinoid cell|2 - adrenal gland|2 - neural plate|2 - endocrine gland|2 - gastrointestinal cancer cell|2 - intestinal mucosa|3 - viscus|3 - ectoderm|3 - gland|3 - gastrointestinal tract|3 - mucosa|3 - intestine|4 - whole body|4 - germ layer|4 - alimentary canal|4 - integument|5 - animal|5 - embryo|6 - tissues, cell types and enzyme sources|6 - embryonic structure|7 - organism form\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pheochromocytoma and Paraganglioma\", \n",
      "        \"name\": \"PCPG\", \n",
      "        \"cell_of_origin\": \"pheochromocytoma cell,neural crest cell,neuroendocrine tumor cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PCPG_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate adenocarcinoma cell|1 - adenocarcinoma cell|2 - carcinoma cell|3 - epithelioma cell|4 - epithelial cell|5 - epithelium|6 - integument|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Prostate adenocarcinoma\", \n",
      "        \"name\": \"PRAD\", \n",
      "        \"cell_of_origin\": \"prostate adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PRAD_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate adenocarcinoma cell|1 - adenocarcinoma cell|2 - carcinoma cell|3 - epithelioma cell|4 - epithelial cell|5 - epithelium|6 - integument|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Prostate adenocarcinoma\", \n",
      "        \"name\": \"PRAD\", \n",
      "        \"cell_of_origin\": \"prostate adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_PRAD_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - rectal adenocarcinoma cell|1 - colorectal adenocarcinoma cell|2 - rectal cancer cell|2 - colonic cancer cell|2 - adenocarcinoma cell|3 - rectum|3 - gastrointestinal cancer cell|3 - colon|3 - carcinoma cell|4 - large intestine|4 - colorectum|4 - gastrointestinal tract|4 - epithelioma cell|5 - intestine|5 - alimentary canal|5 - epithelial cell|6 - viscus|6 - epithelium|7 - whole body|7 - integument|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Rectum adenocarcinoma\", \n",
      "        \"name\": \"READ\", \n",
      "        \"cell_of_origin\": \"rectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_READ_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - rectal adenocarcinoma cell|1 - colorectal adenocarcinoma cell|2 - rectal cancer cell|2 - colonic cancer cell|2 - adenocarcinoma cell|3 - rectum|3 - gastrointestinal cancer cell|3 - colon|3 - carcinoma cell|4 - large intestine|4 - colorectum|4 - gastrointestinal tract|4 - epithelioma cell|5 - intestine|5 - alimentary canal|5 - epithelial cell|6 - viscus|6 - epithelium|7 - whole body|7 - integument|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Rectum adenocarcinoma\", \n",
      "        \"name\": \"READ\", \n",
      "        \"cell_of_origin\": \"rectal adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_READ_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - sarcoma cell|1 - cartilage|1 - skeletal muscle cancer cell|1 - bone cancer cell|2 - connective tissue|2 - skeletal system|2 - skeletal muscle cell|2 - bone|3 - whole body|3 - skeletal muscle|4 - animal|4 - muscle|4 - vertebrate muscular system|5 - tissues, cell types and enzyme sources|5 - muscular system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Sarcoma\", \n",
      "        \"name\": \"SARC\", \n",
      "        \"cell_of_origin\": \"sarcoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_SARC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - sarcoma cell|1 - cartilage|1 - skeletal muscle cancer cell|1 - bone cancer cell|2 - connective tissue|2 - skeletal system|2 - skeletal muscle cell|2 - bone|3 - whole body|3 - skeletal muscle|4 - animal|4 - muscle|4 - vertebrate muscular system|5 - tissues, cell types and enzyme sources|5 - muscular system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Sarcoma\", \n",
      "        \"name\": \"SARC\", \n",
      "        \"cell_of_origin\": \"sarcoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_SARC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin cancer cell|0 - melanoma cell|1 - skin|1 - melanocyte|2 - integument|2 - epidermal cell|3 - whole body|3 - epidermis|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Skin Cutaneous Melanoma\", \n",
      "        \"name\": \"SKCM\", \n",
      "        \"cell_of_origin\": \"skin cancer cell,melanoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_SKCM_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin cancer cell|0 - melanoma cell|1 - skin|1 - melanocyte|2 - integument|2 - epidermal cell|3 - whole body|3 - epidermis|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Skin Cutaneous Melanoma\", \n",
      "        \"name\": \"SKCM\", \n",
      "        \"cell_of_origin\": \"skin cancer cell,melanoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_SKCM_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - gastric adenocarcinoma cell|1 - gastric cancer cell|1 - adenocarcinoma cell|2 - stomach|2 - carcinoma cell|3 - gastrointestinal tract|3 - epithelioma cell|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Stomach adenocarcinoma\", \n",
      "        \"name\": \"STAD\", \n",
      "        \"cell_of_origin\": \"gastric adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_STAD_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - gastric adenocarcinoma cell|1 - gastric cancer cell|1 - adenocarcinoma cell|2 - stomach|2 - carcinoma cell|3 - gastrointestinal tract|3 - epithelioma cell|4 - alimentary canal|4 - epithelial cell|5 - viscus|5 - epithelium|6 - whole body|6 - integument|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Stomach adenocarcinoma\", \n",
      "        \"name\": \"STAD\", \n",
      "        \"cell_of_origin\": \"gastric adenocarcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_STAD_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testicular cancer cell|1 - testis|2 - internal male genital organ|2 - gonad|2 - male reproductive gland|3 - male reproductive system|3 - female reproductive gland|3 - endocrine gland|4 - reproductive system|4 - female reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Testicular Germ Cell Tumors\", \n",
      "        \"name\": \"TGCT\", \n",
      "        \"cell_of_origin\": \"testicular cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_TGCT_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testicular cancer cell|1 - testis|2 - internal male genital organ|2 - gonad|2 - male reproductive gland|3 - male reproductive system|3 - female reproductive gland|3 - endocrine gland|4 - reproductive system|4 - female reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Testicular Germ Cell Tumors\", \n",
      "        \"name\": \"TGCT\", \n",
      "        \"cell_of_origin\": \"testicular cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_TGCT_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid cancer cell|1 - thyroid gland|2 - throat|2 - endocrine gland|3 - neck|3 - gland|4 - trunk|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thyroid carcinoma\", \n",
      "        \"name\": \"THCA\", \n",
      "        \"cell_of_origin\": \"thyroid cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_THCA_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid cancer cell|1 - thyroid gland|2 - throat|2 - endocrine gland|3 - neck|3 - gland|4 - trunk|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thyroid carcinoma\", \n",
      "        \"name\": \"THCA\", \n",
      "        \"cell_of_origin\": \"thyroid cancer cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_THCA_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thymoma cell|1 - thymic cancer cell|2 - thymus|3 - immune system|3 - thorax|3 - endocrine gland|3 - lymphoid tissue|4 - whole body|4 - trunk|4 - gland|4 - hematopoietic system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thymoma\", \n",
      "        \"name\": \"THYM\", \n",
      "        \"cell_of_origin\": \"thymoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_THYM_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thymoma cell|1 - thymic cancer cell|2 - thymus|3 - immune system|3 - thorax|3 - endocrine gland|3 - lymphoid tissue|4 - whole body|4 - trunk|4 - gland|4 - hematopoietic system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thymoma\", \n",
      "        \"name\": \"THYM\", \n",
      "        \"cell_of_origin\": \"thymoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_THYM_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - endometrioid carcinoma cell|1 - uterine endometrium|1 - ovary cancer cell|2 - mucosa|2 - uterus|2 - ovary|3 - integument|3 - internal female genital organ|3 - female reproductive gland|4 - whole body|4 - female reproductive system|4 - endocrine gland|5 - animal|5 - reproductive system|5 - gland|6 - tissues, cell types and enzyme sources|6 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uterine Corpus Endometrial Carcinoma\", \n",
      "        \"name\": \"UCEC\", \n",
      "        \"cell_of_origin\": \"endometrioid carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UCEC_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - endometrioid carcinoma cell|1 - uterine endometrium|1 - ovary cancer cell|2 - mucosa|2 - uterus|2 - ovary|3 - integument|3 - internal female genital organ|3 - female reproductive gland|4 - whole body|4 - female reproductive system|4 - endocrine gland|5 - animal|5 - reproductive system|5 - gland|6 - tissues, cell types and enzyme sources|6 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uterine Corpus Endometrial Carcinoma\", \n",
      "        \"name\": \"UCEC\", \n",
      "        \"cell_of_origin\": \"endometrioid carcinoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UCEC_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine sarcoma cell|1 - sarcoma cell|1 - uterine cancer cell|2 - cartilage|2 - skeletal muscle cancer cell|2 - bone cancer cell|2 - uterus|3 - connective tissue|3 - skeletal system|3 - skeletal muscle cell|3 - bone|3 - internal female genital organ|4 - whole body|4 - skeletal muscle|4 - female reproductive system|5 - animal|5 - muscle|5 - vertebrate muscular system|5 - reproductive system|6 - tissues, cell types and enzyme sources|6 - muscular system|6 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uterine Carcinosarcoma\", \n",
      "        \"name\": \"UCS\", \n",
      "        \"cell_of_origin\": \"uterine sarcoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UCS_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine sarcoma cell|1 - sarcoma cell|1 - uterine cancer cell|2 - cartilage|2 - skeletal muscle cancer cell|2 - bone cancer cell|2 - uterus|3 - connective tissue|3 - skeletal system|3 - skeletal muscle cell|3 - bone|3 - internal female genital organ|4 - whole body|4 - skeletal muscle|4 - female reproductive system|5 - animal|5 - muscle|5 - vertebrate muscular system|5 - reproductive system|6 - tissues, cell types and enzyme sources|6 - muscular system|6 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uterine Carcinosarcoma\", \n",
      "        \"name\": \"UCS\", \n",
      "        \"cell_of_origin\": \"uterine sarcoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UCS_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uveal melanoma cell|1 - melanoma cell|2 - skin cancer cell|2 - melanocyte|3 - skin|3 - epidermal cell|4 - integument|4 - epidermis|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uveal Melanoma\", \n",
      "        \"name\": \"UVM\", \n",
      "        \"cell_of_origin\": \"uveal melanoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UVM_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uveal melanoma cell|1 - melanoma cell|2 - skin cancer cell|2 - melanocyte|3 - skin|3 - epidermal cell|4 - integument|4 - epidermis|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"TCGA\", \n",
      "          \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Uveal Melanoma\", \n",
      "        \"name\": \"UVM\", \n",
      "        \"cell_of_origin\": \"uveal melanoma cell\"\n",
      "      }, \n",
      "      \"name\": \"TCGA_UVM_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adipose_Tissue\", \n",
      "        \"name\": \"Adipose_Tissue\", \n",
      "        \"cell_of_origin\": \"Adipose Tissue\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Adipose_Tissue_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adipose_Tissue\", \n",
      "        \"name\": \"Adipose_Tissue\", \n",
      "        \"cell_of_origin\": \"Adipose Tissue\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Adipose_Tissue_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adrenal_Gland\", \n",
      "        \"name\": \"Adrenal_Gland\", \n",
      "        \"cell_of_origin\": \"Adrenal Gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Adrenal_Gland_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Adrenal_Gland\", \n",
      "        \"name\": \"Adrenal_Gland\", \n",
      "        \"cell_of_origin\": \"Adrenal Gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Adrenal_Gland_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Blood\", \n",
      "        \"name\": \"Blood\", \n",
      "        \"cell_of_origin\": \"Blood\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Blood_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Blood\", \n",
      "        \"name\": \"Blood\", \n",
      "        \"cell_of_origin\": \"Blood\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Blood_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Blood_Vessel\", \n",
      "        \"name\": \"Blood_Vessel\", \n",
      "        \"cell_of_origin\": \"Blood Vessel\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Blood_Vessel_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Blood_Vessel\", \n",
      "        \"name\": \"Blood_Vessel\", \n",
      "        \"cell_of_origin\": \"Blood Vessel\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Blood_Vessel_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Brain\", \n",
      "        \"name\": \"Brain\", \n",
      "        \"cell_of_origin\": \"Brain\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Brain_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Brain\", \n",
      "        \"name\": \"Brain\", \n",
      "        \"cell_of_origin\": \"Brain\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Brain_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - breast|1 - thorax|1 - female reproductive gland|1 - excretory gland|2 - trunk|2 - female reproductive system|2 - endocrine gland|2 - gland|3 - whole body|3 - reproductive system|4 - animal|4 - urogenital system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Breast\", \n",
      "        \"name\": \"Breast\", \n",
      "        \"cell_of_origin\": \"Breast\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Breast_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - breast|1 - thorax|1 - female reproductive gland|1 - excretory gland|2 - trunk|2 - female reproductive system|2 - endocrine gland|2 - gland|3 - whole body|3 - reproductive system|4 - animal|4 - urogenital system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Breast\", \n",
      "        \"name\": \"Breast\", \n",
      "        \"cell_of_origin\": \"Breast\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Breast_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Colon\", \n",
      "        \"name\": \"Colon\", \n",
      "        \"cell_of_origin\": \"Colon\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Colon_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Colon\", \n",
      "        \"name\": \"Colon\", \n",
      "        \"cell_of_origin\": \"Colon\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Colon_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Esophagus\", \n",
      "        \"name\": \"Esophagus\", \n",
      "        \"cell_of_origin\": \"Esophagus\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Esophagus_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Esophagus\", \n",
      "        \"name\": \"Esophagus\", \n",
      "        \"cell_of_origin\": \"Esophagus\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Esophagus_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Heart\", \n",
      "        \"name\": \"Heart\", \n",
      "        \"cell_of_origin\": \"Heart\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Heart_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Heart\", \n",
      "        \"name\": \"Heart\", \n",
      "        \"cell_of_origin\": \"Heart\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Heart_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Liver\", \n",
      "        \"name\": \"Liver\", \n",
      "        \"cell_of_origin\": \"Liver\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Liver_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Liver\", \n",
      "        \"name\": \"Liver\", \n",
      "        \"cell_of_origin\": \"Liver\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Liver_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung\", \n",
      "        \"name\": \"Lung\", \n",
      "        \"cell_of_origin\": \"Lung\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Lung_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Lung\", \n",
      "        \"name\": \"Lung\", \n",
      "        \"cell_of_origin\": \"Lung\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Lung_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Muscle\", \n",
      "        \"name\": \"Muscle\", \n",
      "        \"cell_of_origin\": \"Muscle\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Muscle_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Muscle\", \n",
      "        \"name\": \"Muscle\", \n",
      "        \"cell_of_origin\": \"Muscle\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Muscle_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nerve|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Nerve\", \n",
      "        \"name\": \"Nerve\", \n",
      "        \"cell_of_origin\": \"Nerve\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Nerve_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nerve|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Nerve\", \n",
      "        \"name\": \"Nerve\", \n",
      "        \"cell_of_origin\": \"Nerve\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Nerve_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pancreas\", \n",
      "        \"name\": \"Pancreas\", \n",
      "        \"cell_of_origin\": \"Pancreas\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Pancreas_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pancreas\", \n",
      "        \"name\": \"Pancreas\", \n",
      "        \"cell_of_origin\": \"Pancreas\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Pancreas_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pituitary\", \n",
      "        \"name\": \"Pituitary\", \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Pituitary_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Pituitary\", \n",
      "        \"name\": \"Pituitary\", \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Pituitary_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Prostate\", \n",
      "        \"name\": \"Prostate\", \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Prostate_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Prostate\", \n",
      "        \"name\": \"Prostate\", \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Prostate_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Skin\", \n",
      "        \"name\": \"Skin\", \n",
      "        \"cell_of_origin\": \"Skin\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Skin_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Skin\", \n",
      "        \"name\": \"Skin\", \n",
      "        \"cell_of_origin\": \"Skin\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Skin_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Spleen\", \n",
      "        \"name\": \"Spleen\", \n",
      "        \"cell_of_origin\": \"Spleen\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Spleen_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Spleen\", \n",
      "        \"name\": \"Spleen\", \n",
      "        \"cell_of_origin\": \"Spleen\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Spleen_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Stomach\", \n",
      "        \"name\": \"Stomach\", \n",
      "        \"cell_of_origin\": \"Stomach\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Stomach_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Stomach\", \n",
      "        \"name\": \"Stomach\", \n",
      "        \"cell_of_origin\": \"Stomach\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Stomach_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Testis\", \n",
      "        \"name\": \"Testis\", \n",
      "        \"cell_of_origin\": \"Testis\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Testis_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Testis\", \n",
      "        \"name\": \"Testis\", \n",
      "        \"cell_of_origin\": \"Testis\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Testis_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thyroid\", \n",
      "        \"name\": \"Thyroid\", \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Thyroid_Correlation\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"P-value (-log10)\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GTEx\", \n",
      "          \"description\": \"Genotype-Tissue Expression (GTEx) program\\n\\n        Genotype-Tissue Expression (GTEx) program is providing valuable insights into the mechanisms of gene regulation by studying human gene expression and regulation in multiple tissues from health individuals; exploring disease-related perturbations in a variety of human diseases; and examining sexual dimorphisms in gene expression and regulation in multiple tissues. Genetic variation between individuals underlying the many differences in gene expression  will be examined for correlation with differences in gene expression level to identify regions of the genome that influence whether and how much a gene is expressed. Identifying unique genomic variations associated with gene expression is expected to stimulate research towards understanding the genetic basis of complex diseases. \\n        \\n        https://www.gtexportal.org/home/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Thyroid\", \n",
      "        \"name\": \"Thyroid\", \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      \"name\": \"GTEx_Thyroid_Pvalue\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adipose_tissue\", \n",
      "        \"name\": \"adipose_tissue\", \n",
      "        \"cell_of_origin\": \"adipose tissue\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adipose_tissue_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adipose tissue|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adipose_tissue\", \n",
      "        \"name\": \"adipose_tissue\", \n",
      "        \"cell_of_origin\": \"adipose tissue\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adipose_tissue_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal cortex|1 - adrenal gland|2 - viscus|2 - endocrine gland|3 - whole body|3 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adrenal_cortex\", \n",
      "        \"name\": \"adrenal_cortex\", \n",
      "        \"cell_of_origin\": \"adrenal cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adrenal_cortex_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal cortex|1 - adrenal gland|2 - viscus|2 - endocrine gland|3 - whole body|3 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adrenal_cortex\", \n",
      "        \"name\": \"adrenal_cortex\", \n",
      "        \"cell_of_origin\": \"adrenal cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adrenal_cortex_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adrenal_gland\", \n",
      "        \"name\": \"adrenal_gland\", \n",
      "        \"cell_of_origin\": \"adrenal gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adrenal_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - adrenal gland|1 - viscus|1 - endocrine gland|2 - whole body|2 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"adrenal_gland\", \n",
      "        \"name\": \"adrenal_gland\", \n",
      "        \"cell_of_origin\": \"adrenal gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_adrenal_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - animal|1 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"all_tissues\", \n",
      "        \"name\": \"all_tissues\", \n",
      "        \"cell_of_origin\": \"animal\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_all_tissues_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - animal|1 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"all_tissues\", \n",
      "        \"name\": \"all_tissues\", \n",
      "        \"cell_of_origin\": \"animal\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_all_tissues_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - amygdala|1 - temporal lobe|1 - avian pallium|1 - basal ganglion|1 - limbic system|2 - cerebral lobe|2 - telencephalon|2 - ganglion|2 - cerebral cortex|2 - brain|3 - forebrain|3 - nervous system|3 - cerebral hemisphere|3 - head|3 - central nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"amygdala\", \n",
      "        \"name\": \"amygdala\", \n",
      "        \"cell_of_origin\": \"amygdala\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_amygdala_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - amygdala|1 - temporal lobe|1 - avian pallium|1 - basal ganglion|1 - limbic system|2 - cerebral lobe|2 - telencephalon|2 - ganglion|2 - cerebral cortex|2 - brain|3 - forebrain|3 - nervous system|3 - cerebral hemisphere|3 - head|3 - central nervous system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"amygdala\", \n",
      "        \"name\": \"amygdala\", \n",
      "        \"cell_of_origin\": \"amygdala\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_amygdala_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - aorta|1 - artery|2 - blood vessel|3 - vascular system|4 - cardiovascular system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"aorta\", \n",
      "        \"name\": \"aorta\", \n",
      "        \"cell_of_origin\": \"aorta\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_aorta_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - aorta|1 - artery|2 - blood vessel|3 - vascular system|4 - cardiovascular system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"aorta\", \n",
      "        \"name\": \"aorta\", \n",
      "        \"cell_of_origin\": \"aorta\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_aorta_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - artery|1 - blood vessel|2 - vascular system|3 - cardiovascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"artery\", \n",
      "        \"name\": \"artery\", \n",
      "        \"cell_of_origin\": \"artery\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_artery_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - artery|1 - blood vessel|2 - vascular system|3 - cardiovascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"artery\", \n",
      "        \"name\": \"artery\", \n",
      "        \"cell_of_origin\": \"artery\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_artery_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - astrocyte|1 - astroglial cell|1 - macroglia|1 - astroblast|2 - astroglia|2 - glial cell|2 - glia|2 - neuroblast|3 - spinal cord|3 - brain|3 - embryo|4 - central nervous system|4 - head|4 - embryonic structure|5 - nervous system|5 - whole body|5 - organism form|6 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"astrocyte\", \n",
      "        \"name\": \"astrocyte\", \n",
      "        \"cell_of_origin\": \"astrocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_astrocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - astrocyte|1 - astroglial cell|1 - macroglia|1 - astroblast|2 - astroglia|2 - glial cell|2 - glia|2 - neuroblast|3 - spinal cord|3 - brain|3 - embryo|4 - central nervous system|4 - head|4 - embryonic structure|5 - nervous system|5 - whole body|5 - organism form|6 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"astrocyte\", \n",
      "        \"name\": \"astrocyte\", \n",
      "        \"cell_of_origin\": \"astrocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_astrocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - B-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"b_lymphocyte\", \n",
      "        \"name\": \"b_lymphocyte\", \n",
      "        \"cell_of_origin\": \"B-lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - B-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"b_lymphocyte\", \n",
      "        \"name\": \"b_lymphocyte\", \n",
      "        \"cell_of_origin\": \"B-lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - basal ganglion|1 - ganglion|1 - cerebral cortex|2 - nervous system|2 - cerebral hemisphere|3 - whole body|3 - telencephalon|4 - animal|4 - forebrain|5 - tissues, cell types and enzyme sources|5 - brain|6 - head|6 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"basal_ganglion\", \n",
      "        \"name\": \"basal_ganglion\", \n",
      "        \"cell_of_origin\": \"basal ganglion\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_basal_ganglion_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - basal ganglion|1 - ganglion|1 - cerebral cortex|2 - nervous system|2 - cerebral hemisphere|3 - whole body|3 - telencephalon|4 - animal|4 - forebrain|5 - tissues, cell types and enzyme sources|5 - brain|6 - head|6 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"basal_ganglion\", \n",
      "        \"name\": \"basal_ganglion\", \n",
      "        \"cell_of_origin\": \"basal ganglion\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_basal_ganglion_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - basophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"basophil\", \n",
      "        \"name\": \"basophil\", \n",
      "        \"cell_of_origin\": \"basophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_basophil_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - basophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"basophil\", \n",
      "        \"name\": \"basophil\", \n",
      "        \"cell_of_origin\": \"basophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_basophil_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood\", \n",
      "        \"name\": \"blood\", \n",
      "        \"cell_of_origin\": \"blood\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood\", \n",
      "        \"name\": \"blood\", \n",
      "        \"cell_of_origin\": \"blood\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood plasma|1 - blood|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_plasma\", \n",
      "        \"name\": \"blood_plasma\", \n",
      "        \"cell_of_origin\": \"blood plasma\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_plasma_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood plasma|1 - blood|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_plasma\", \n",
      "        \"name\": \"blood_plasma\", \n",
      "        \"cell_of_origin\": \"blood plasma\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_plasma_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood platelet|1 - megakaryocyte|1 - blood plasma|2 - megakaryoblast|2 - marrow cell|2 - blood|3 - bone marrow|3 - bone marrow cell|3 - hematopoietic cell|3 - myeloid progenitor cell|3 - hematopoietic system|4 - skeletal system|4 - hematopoietic stem cell|4 - whole body|5 - blast cell|5 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_platelet\", \n",
      "        \"name\": \"blood_platelet\", \n",
      "        \"cell_of_origin\": \"blood platelet\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_platelet_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood platelet|1 - megakaryocyte|1 - blood plasma|2 - megakaryoblast|2 - marrow cell|2 - blood|3 - bone marrow|3 - bone marrow cell|3 - hematopoietic cell|3 - myeloid progenitor cell|3 - hematopoietic system|4 - skeletal system|4 - hematopoietic stem cell|4 - whole body|5 - blast cell|5 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_platelet\", \n",
      "        \"name\": \"blood_platelet\", \n",
      "        \"cell_of_origin\": \"blood platelet\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_platelet_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_vessel\", \n",
      "        \"name\": \"blood_vessel\", \n",
      "        \"cell_of_origin\": \"blood vessel\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_vessel_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - blood vessel|1 - vascular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"blood_vessel\", \n",
      "        \"name\": \"blood_vessel\", \n",
      "        \"cell_of_origin\": \"blood vessel\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_blood_vessel_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bone|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bone\", \n",
      "        \"name\": \"bone\", \n",
      "        \"cell_of_origin\": \"bone\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bone_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bone|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bone\", \n",
      "        \"name\": \"bone\", \n",
      "        \"cell_of_origin\": \"bone\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bone_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bone marrow|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bone_marrow\", \n",
      "        \"name\": \"bone_marrow\", \n",
      "        \"cell_of_origin\": \"bone marrow\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bone_marrow_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bone marrow|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bone_marrow\", \n",
      "        \"name\": \"bone_marrow\", \n",
      "        \"cell_of_origin\": \"bone marrow\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bone_marrow_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"brain\", \n",
      "        \"name\": \"brain\", \n",
      "        \"cell_of_origin\": \"brain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_brain_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - brain|1 - head|1 - central nervous system|2 - whole body|2 - nervous system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"brain\", \n",
      "        \"name\": \"brain\", \n",
      "        \"cell_of_origin\": \"brain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_brain_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bronchial epithelial cell|1 - bronchial epithelium|2 - respiratory epithelium|2 - bronchus|3 - bronchoalveolar system|3 - epithelium|4 - lung|4 - integument|5 - respiratory system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bronchial_epithelial_cell\", \n",
      "        \"name\": \"bronchial_epithelial_cell\", \n",
      "        \"cell_of_origin\": \"bronchial epithelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bronchial_epithelial_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bronchial epithelial cell|1 - bronchial epithelium|2 - respiratory epithelium|2 - bronchus|3 - bronchoalveolar system|3 - epithelium|4 - lung|4 - integument|5 - respiratory system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bronchial_epithelial_cell\", \n",
      "        \"name\": \"bronchial_epithelial_cell\", \n",
      "        \"cell_of_origin\": \"bronchial epithelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bronchial_epithelial_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bronchus|1 - bronchoalveolar system|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bronchus\", \n",
      "        \"name\": \"bronchus\", \n",
      "        \"cell_of_origin\": \"bronchus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bronchus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - bronchus|1 - bronchoalveolar system|2 - lung|3 - respiratory system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"bronchus\", \n",
      "        \"name\": \"bronchus\", \n",
      "        \"cell_of_origin\": \"bronchus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_bronchus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cardiac muscle|1 - muscle|1 - vertebrate muscular system|1 - heart|2 - muscular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cardiac_muscle\", \n",
      "        \"name\": \"cardiac_muscle\", \n",
      "        \"cell_of_origin\": \"cardiac muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cardiac_muscle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cardiac muscle|1 - muscle|1 - vertebrate muscular system|1 - heart|2 - muscular system|2 - cardiovascular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cardiac_muscle\", \n",
      "        \"name\": \"cardiac_muscle\", \n",
      "        \"cell_of_origin\": \"cardiac muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cardiac_muscle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cartilage|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cartilage\", \n",
      "        \"name\": \"cartilage\", \n",
      "        \"cell_of_origin\": \"cartilage\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cartilage_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cartilage|1 - connective tissue|1 - skeletal system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cartilage\", \n",
      "        \"name\": \"cartilage\", \n",
      "        \"cell_of_origin\": \"cartilage\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cartilage_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - caudate nucleus|1 - corpus striatum|1 - neostriatum|2 - basal ganglion|3 - ganglion|3 - cerebral cortex|4 - nervous system|4 - cerebral hemisphere|5 - whole body|5 - telencephalon|6 - animal|6 - forebrain|7 - tissues, cell types and enzyme sources|7 - brain|8 - head|8 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"caudate_nucleus\", \n",
      "        \"name\": \"caudate_nucleus\", \n",
      "        \"cell_of_origin\": \"caudate nucleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_caudate_nucleus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - caudate nucleus|1 - corpus striatum|1 - neostriatum|2 - basal ganglion|3 - ganglion|3 - cerebral cortex|4 - nervous system|4 - cerebral hemisphere|5 - whole body|5 - telencephalon|6 - animal|6 - forebrain|7 - tissues, cell types and enzyme sources|7 - brain|8 - head|8 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"caudate_nucleus\", \n",
      "        \"name\": \"caudate_nucleus\", \n",
      "        \"cell_of_origin\": \"caudate nucleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_caudate_nucleus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - caudate putamen|1 - nucleus lentiformis|1 - neostriatum|2 - corpus striatum|3 - basal ganglion|4 - ganglion|4 - cerebral cortex|5 - nervous system|5 - cerebral hemisphere|6 - whole body|6 - telencephalon|7 - animal|7 - forebrain|8 - tissues, cell types and enzyme sources|8 - brain|9 - head|9 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"caudate_putamen\", \n",
      "        \"name\": \"caudate_putamen\", \n",
      "        \"cell_of_origin\": \"caudate putamen\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_caudate_putamen_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - caudate putamen|1 - nucleus lentiformis|1 - neostriatum|2 - corpus striatum|3 - basal ganglion|4 - ganglion|4 - cerebral cortex|5 - nervous system|5 - cerebral hemisphere|6 - whole body|6 - telencephalon|7 - animal|7 - forebrain|8 - tissues, cell types and enzyme sources|8 - brain|9 - head|9 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"caudate_putamen\", \n",
      "        \"name\": \"caudate_putamen\", \n",
      "        \"cell_of_origin\": \"caudate putamen\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_caudate_putamen_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cecum|1 - large intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cecum\", \n",
      "        \"name\": \"cecum\", \n",
      "        \"cell_of_origin\": \"cecum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cecum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cecum|1 - large intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cecum\", \n",
      "        \"name\": \"cecum\", \n",
      "        \"cell_of_origin\": \"cecum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cecum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - central nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"central_nervous_system\", \n",
      "        \"name\": \"central_nervous_system\", \n",
      "        \"cell_of_origin\": \"central nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_central_nervous_system_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - central nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"central_nervous_system\", \n",
      "        \"name\": \"central_nervous_system\", \n",
      "        \"cell_of_origin\": \"central nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_central_nervous_system_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellar cortex|1 - cerebellum|2 - metencephalon|3 - hindbrain|4 - brain stem|5 - brain|6 - head|6 - central nervous system|7 - whole body|7 - nervous system|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebellar_cortex\", \n",
      "        \"name\": \"cerebellar_cortex\", \n",
      "        \"cell_of_origin\": \"cerebellar cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebellar_cortex_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellar cortex|1 - cerebellum|2 - metencephalon|3 - hindbrain|4 - brain stem|5 - brain|6 - head|6 - central nervous system|7 - whole body|7 - nervous system|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebellar_cortex\", \n",
      "        \"name\": \"cerebellar_cortex\", \n",
      "        \"cell_of_origin\": \"cerebellar cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebellar_cortex_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellum|1 - metencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebellum\", \n",
      "        \"name\": \"cerebellum\", \n",
      "        \"cell_of_origin\": \"cerebellum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebellum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebellum|1 - metencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebellum\", \n",
      "        \"name\": \"cerebellum\", \n",
      "        \"cell_of_origin\": \"cerebellum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebellum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebral cortex|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebral_cortex\", \n",
      "        \"name\": \"cerebral_cortex\", \n",
      "        \"cell_of_origin\": \"cerebral cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebral_cortex_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cerebral cortex|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cerebral_cortex\", \n",
      "        \"name\": \"cerebral_cortex\", \n",
      "        \"cell_of_origin\": \"cerebral cortex\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cerebral_cortex_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - chondrocyte|1 - cartilage|2 - connective tissue|2 - skeletal system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"chondrocyte\", \n",
      "        \"name\": \"chondrocyte\", \n",
      "        \"cell_of_origin\": \"chondrocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_chondrocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - chondrocyte|1 - cartilage|2 - connective tissue|2 - skeletal system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"chondrocyte\", \n",
      "        \"name\": \"chondrocyte\", \n",
      "        \"cell_of_origin\": \"chondrocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_chondrocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - choroid|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"choroid\", \n",
      "        \"name\": \"choroid\", \n",
      "        \"cell_of_origin\": \"choroid\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_choroid_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - choroid|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"choroid\", \n",
      "        \"name\": \"choroid\", \n",
      "        \"cell_of_origin\": \"choroid\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_choroid_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cochlea|1 - vestibular system|2 - vestibular labyrinth|3 - inner ear|4 - ear|5 - head|5 - sense organ|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cochlea\", \n",
      "        \"name\": \"cochlea\", \n",
      "        \"cell_of_origin\": \"cochlea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cochlea_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cochlea|1 - vestibular system|2 - vestibular labyrinth|3 - inner ear|4 - ear|5 - head|5 - sense organ|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cochlea\", \n",
      "        \"name\": \"cochlea\", \n",
      "        \"cell_of_origin\": \"cochlea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cochlea_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"colon\", \n",
      "        \"name\": \"colon\", \n",
      "        \"cell_of_origin\": \"colon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_colon_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - colon|1 - large intestine|1 - colorectum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"colon\", \n",
      "        \"name\": \"colon\", \n",
      "        \"cell_of_origin\": \"colon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_colon_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cornea|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cornea\", \n",
      "        \"name\": \"cornea\", \n",
      "        \"cell_of_origin\": \"cornea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cornea_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - cornea|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"cornea\", \n",
      "        \"name\": \"cornea\", \n",
      "        \"cell_of_origin\": \"cornea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_cornea_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus callosum|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_callosum\", \n",
      "        \"name\": \"corpus_callosum\", \n",
      "        \"cell_of_origin\": \"corpus callosum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_callosum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus callosum|1 - cerebral hemisphere|2 - telencephalon|3 - forebrain|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_callosum\", \n",
      "        \"name\": \"corpus_callosum\", \n",
      "        \"cell_of_origin\": \"corpus callosum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_callosum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus luteum|1 - mature ovarian follicle|2 - ovarian follicle|3 - ovary|4 - female reproductive gland|4 - internal female genital organ|5 - female reproductive system|5 - endocrine gland|6 - reproductive system|6 - gland|7 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_luteum\", \n",
      "        \"name\": \"corpus_luteum\", \n",
      "        \"cell_of_origin\": \"corpus luteum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_luteum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus luteum|1 - mature ovarian follicle|2 - ovarian follicle|3 - ovary|4 - female reproductive gland|4 - internal female genital organ|5 - female reproductive system|5 - endocrine gland|6 - reproductive system|6 - gland|7 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_luteum\", \n",
      "        \"name\": \"corpus_luteum\", \n",
      "        \"cell_of_origin\": \"corpus luteum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_luteum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus striatum|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_striatum\", \n",
      "        \"name\": \"corpus_striatum\", \n",
      "        \"cell_of_origin\": \"corpus striatum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_striatum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - corpus striatum|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"corpus_striatum\", \n",
      "        \"name\": \"corpus_striatum\", \n",
      "        \"cell_of_origin\": \"corpus striatum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_corpus_striatum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - culture condition:CD8+ cell|1 - culture condition:antigen-presenting cell|2 - culture condition|3 - cell culture|4 - other source|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"culture_condition_cd8_cell\", \n",
      "        \"name\": \"culture_condition_cd8_cell\", \n",
      "        \"cell_of_origin\": \"culture condition:CD8+ cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_culture_condition_cd8_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - culture condition:CD8+ cell|1 - culture condition:antigen-presenting cell|2 - culture condition|3 - cell culture|4 - other source|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"culture_condition_cd8_cell\", \n",
      "        \"name\": \"culture_condition_cd8_cell\", \n",
      "        \"cell_of_origin\": \"culture condition:CD8+ cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_culture_condition_cd8_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - dendritic cell|1 - lymph node|1 - spleen|1 - peripheral blood|2 - immune system|2 - endocrine gland|2 - lymphoid tissue|2 - viscus|2 - blood|3 - whole body|3 - gland|3 - hematopoietic system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"dendritic_cell\", \n",
      "        \"name\": \"dendritic_cell\", \n",
      "        \"cell_of_origin\": \"dendritic cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_dendritic_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - dendritic cell|1 - lymph node|1 - spleen|1 - peripheral blood|2 - immune system|2 - endocrine gland|2 - lymphoid tissue|2 - viscus|2 - blood|3 - whole body|3 - gland|3 - hematopoietic system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"dendritic_cell\", \n",
      "        \"name\": \"dendritic_cell\", \n",
      "        \"cell_of_origin\": \"dendritic cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - dentate gyrus|1 - hippocampus|2 - temporal lobe|2 - limbic system|3 - cerebral lobe|3 - brain|4 - cerebral cortex|4 - head|4 - central nervous system|5 - cerebral hemisphere|5 - whole body|5 - nervous system|6 - telencephalon|6 - animal|7 - forebrain|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"dentate_gyrus\", \n",
      "        \"name\": \"dentate_gyrus\", \n",
      "        \"cell_of_origin\": \"dentate gyrus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_dentate_gyrus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - dentate gyrus|1 - hippocampus|2 - temporal lobe|2 - limbic system|3 - cerebral lobe|3 - brain|4 - cerebral cortex|4 - head|4 - central nervous system|5 - cerebral hemisphere|5 - whole body|5 - nervous system|6 - telencephalon|6 - animal|7 - forebrain|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"dentate_gyrus\", \n",
      "        \"name\": \"dentate_gyrus\", \n",
      "        \"cell_of_origin\": \"dentate gyrus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_dentate_gyrus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - diencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"diencephalon\", \n",
      "        \"name\": \"diencephalon\", \n",
      "        \"cell_of_origin\": \"diencephalon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_diencephalon_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - diencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"diencephalon\", \n",
      "        \"name\": \"diencephalon\", \n",
      "        \"cell_of_origin\": \"diencephalon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_diencephalon_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - duodenum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"duodenum\", \n",
      "        \"name\": \"duodenum\", \n",
      "        \"cell_of_origin\": \"duodenum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_duodenum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - duodenum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"duodenum\", \n",
      "        \"name\": \"duodenum\", \n",
      "        \"cell_of_origin\": \"duodenum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_duodenum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ear|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ear\", \n",
      "        \"name\": \"ear\", \n",
      "        \"cell_of_origin\": \"ear\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ear_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ear|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ear\", \n",
      "        \"name\": \"ear\", \n",
      "        \"cell_of_origin\": \"ear\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ear_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - embryo|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"embryo\", \n",
      "        \"name\": \"embryo\", \n",
      "        \"cell_of_origin\": \"embryo\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_embryo_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - embryo|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"embryo\", \n",
      "        \"name\": \"embryo\", \n",
      "        \"cell_of_origin\": \"embryo\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_embryo_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - eosinophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"eosinophil\", \n",
      "        \"name\": \"eosinophil\", \n",
      "        \"cell_of_origin\": \"eosinophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_eosinophil_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - eosinophil|1 - granulocyte|2 - leukocyte|3 - hematopoietic cell|4 - blood|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"eosinophil\", \n",
      "        \"name\": \"eosinophil\", \n",
      "        \"cell_of_origin\": \"eosinophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_eosinophil_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - epidermis|1 - skin|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"epidermis\", \n",
      "        \"name\": \"epidermis\", \n",
      "        \"cell_of_origin\": \"epidermis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_epidermis_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - epidermis|1 - skin|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"epidermis\", \n",
      "        \"name\": \"epidermis\", \n",
      "        \"cell_of_origin\": \"epidermis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_epidermis_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"esophagus\", \n",
      "        \"name\": \"esophagus\", \n",
      "        \"cell_of_origin\": \"esophagus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_esophagus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - esophagus|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"esophagus\", \n",
      "        \"name\": \"esophagus\", \n",
      "        \"cell_of_origin\": \"esophagus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_esophagus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - eye|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"eye\", \n",
      "        \"name\": \"eye\", \n",
      "        \"cell_of_origin\": \"eye\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_eye_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - eye|1 - head|1 - sense organ|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"eye\", \n",
      "        \"name\": \"eye\", \n",
      "        \"cell_of_origin\": \"eye\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_eye_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - fetus|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"fetus\", \n",
      "        \"name\": \"fetus\", \n",
      "        \"cell_of_origin\": \"fetus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_fetus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - fetus|1 - embryonic structure|2 - whole body|2 - organism form|3 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"fetus\", \n",
      "        \"name\": \"fetus\", \n",
      "        \"cell_of_origin\": \"fetus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_fetus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - forebrain|1 - brain|2 - head|2 - central nervous system|3 - whole body|3 - nervous system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"forebrain\", \n",
      "        \"name\": \"forebrain\", \n",
      "        \"cell_of_origin\": \"forebrain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_forebrain_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - forebrain|1 - brain|2 - head|2 - central nervous system|3 - whole body|3 - nervous system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"forebrain\", \n",
      "        \"name\": \"forebrain\", \n",
      "        \"cell_of_origin\": \"forebrain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_forebrain_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - frontal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"frontal_lobe\", \n",
      "        \"name\": \"frontal_lobe\", \n",
      "        \"cell_of_origin\": \"frontal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_frontal_lobe_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - frontal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"frontal_lobe\", \n",
      "        \"name\": \"frontal_lobe\", \n",
      "        \"cell_of_origin\": \"frontal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_frontal_lobe_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - gastrointestinal tract|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"gastrointestinal_tract\", \n",
      "        \"name\": \"gastrointestinal_tract\", \n",
      "        \"cell_of_origin\": \"gastrointestinal tract\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_gastrointestinal_tract_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - gastrointestinal tract|1 - alimentary canal|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"gastrointestinal_tract\", \n",
      "        \"name\": \"gastrointestinal_tract\", \n",
      "        \"cell_of_origin\": \"gastrointestinal tract\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_gastrointestinal_tract_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glia|1 - spinal cord|1 - brain|2 - central nervous system|2 - head|3 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"glia\", \n",
      "        \"name\": \"glia\", \n",
      "        \"cell_of_origin\": \"glia\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_glia_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - glia|1 - spinal cord|1 - brain|2 - central nervous system|2 - head|3 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"glia\", \n",
      "        \"name\": \"glia\", \n",
      "        \"cell_of_origin\": \"glia\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_glia_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - granulocyte|1 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"granulocyte\", \n",
      "        \"name\": \"granulocyte\", \n",
      "        \"cell_of_origin\": \"granulocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_granulocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - granulocyte|1 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"granulocyte\", \n",
      "        \"name\": \"granulocyte\", \n",
      "        \"cell_of_origin\": \"granulocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_granulocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hair follicle|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hair_follicle\", \n",
      "        \"name\": \"hair_follicle\", \n",
      "        \"cell_of_origin\": \"hair follicle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hair_follicle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hair follicle|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hair_follicle\", \n",
      "        \"name\": \"hair_follicle\", \n",
      "        \"cell_of_origin\": \"hair follicle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hair_follicle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"heart\", \n",
      "        \"name\": \"heart\", \n",
      "        \"cell_of_origin\": \"heart\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_heart_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - heart|1 - cardiovascular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"heart\", \n",
      "        \"name\": \"heart\", \n",
      "        \"cell_of_origin\": \"heart\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_heart_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hematopoietic stem cell|1 - blast cell|1 - adult stem cell|2 - blood|2 - whole body|3 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hematopoietic_stem_cell\", \n",
      "        \"name\": \"hematopoietic_stem_cell\", \n",
      "        \"cell_of_origin\": \"hematopoietic stem cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hematopoietic_stem_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hematopoietic stem cell|1 - blast cell|1 - adult stem cell|2 - blood|2 - whole body|3 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hematopoietic_stem_cell\", \n",
      "        \"name\": \"hematopoietic_stem_cell\", \n",
      "        \"cell_of_origin\": \"hematopoietic stem cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hematopoietic_stem_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hepatocyte\", \n",
      "        \"name\": \"hepatocyte\", \n",
      "        \"cell_of_origin\": \"hepatocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hepatocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hepatocyte|1 - liver|2 - digestive gland|2 - viscus|3 - endocrine gland|3 - whole body|4 - gland|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hepatocyte\", \n",
      "        \"name\": \"hepatocyte\", \n",
      "        \"cell_of_origin\": \"hepatocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hepatocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hippocampus|1 - temporal lobe|1 - limbic system|2 - cerebral lobe|2 - brain|3 - cerebral cortex|3 - head|3 - central nervous system|4 - cerebral hemisphere|4 - whole body|4 - nervous system|5 - telencephalon|5 - animal|6 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hippocampus\", \n",
      "        \"name\": \"hippocampus\", \n",
      "        \"cell_of_origin\": \"hippocampus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hippocampus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hippocampus|1 - temporal lobe|1 - limbic system|2 - cerebral lobe|2 - brain|3 - cerebral cortex|3 - head|3 - central nervous system|4 - cerebral hemisphere|4 - whole body|4 - nervous system|5 - telencephalon|5 - animal|6 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hippocampus\", \n",
      "        \"name\": \"hippocampus\", \n",
      "        \"cell_of_origin\": \"hippocampus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hippocampus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hypophysis\", \n",
      "        \"name\": \"hypophysis\", \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hypophysis_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypophysis|1 - cerebellum|1 - endocrine gland|2 - metencephalon|2 - gland|3 - hindbrain|3 - whole body|4 - brain stem|4 - animal|5 - brain|5 - tissues, cell types and enzyme sources|6 - head|6 - central nervous system|7 - nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hypophysis\", \n",
      "        \"name\": \"hypophysis\", \n",
      "        \"cell_of_origin\": \"hypophysis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hypophysis_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypothalamus|1 - diencephalon|1 - endocrine gland|1 - limbic system|2 - forebrain|2 - gland|2 - brain|3 - whole body|3 - head|3 - central nervous system|4 - animal|4 - nervous system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hypothalamus\", \n",
      "        \"name\": \"hypothalamus\", \n",
      "        \"cell_of_origin\": \"hypothalamus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hypothalamus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - hypothalamus|1 - diencephalon|1 - endocrine gland|1 - limbic system|2 - forebrain|2 - gland|2 - brain|3 - whole body|3 - head|3 - central nervous system|4 - animal|4 - nervous system|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"hypothalamus\", \n",
      "        \"name\": \"hypothalamus\", \n",
      "        \"cell_of_origin\": \"hypothalamus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_hypothalamus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ileum|1 - small intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ileum\", \n",
      "        \"name\": \"ileum\", \n",
      "        \"cell_of_origin\": \"ileum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ileum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ileum|1 - small intestine|1 - ileocecum|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ileum\", \n",
      "        \"name\": \"ileum\", \n",
      "        \"cell_of_origin\": \"ileum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ileum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - intestine|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"intestine\", \n",
      "        \"name\": \"intestine\", \n",
      "        \"cell_of_origin\": \"intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_intestine_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - intestine|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"intestine\", \n",
      "        \"name\": \"intestine\", \n",
      "        \"cell_of_origin\": \"intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_intestine_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - jejunum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"jejunum\", \n",
      "        \"name\": \"jejunum\", \n",
      "        \"cell_of_origin\": \"jejunum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_jejunum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - jejunum|1 - small intestine|2 - intestine|3 - gastrointestinal tract|4 - alimentary canal|5 - viscus|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"jejunum\", \n",
      "        \"name\": \"jejunum\", \n",
      "        \"cell_of_origin\": \"jejunum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_jejunum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - keratinocyte|1 - stratum spinosum|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"keratinocyte\", \n",
      "        \"name\": \"keratinocyte\", \n",
      "        \"cell_of_origin\": \"keratinocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_keratinocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - keratinocyte|1 - stratum spinosum|1 - epidermal cell|2 - epidermis|3 - skin|4 - integument|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"keratinocyte\", \n",
      "        \"name\": \"keratinocyte\", \n",
      "        \"cell_of_origin\": \"keratinocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_keratinocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - kidney|1 - excretory gland|1 - viscus|1 - urinary system|1 - urinary tract|2 - gland|2 - whole body|2 - urogenital system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"kidney\", \n",
      "        \"name\": \"kidney\", \n",
      "        \"cell_of_origin\": \"kidney\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_kidney_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - kidney|1 - excretory gland|1 - viscus|1 - urinary system|1 - urinary tract|2 - gland|2 - whole body|2 - urogenital system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"kidney\", \n",
      "        \"name\": \"kidney\", \n",
      "        \"cell_of_origin\": \"kidney\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_kidney_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - large intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"large_intestine\", \n",
      "        \"name\": \"large_intestine\", \n",
      "        \"cell_of_origin\": \"large intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_large_intestine_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - large intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"large_intestine\", \n",
      "        \"name\": \"large_intestine\", \n",
      "        \"cell_of_origin\": \"large intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_large_intestine_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lens|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lens\", \n",
      "        \"name\": \"lens\", \n",
      "        \"cell_of_origin\": \"lens\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lens_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lens|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lens\", \n",
      "        \"name\": \"lens\", \n",
      "        \"cell_of_origin\": \"lens\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lens_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - leukocyte|1 - hematopoietic cell|2 - blood|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"leukocyte\", \n",
      "        \"name\": \"leukocyte\", \n",
      "        \"cell_of_origin\": \"leukocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_leukocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - leukocyte|1 - hematopoietic cell|2 - blood|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"leukocyte\", \n",
      "        \"name\": \"leukocyte\", \n",
      "        \"cell_of_origin\": \"leukocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_leukocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"liver\", \n",
      "        \"name\": \"liver\", \n",
      "        \"cell_of_origin\": \"liver\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_liver_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - liver|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"liver\", \n",
      "        \"name\": \"liver\", \n",
      "        \"cell_of_origin\": \"liver\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_liver_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - locus ceruleus|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"locus_ceruleus\", \n",
      "        \"name\": \"locus_ceruleus\", \n",
      "        \"cell_of_origin\": \"locus ceruleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_locus_ceruleus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - locus ceruleus|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"locus_ceruleus\", \n",
      "        \"name\": \"locus_ceruleus\", \n",
      "        \"cell_of_origin\": \"locus ceruleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_locus_ceruleus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lung\", \n",
      "        \"name\": \"lung\", \n",
      "        \"cell_of_origin\": \"lung\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lung_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lung|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lung\", \n",
      "        \"name\": \"lung\", \n",
      "        \"cell_of_origin\": \"lung\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lung_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lymph node|1 - immune system|1 - endocrine gland|1 - lymphoid tissue|2 - whole body|2 - gland|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lymph_node\", \n",
      "        \"name\": \"lymph_node\", \n",
      "        \"cell_of_origin\": \"lymph node\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lymph_node_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lymph node|1 - immune system|1 - endocrine gland|1 - lymphoid tissue|2 - whole body|2 - gland|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lymph_node\", \n",
      "        \"name\": \"lymph_node\", \n",
      "        \"cell_of_origin\": \"lymph node\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lymph_node_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lymphocyte|1 - leukocyte|1 - lymphoid cell|2 - hematopoietic cell|2 - lymphoid tissue|3 - blood|3 - immune system|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lymphocyte\", \n",
      "        \"name\": \"lymphocyte\", \n",
      "        \"cell_of_origin\": \"lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - lymphocyte|1 - leukocyte|1 - lymphoid cell|2 - hematopoietic cell|2 - lymphoid tissue|3 - blood|3 - immune system|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"lymphocyte\", \n",
      "        \"name\": \"lymphocyte\", \n",
      "        \"cell_of_origin\": \"lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - macrophage|1 - immune system|1 - monocyte|2 - whole body|2 - leukocyte|2 - mononuclear phagocyte|3 - animal|3 - hematopoietic cell|3 - mononuclear cell|3 - marrow cell|3 - phagocyte|4 - tissues, cell types and enzyme sources|4 - blood|4 - myeloid progenitor cell|5 - hematopoietic system|5 - hematopoietic stem cell|6 - blast cell|6 - adult stem cell\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"macrophage\", \n",
      "        \"name\": \"macrophage\", \n",
      "        \"cell_of_origin\": \"macrophage\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_macrophage_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - macrophage|1 - immune system|1 - monocyte|2 - whole body|2 - leukocyte|2 - mononuclear phagocyte|3 - animal|3 - hematopoietic cell|3 - mononuclear cell|3 - marrow cell|3 - phagocyte|4 - tissues, cell types and enzyme sources|4 - blood|4 - myeloid progenitor cell|5 - hematopoietic system|5 - hematopoietic stem cell|6 - blast cell|6 - adult stem cell\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"macrophage\", \n",
      "        \"name\": \"macrophage\", \n",
      "        \"cell_of_origin\": \"macrophage\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_macrophage_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mammary epithelium|1 - breast epithelium|2 - breast|2 - epithelium|3 - thorax|3 - female reproductive gland|3 - excretory gland|3 - integument|4 - trunk|4 - female reproductive system|4 - endocrine gland|4 - gland|4 - whole body|5 - reproductive system|5 - animal|6 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mammary_epithelium\", \n",
      "        \"name\": \"mammary_epithelium\", \n",
      "        \"cell_of_origin\": \"mammary epithelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mammary_epithelium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mammary epithelium|1 - breast epithelium|2 - breast|2 - epithelium|3 - thorax|3 - female reproductive gland|3 - excretory gland|3 - integument|4 - trunk|4 - female reproductive system|4 - endocrine gland|4 - gland|4 - whole body|5 - reproductive system|5 - animal|6 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mammary_epithelium\", \n",
      "        \"name\": \"mammary_epithelium\", \n",
      "        \"cell_of_origin\": \"mammary epithelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mammary_epithelium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mammary gland|1 - breast|2 - thorax|2 - female reproductive gland|2 - excretory gland|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mammary_gland\", \n",
      "        \"name\": \"mammary_gland\", \n",
      "        \"cell_of_origin\": \"mammary gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mammary_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mammary gland|1 - breast|2 - thorax|2 - female reproductive gland|2 - excretory gland|3 - trunk|3 - female reproductive system|3 - endocrine gland|3 - gland|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mammary_gland\", \n",
      "        \"name\": \"mammary_gland\", \n",
      "        \"cell_of_origin\": \"mammary gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mammary_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mast cell|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mast_cell\", \n",
      "        \"name\": \"mast_cell\", \n",
      "        \"cell_of_origin\": \"mast cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mast_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mast cell|1 - connective tissue|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mast_cell\", \n",
      "        \"name\": \"mast_cell\", \n",
      "        \"cell_of_origin\": \"mast cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mast_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - medulla oblongata|1 - myelencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"medulla_oblongata\", \n",
      "        \"name\": \"medulla_oblongata\", \n",
      "        \"cell_of_origin\": \"medulla oblongata\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_medulla_oblongata_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - medulla oblongata|1 - myelencephalon|2 - hindbrain|3 - brain stem|4 - brain|5 - head|5 - central nervous system|6 - whole body|6 - nervous system|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"medulla_oblongata\", \n",
      "        \"name\": \"medulla_oblongata\", \n",
      "        \"cell_of_origin\": \"medulla oblongata\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_medulla_oblongata_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - megakaryocyte|1 - megakaryoblast|1 - marrow cell|2 - bone marrow|2 - bone marrow cell|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - skeletal system|3 - blood|3 - hematopoietic stem cell|4 - whole body|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"megakaryocyte\", \n",
      "        \"name\": \"megakaryocyte\", \n",
      "        \"cell_of_origin\": \"megakaryocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_megakaryocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - megakaryocyte|1 - megakaryoblast|1 - marrow cell|2 - bone marrow|2 - bone marrow cell|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - skeletal system|3 - blood|3 - hematopoietic stem cell|4 - whole body|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"megakaryocyte\", \n",
      "        \"name\": \"megakaryocyte\", \n",
      "        \"cell_of_origin\": \"megakaryocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_megakaryocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - midbrain|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"midbrain\", \n",
      "        \"name\": \"midbrain\", \n",
      "        \"cell_of_origin\": \"midbrain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_midbrain_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - midbrain|1 - brain stem|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"midbrain\", \n",
      "        \"name\": \"midbrain\", \n",
      "        \"cell_of_origin\": \"midbrain\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_midbrain_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - monocyte|1 - leukocyte|1 - mononuclear phagocyte|2 - hematopoietic cell|2 - mononuclear cell|2 - marrow cell|2 - phagocyte|3 - blood|3 - myeloid progenitor cell|4 - hematopoietic system|4 - hematopoietic stem cell|5 - whole body|5 - blast cell|5 - adult stem cell|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"monocyte\", \n",
      "        \"name\": \"monocyte\", \n",
      "        \"cell_of_origin\": \"monocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_monocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - monocyte|1 - leukocyte|1 - mononuclear phagocyte|2 - hematopoietic cell|2 - mononuclear cell|2 - marrow cell|2 - phagocyte|3 - blood|3 - myeloid progenitor cell|4 - hematopoietic system|4 - hematopoietic stem cell|5 - whole body|5 - blast cell|5 - adult stem cell|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"monocyte\", \n",
      "        \"name\": \"monocyte\", \n",
      "        \"cell_of_origin\": \"monocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_monocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mononuclear phagocyte|1 - mononuclear cell|1 - marrow cell|1 - phagocyte|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - blood|3 - hematopoietic stem cell|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mononuclear_phagocyte\", \n",
      "        \"name\": \"mononuclear_phagocyte\", \n",
      "        \"cell_of_origin\": \"mononuclear phagocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mononuclear_phagocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - mononuclear phagocyte|1 - mononuclear cell|1 - marrow cell|1 - phagocyte|2 - hematopoietic cell|2 - myeloid progenitor cell|3 - blood|3 - hematopoietic stem cell|4 - hematopoietic system|4 - blast cell|4 - adult stem cell|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"mononuclear_phagocyte\", \n",
      "        \"name\": \"mononuclear_phagocyte\", \n",
      "        \"cell_of_origin\": \"mononuclear phagocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_mononuclear_phagocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"muscle\", \n",
      "        \"name\": \"muscle\", \n",
      "        \"cell_of_origin\": \"muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_muscle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - muscle|1 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"muscle\", \n",
      "        \"name\": \"muscle\", \n",
      "        \"cell_of_origin\": \"muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_muscle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - myometrium|1 - smooth muscle|1 - uterus|2 - muscle|2 - vertebrate muscular system|2 - internal female genital organ|3 - muscular system|3 - female reproductive system|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"myometrium\", \n",
      "        \"name\": \"myometrium\", \n",
      "        \"cell_of_origin\": \"myometrium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_myometrium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - myometrium|1 - smooth muscle|1 - uterus|2 - muscle|2 - vertebrate muscular system|2 - internal female genital organ|3 - muscular system|3 - female reproductive system|4 - whole body|4 - reproductive system|5 - animal|5 - urogenital system|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"myometrium\", \n",
      "        \"name\": \"myometrium\", \n",
      "        \"cell_of_origin\": \"myometrium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_myometrium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - natural killer cell|1 - null cell|1 - large granular lymphocyte|2 - lymphocyte|3 - leukocyte|3 - lymphoid cell|4 - hematopoietic cell|4 - lymphoid tissue|5 - blood|5 - immune system|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"natural_killer_cell\", \n",
      "        \"name\": \"natural_killer_cell\", \n",
      "        \"cell_of_origin\": \"natural killer cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - natural killer cell|1 - null cell|1 - large granular lymphocyte|2 - lymphocyte|3 - leukocyte|3 - lymphoid cell|4 - hematopoietic cell|4 - lymphoid tissue|5 - blood|5 - immune system|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"natural_killer_cell\", \n",
      "        \"name\": \"natural_killer_cell\", \n",
      "        \"cell_of_origin\": \"natural killer cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nephron|1 - kidney|2 - excretory gland|2 - viscus|2 - urinary system|2 - urinary tract|3 - gland|3 - whole body|3 - urogenital system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nephron\", \n",
      "        \"name\": \"nephron\", \n",
      "        \"cell_of_origin\": \"nephron\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nephron_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nephron|1 - kidney|2 - excretory gland|2 - viscus|2 - urinary system|2 - urinary tract|3 - gland|3 - whole body|3 - urogenital system|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nephron\", \n",
      "        \"name\": \"nephron\", \n",
      "        \"cell_of_origin\": \"nephron\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nephron_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nervous system|1 - whole body|2 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nervous_system\", \n",
      "        \"name\": \"nervous_system\", \n",
      "        \"cell_of_origin\": \"nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nervous_system_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nervous system|1 - whole body|2 - animal|3 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nervous_system\", \n",
      "        \"name\": \"nervous_system\", \n",
      "        \"cell_of_origin\": \"nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nervous_system_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - neuron|1 - nervous system|1 - nerve|1 - neuroblast|2 - whole body|2 - embryo|3 - animal|3 - embryonic structure|4 - tissues, cell types and enzyme sources|4 - organism form\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"neuron\", \n",
      "        \"name\": \"neuron\", \n",
      "        \"cell_of_origin\": \"neuron\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_neuron_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - neuron|1 - nervous system|1 - nerve|1 - neuroblast|2 - whole body|2 - embryo|3 - animal|3 - embryonic structure|4 - tissues, cell types and enzyme sources|4 - organism form\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"neuron\", \n",
      "        \"name\": \"neuron\", \n",
      "        \"cell_of_origin\": \"neuron\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_neuron_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - neutrophil|1 - granulocyte|1 - phagocyte|2 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"neutrophil\", \n",
      "        \"name\": \"neutrophil\", \n",
      "        \"cell_of_origin\": \"neutrophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_neutrophil_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - neutrophil|1 - granulocyte|1 - phagocyte|2 - leukocyte|2 - hematopoietic cell|3 - blood|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"neutrophil\", \n",
      "        \"name\": \"neutrophil\", \n",
      "        \"cell_of_origin\": \"neutrophil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_neutrophil_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nucleus accumbens|1 - lateral ventricle|2 - brain ventricle|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nucleus_accumbens\", \n",
      "        \"name\": \"nucleus_accumbens\", \n",
      "        \"cell_of_origin\": \"nucleus accumbens\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nucleus_accumbens_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - nucleus accumbens|1 - lateral ventricle|2 - brain ventricle|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"nucleus_accumbens\", \n",
      "        \"name\": \"nucleus_accumbens\", \n",
      "        \"cell_of_origin\": \"nucleus accumbens\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_nucleus_accumbens_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - occipital lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"occipital_lobe\", \n",
      "        \"name\": \"occipital_lobe\", \n",
      "        \"cell_of_origin\": \"occipital lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_occipital_lobe_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - occipital lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"occipital_lobe\", \n",
      "        \"name\": \"occipital_lobe\", \n",
      "        \"cell_of_origin\": \"occipital lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_occipital_lobe_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - occipital pole|1 - occipital lobe|2 - cerebral lobe|3 - cerebral cortex|4 - cerebral hemisphere|5 - telencephalon|6 - forebrain|7 - brain|8 - head|8 - central nervous system|9 - whole body|9 - nervous system|10 - animal|11 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"occipital_pole\", \n",
      "        \"name\": \"occipital_pole\", \n",
      "        \"cell_of_origin\": \"occipital pole\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_occipital_pole_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - occipital pole|1 - occipital lobe|2 - cerebral lobe|3 - cerebral cortex|4 - cerebral hemisphere|5 - telencephalon|6 - forebrain|7 - brain|8 - head|8 - central nervous system|9 - whole body|9 - nervous system|10 - animal|11 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"occipital_pole\", \n",
      "        \"name\": \"occipital_pole\", \n",
      "        \"cell_of_origin\": \"occipital pole\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_occipital_pole_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - osteoblast|1 - bone|1 - osteogenic cell|1 - preosteoblast|2 - connective tissue|2 - skeletal system|2 - periosteum|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"osteoblast\", \n",
      "        \"name\": \"osteoblast\", \n",
      "        \"cell_of_origin\": \"osteoblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_osteoblast_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - osteoblast|1 - bone|1 - osteogenic cell|1 - preosteoblast|2 - connective tissue|2 - skeletal system|2 - periosteum|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"osteoblast\", \n",
      "        \"name\": \"osteoblast\", \n",
      "        \"cell_of_origin\": \"osteoblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_osteoblast_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian follicle|1 - ovary|2 - female reproductive gland|2 - internal female genital organ|3 - female reproductive system|3 - endocrine gland|4 - reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ovarian_follicle\", \n",
      "        \"name\": \"ovarian_follicle\", \n",
      "        \"cell_of_origin\": \"ovarian follicle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ovarian_follicle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovarian follicle|1 - ovary|2 - female reproductive gland|2 - internal female genital organ|3 - female reproductive system|3 - endocrine gland|4 - reproductive system|4 - gland|5 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ovarian_follicle\", \n",
      "        \"name\": \"ovarian_follicle\", \n",
      "        \"cell_of_origin\": \"ovarian follicle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ovarian_follicle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovary|1 - female reproductive gland|1 - internal female genital organ|2 - female reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ovary\", \n",
      "        \"name\": \"ovary\", \n",
      "        \"cell_of_origin\": \"ovary\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ovary_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - ovary|1 - female reproductive gland|1 - internal female genital organ|2 - female reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"ovary\", \n",
      "        \"name\": \"ovary\", \n",
      "        \"cell_of_origin\": \"ovary\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_ovary_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - oviduct|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"oviduct\", \n",
      "        \"name\": \"oviduct\", \n",
      "        \"cell_of_origin\": \"oviduct\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_oviduct_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - oviduct|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"oviduct\", \n",
      "        \"name\": \"oviduct\", \n",
      "        \"cell_of_origin\": \"oviduct\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_oviduct_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pancreas\", \n",
      "        \"name\": \"pancreas\", \n",
      "        \"cell_of_origin\": \"pancreas\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pancreas_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreas|1 - digestive gland|1 - viscus|2 - endocrine gland|2 - whole body|3 - gland|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pancreas\", \n",
      "        \"name\": \"pancreas\", \n",
      "        \"cell_of_origin\": \"pancreas\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pancreas_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic islet|1 - endocrine pancreas|2 - pancreas|3 - digestive gland|3 - viscus|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pancreatic_islet\", \n",
      "        \"name\": \"pancreatic_islet\", \n",
      "        \"cell_of_origin\": \"pancreatic islet\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pancreatic_islet_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pancreatic islet|1 - endocrine pancreas|2 - pancreas|3 - digestive gland|3 - viscus|4 - endocrine gland|4 - whole body|5 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pancreatic_islet\", \n",
      "        \"name\": \"pancreatic_islet\", \n",
      "        \"cell_of_origin\": \"pancreatic islet\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pancreatic_islet_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - parietal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"parietal_lobe\", \n",
      "        \"name\": \"parietal_lobe\", \n",
      "        \"cell_of_origin\": \"parietal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_parietal_lobe_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - parietal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"parietal_lobe\", \n",
      "        \"name\": \"parietal_lobe\", \n",
      "        \"cell_of_origin\": \"parietal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_parietal_lobe_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - peripheral nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"peripheral_nervous_system\", \n",
      "        \"name\": \"peripheral_nervous_system\", \n",
      "        \"cell_of_origin\": \"peripheral nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_peripheral_nervous_system_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - peripheral nervous system|1 - nervous system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"peripheral_nervous_system\", \n",
      "        \"name\": \"peripheral_nervous_system\", \n",
      "        \"cell_of_origin\": \"peripheral nervous system\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_peripheral_nervous_system_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - placenta|1 - fetus|1 - internal female genital organ|2 - embryonic structure|2 - female reproductive system|3 - whole body|3 - organism form|3 - reproductive system|4 - animal|4 - tissues, cell types and enzyme sources|4 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"placenta\", \n",
      "        \"name\": \"placenta\", \n",
      "        \"cell_of_origin\": \"placenta\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_placenta_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - placenta|1 - fetus|1 - internal female genital organ|2 - embryonic structure|2 - female reproductive system|3 - whole body|3 - organism form|3 - reproductive system|4 - animal|4 - tissues, cell types and enzyme sources|4 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"placenta\", \n",
      "        \"name\": \"placenta\", \n",
      "        \"cell_of_origin\": \"placenta\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_placenta_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - podocyte|1 - glomerular epithelium|1 - capsular epithelium|1 - epithelial cell|2 - renal glomerular capsule|2 - epithelium|3 - renal corpuscle|3 - integument|4 - nephron|4 - whole body|5 - kidney|5 - animal|6 - excretory gland|6 - viscus|6 - urinary system|6 - urinary tract|6 - tissues, cell types and enzyme sources|7 - gland|7 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"podocyte\", \n",
      "        \"name\": \"podocyte\", \n",
      "        \"cell_of_origin\": \"podocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_podocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - podocyte|1 - glomerular epithelium|1 - capsular epithelium|1 - epithelial cell|2 - renal glomerular capsule|2 - epithelium|3 - renal corpuscle|3 - integument|4 - nephron|4 - whole body|5 - kidney|5 - animal|6 - excretory gland|6 - viscus|6 - urinary system|6 - urinary tract|6 - tissues, cell types and enzyme sources|7 - gland|7 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"podocyte\", \n",
      "        \"name\": \"podocyte\", \n",
      "        \"cell_of_origin\": \"podocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_podocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pons|1 - brain stem|1 - metencephalon|2 - brain|2 - hindbrain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pons\", \n",
      "        \"name\": \"pons\", \n",
      "        \"cell_of_origin\": \"pons\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pons_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - pons|1 - brain stem|1 - metencephalon|2 - brain|2 - hindbrain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"pons\", \n",
      "        \"name\": \"pons\", \n",
      "        \"cell_of_origin\": \"pons\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_pons_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"prostate_gland\", \n",
      "        \"name\": \"prostate_gland\", \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_prostate_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - prostate gland|1 - internal male genital organ|1 - male reproductive gland|2 - male reproductive system|2 - endocrine gland|3 - reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"prostate_gland\", \n",
      "        \"name\": \"prostate_gland\", \n",
      "        \"cell_of_origin\": \"prostate gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_prostate_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal glomerulus|1 - renal corpuscle|2 - nephron|3 - kidney|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|5 - gland|5 - whole body|5 - urogenital system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"renal_glomerulus\", \n",
      "        \"name\": \"renal_glomerulus\", \n",
      "        \"cell_of_origin\": \"renal glomerulus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_renal_glomerulus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal glomerulus|1 - renal corpuscle|2 - nephron|3 - kidney|4 - excretory gland|4 - viscus|4 - urinary system|4 - urinary tract|5 - gland|5 - whole body|5 - urogenital system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"renal_glomerulus\", \n",
      "        \"name\": \"renal_glomerulus\", \n",
      "        \"cell_of_origin\": \"renal glomerulus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_renal_glomerulus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal tubule|1 - nephron|2 - kidney|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|4 - gland|4 - whole body|4 - urogenital system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"renal_tubule\", \n",
      "        \"name\": \"renal_tubule\", \n",
      "        \"cell_of_origin\": \"renal tubule\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_renal_tubule_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - renal tubule|1 - nephron|2 - kidney|3 - excretory gland|3 - viscus|3 - urinary system|3 - urinary tract|4 - gland|4 - whole body|4 - urogenital system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"renal_tubule\", \n",
      "        \"name\": \"renal_tubule\", \n",
      "        \"cell_of_origin\": \"renal tubule\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_renal_tubule_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - retina|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"retina\", \n",
      "        \"name\": \"retina\", \n",
      "        \"cell_of_origin\": \"retina\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_retina_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - retina|1 - eye|2 - head|2 - sense organ|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"retina\", \n",
      "        \"name\": \"retina\", \n",
      "        \"cell_of_origin\": \"retina\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_retina_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - salivary gland|1 - mouth|1 - exocrine gland|2 - head|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"salivary_gland\", \n",
      "        \"name\": \"salivary_gland\", \n",
      "        \"cell_of_origin\": \"salivary gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_salivary_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - salivary gland|1 - mouth|1 - exocrine gland|2 - head|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"salivary_gland\", \n",
      "        \"name\": \"salivary_gland\", \n",
      "        \"cell_of_origin\": \"salivary gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_salivary_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - serum|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"serum\", \n",
      "        \"name\": \"serum\", \n",
      "        \"cell_of_origin\": \"serum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_serum_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - serum|1 - hematopoietic system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"serum\", \n",
      "        \"name\": \"serum\", \n",
      "        \"cell_of_origin\": \"serum\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_serum_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skeletal muscle|1 - muscle|1 - vertebrate muscular system|1 - skeletal system|2 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skeletal_muscle\", \n",
      "        \"name\": \"skeletal_muscle\", \n",
      "        \"cell_of_origin\": \"skeletal muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skeletal_muscle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skeletal muscle|1 - muscle|1 - vertebrate muscular system|1 - skeletal system|2 - muscular system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skeletal_muscle\", \n",
      "        \"name\": \"skeletal_muscle\", \n",
      "        \"cell_of_origin\": \"skeletal muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skeletal_muscle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skin\", \n",
      "        \"name\": \"skin\", \n",
      "        \"cell_of_origin\": \"skin\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skin_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin|1 - integument|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skin\", \n",
      "        \"name\": \"skin\", \n",
      "        \"cell_of_origin\": \"skin\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skin_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin fibroblast|1 - fibroblast|1 - skin|2 - connective tissue|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skin_fibroblast\", \n",
      "        \"name\": \"skin_fibroblast\", \n",
      "        \"cell_of_origin\": \"skin fibroblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skin_fibroblast_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - skin fibroblast|1 - fibroblast|1 - skin|2 - connective tissue|2 - integument|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"skin_fibroblast\", \n",
      "        \"name\": \"skin_fibroblast\", \n",
      "        \"cell_of_origin\": \"skin fibroblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_skin_fibroblast_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - small intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"small_intestine\", \n",
      "        \"name\": \"small_intestine\", \n",
      "        \"cell_of_origin\": \"small intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_small_intestine_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - small intestine|1 - intestine|2 - gastrointestinal tract|3 - alimentary canal|4 - viscus|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"small_intestine\", \n",
      "        \"name\": \"small_intestine\", \n",
      "        \"cell_of_origin\": \"small intestine\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_small_intestine_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - smooth muscle|1 - muscle|1 - vertebrate muscular system|2 - muscular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"smooth_muscle\", \n",
      "        \"name\": \"smooth_muscle\", \n",
      "        \"cell_of_origin\": \"smooth muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_smooth_muscle_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - smooth muscle|1 - muscle|1 - vertebrate muscular system|2 - muscular system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"smooth_muscle\", \n",
      "        \"name\": \"smooth_muscle\", \n",
      "        \"cell_of_origin\": \"smooth muscle\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_smooth_muscle_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatid|1 - secondary spermatocyte|2 - primary spermatocyte|3 - spermatocyte|4 - spermatogonium|4 - semen|4 - germ cell|5 - male reproductive system|5 - reproductive system|5 - gametocyte|6 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatid\", \n",
      "        \"name\": \"spermatid\", \n",
      "        \"cell_of_origin\": \"spermatid\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatid_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatid|1 - secondary spermatocyte|2 - primary spermatocyte|3 - spermatocyte|4 - spermatogonium|4 - semen|4 - germ cell|5 - male reproductive system|5 - reproductive system|5 - gametocyte|6 - urogenital system|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatid\", \n",
      "        \"name\": \"spermatid\", \n",
      "        \"cell_of_origin\": \"spermatid\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatid_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatocyte|1 - spermatogonium|1 - semen|1 - germ cell|2 - male reproductive system|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatocyte\", \n",
      "        \"name\": \"spermatocyte\", \n",
      "        \"cell_of_origin\": \"spermatocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatocyte|1 - spermatogonium|1 - semen|1 - germ cell|2 - male reproductive system|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatocyte\", \n",
      "        \"name\": \"spermatocyte\", \n",
      "        \"cell_of_origin\": \"spermatocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatogonium|1 - male reproductive system|1 - germ cell|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatogonium\", \n",
      "        \"name\": \"spermatogonium\", \n",
      "        \"cell_of_origin\": \"spermatogonium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatogonium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spermatogonium|1 - male reproductive system|1 - germ cell|2 - reproductive system|2 - gametocyte|3 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spermatogonium\", \n",
      "        \"name\": \"spermatogonium\", \n",
      "        \"cell_of_origin\": \"spermatogonium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spermatogonium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spinal cord|1 - central nervous system|2 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spinal_cord\", \n",
      "        \"name\": \"spinal_cord\", \n",
      "        \"cell_of_origin\": \"spinal cord\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spinal_cord_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spinal cord|1 - central nervous system|2 - nervous system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spinal_cord\", \n",
      "        \"name\": \"spinal_cord\", \n",
      "        \"cell_of_origin\": \"spinal cord\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spinal_cord_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spleen\", \n",
      "        \"name\": \"spleen\", \n",
      "        \"cell_of_origin\": \"spleen\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spleen_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - spleen|1 - immune system|1 - viscus|1 - lymphoid tissue|2 - whole body|2 - hematopoietic system|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"spleen\", \n",
      "        \"name\": \"spleen\", \n",
      "        \"cell_of_origin\": \"spleen\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_spleen_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"stomach\", \n",
      "        \"name\": \"stomach\", \n",
      "        \"cell_of_origin\": \"stomach\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_stomach_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - stomach|1 - gastrointestinal tract|2 - alimentary canal|3 - viscus|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"stomach\", \n",
      "        \"name\": \"stomach\", \n",
      "        \"cell_of_origin\": \"stomach\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_stomach_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - substantia nigra|1 - midbrain|1 - basal ganglion|2 - brain stem|2 - ganglion|2 - cerebral cortex|3 - brain|3 - nervous system|3 - cerebral hemisphere|4 - head|4 - central nervous system|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"substantia_nigra\", \n",
      "        \"name\": \"substantia_nigra\", \n",
      "        \"cell_of_origin\": \"substantia nigra\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_substantia_nigra_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - substantia nigra|1 - midbrain|1 - basal ganglion|2 - brain stem|2 - ganglion|2 - cerebral cortex|3 - brain|3 - nervous system|3 - cerebral hemisphere|4 - head|4 - central nervous system|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"substantia_nigra\", \n",
      "        \"name\": \"substantia_nigra\", \n",
      "        \"cell_of_origin\": \"substantia nigra\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_substantia_nigra_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - subthalamic nucleus|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"subthalamic_nucleus\", \n",
      "        \"name\": \"subthalamic_nucleus\", \n",
      "        \"cell_of_origin\": \"subthalamic nucleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_subthalamic_nucleus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - subthalamic nucleus|1 - basal ganglion|2 - ganglion|2 - cerebral cortex|3 - nervous system|3 - cerebral hemisphere|4 - whole body|4 - telencephalon|5 - animal|5 - forebrain|6 - tissues, cell types and enzyme sources|6 - brain|7 - head|7 - central nervous system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"subthalamic_nucleus\", \n",
      "        \"name\": \"subthalamic_nucleus\", \n",
      "        \"cell_of_origin\": \"subthalamic nucleus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_subthalamic_nucleus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - T-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"t_lymphocyte\", \n",
      "        \"name\": \"t_lymphocyte\", \n",
      "        \"cell_of_origin\": \"T-lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - T-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"t_lymphocyte\", \n",
      "        \"name\": \"t_lymphocyte\", \n",
      "        \"cell_of_origin\": \"T-lymphocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tear gland|1 - eye|1 - excretory gland|2 - head|2 - sense organ|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tear_gland\", \n",
      "        \"name\": \"tear_gland\", \n",
      "        \"cell_of_origin\": \"tear gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tear_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tear gland|1 - eye|1 - excretory gland|2 - head|2 - sense organ|2 - gland|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tear_gland\", \n",
      "        \"name\": \"tear_gland\", \n",
      "        \"cell_of_origin\": \"tear gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tear_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - telencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"telencephalon\", \n",
      "        \"name\": \"telencephalon\", \n",
      "        \"cell_of_origin\": \"telencephalon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_telencephalon_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - telencephalon|1 - forebrain|2 - brain|3 - head|3 - central nervous system|4 - whole body|4 - nervous system|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"telencephalon\", \n",
      "        \"name\": \"telencephalon\", \n",
      "        \"cell_of_origin\": \"telencephalon\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_telencephalon_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - temporal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"temporal_lobe\", \n",
      "        \"name\": \"temporal_lobe\", \n",
      "        \"cell_of_origin\": \"temporal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_temporal_lobe_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - temporal lobe|1 - cerebral lobe|2 - cerebral cortex|3 - cerebral hemisphere|4 - telencephalon|5 - forebrain|6 - brain|7 - head|7 - central nervous system|8 - whole body|8 - nervous system|9 - animal|10 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"temporal_lobe\", \n",
      "        \"name\": \"temporal_lobe\", \n",
      "        \"cell_of_origin\": \"temporal lobe\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_temporal_lobe_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"testis\", \n",
      "        \"name\": \"testis\", \n",
      "        \"cell_of_origin\": \"testis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_testis_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - testis|1 - internal male genital organ|1 - gonad|1 - male reproductive gland|2 - male reproductive system|2 - female reproductive gland|2 - endocrine gland|3 - reproductive system|3 - female reproductive system|3 - gland|4 - urogenital system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"testis\", \n",
      "        \"name\": \"testis\", \n",
      "        \"cell_of_origin\": \"testis\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_testis_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thalamus|1 - diencephalon|2 - forebrain|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thalamus\", \n",
      "        \"name\": \"thalamus\", \n",
      "        \"cell_of_origin\": \"thalamus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thalamus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thalamus|1 - diencephalon|2 - forebrain|3 - brain|4 - head|4 - central nervous system|5 - whole body|5 - nervous system|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thalamus\", \n",
      "        \"name\": \"thalamus\", \n",
      "        \"cell_of_origin\": \"thalamus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thalamus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thymocyte|1 - thymus|1 - pre-T-lymphocyte|2 - immune system|2 - thorax|2 - endocrine gland|2 - lymphoid tissue|2 - T-lymphocyte|3 - whole body|3 - trunk|3 - gland|3 - hematopoietic system|3 - lymphocyte|4 - animal|4 - leukocyte|4 - lymphoid cell|5 - tissues, cell types and enzyme sources|5 - hematopoietic cell|6 - blood\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thymocyte\", \n",
      "        \"name\": \"thymocyte\", \n",
      "        \"cell_of_origin\": \"thymocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thymocyte|1 - thymus|1 - pre-T-lymphocyte|2 - immune system|2 - thorax|2 - endocrine gland|2 - lymphoid tissue|2 - T-lymphocyte|3 - whole body|3 - trunk|3 - gland|3 - hematopoietic system|3 - lymphocyte|4 - animal|4 - leukocyte|4 - lymphoid cell|5 - tissues, cell types and enzyme sources|5 - hematopoietic cell|6 - blood\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thymocyte\", \n",
      "        \"name\": \"thymocyte\", \n",
      "        \"cell_of_origin\": \"thymocyte\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thyroid_gland\", \n",
      "        \"name\": \"thyroid_gland\", \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thyroid_gland_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - thyroid gland|1 - throat|1 - endocrine gland|2 - neck|2 - gland|3 - trunk|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"thyroid_gland\", \n",
      "        \"name\": \"thyroid_gland\", \n",
      "        \"cell_of_origin\": \"thyroid gland\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_thyroid_gland_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tonsil|1 - lymphoid tissue|2 - immune system|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tonsil\", \n",
      "        \"name\": \"tonsil\", \n",
      "        \"cell_of_origin\": \"tonsil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tonsil_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tonsil|1 - lymphoid tissue|2 - immune system|2 - hematopoietic system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tonsil\", \n",
      "        \"name\": \"tonsil\", \n",
      "        \"cell_of_origin\": \"tonsil\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tonsil_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tooth|1 - mouth|2 - head|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tooth\", \n",
      "        \"name\": \"tooth\", \n",
      "        \"cell_of_origin\": \"tooth\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tooth_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - tooth|1 - mouth|2 - head|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"tooth\", \n",
      "        \"name\": \"tooth\", \n",
      "        \"cell_of_origin\": \"tooth\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_tooth_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - trachea|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"trachea\", \n",
      "        \"name\": \"trachea\", \n",
      "        \"cell_of_origin\": \"trachea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_trachea_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - trachea|1 - respiratory system|2 - whole body|3 - animal|4 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"trachea\", \n",
      "        \"name\": \"trachea\", \n",
      "        \"cell_of_origin\": \"trachea\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_trachea_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - trophoblast|1 - blastula|2 - morula|3 - blastomere|4 - zygote|5 - embryonic structure|6 - whole body|6 - organism form|7 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"trophoblast\", \n",
      "        \"name\": \"trophoblast\", \n",
      "        \"cell_of_origin\": \"trophoblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_trophoblast_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - trophoblast|1 - blastula|2 - morula|3 - blastomere|4 - zygote|5 - embryonic structure|6 - whole body|6 - organism form|7 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"trophoblast\", \n",
      "        \"name\": \"trophoblast\", \n",
      "        \"cell_of_origin\": \"trophoblast\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_trophoblast_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical cord|1 - placenta|1 - connecting stalk|1 - fetus|2 - internal female genital organ|2 - mesoderm|2 - embryonic structure|3 - female reproductive system|3 - germ layer|3 - whole body|3 - organism form|4 - reproductive system|4 - embryo|4 - animal|4 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"umbilical_cord\", \n",
      "        \"name\": \"umbilical_cord\", \n",
      "        \"cell_of_origin\": \"umbilical cord\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_umbilical_cord_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical cord|1 - placenta|1 - connecting stalk|1 - fetus|2 - internal female genital organ|2 - mesoderm|2 - embryonic structure|3 - female reproductive system|3 - germ layer|3 - whole body|3 - organism form|4 - reproductive system|4 - embryo|4 - animal|4 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"umbilical_cord\", \n",
      "        \"name\": \"umbilical_cord\", \n",
      "        \"cell_of_origin\": \"umbilical cord\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_umbilical_cord_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical vein endothelial cell|1 - umbilical vein endothelium|1 - vascular endothelial cell|2 - umbilical vein|2 - endothelium|2 - epithelium|2 - vascular endothelium|2 - endothelial cell|3 - umbilical cord|3 - integument|3 - blood vessel endothelium|3 - heart|4 - placenta|4 - connecting stalk|4 - fetus|4 - whole body|4 - blood vessel|4 - cardiovascular system|5 - internal female genital organ|5 - mesoderm|5 - embryonic structure|5 - animal|5 - vascular system|6 - female reproductive system|6 - germ layer|6 - organism form|6 - tissues, cell types and enzyme sources|7 - reproductive system|7 - embryo|8 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"name\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"cell_of_origin\": \"umbilical vein endothelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_umbilical_vein_endothelial_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - umbilical vein endothelial cell|1 - umbilical vein endothelium|1 - vascular endothelial cell|2 - umbilical vein|2 - endothelium|2 - epithelium|2 - vascular endothelium|2 - endothelial cell|3 - umbilical cord|3 - integument|3 - blood vessel endothelium|3 - heart|4 - placenta|4 - connecting stalk|4 - fetus|4 - whole body|4 - blood vessel|4 - cardiovascular system|5 - internal female genital organ|5 - mesoderm|5 - embryonic structure|5 - animal|5 - vascular system|6 - female reproductive system|6 - germ layer|6 - organism form|6 - tissues, cell types and enzyme sources|7 - reproductive system|7 - embryo|8 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"name\": \"umbilical_vein_endothelial_cell\", \n",
      "        \"cell_of_origin\": \"umbilical vein endothelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_umbilical_vein_endothelial_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder|1 - bladder|1 - urinary system|1 - urinary tract|2 - viscus|2 - urogenital system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"urinary_bladder\", \n",
      "        \"name\": \"urinary_bladder\", \n",
      "        \"cell_of_origin\": \"urinary bladder\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_urinary_bladder_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - urinary bladder|1 - bladder|1 - urinary system|1 - urinary tract|2 - viscus|2 - urogenital system|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"urinary_bladder\", \n",
      "        \"name\": \"urinary_bladder\", \n",
      "        \"cell_of_origin\": \"urinary bladder\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_urinary_bladder_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uroepithelium|1 - epithelium|1 - urinary tract|2 - integument|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uroepithelium\", \n",
      "        \"name\": \"uroepithelium\", \n",
      "        \"cell_of_origin\": \"uroepithelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uroepithelium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uroepithelium|1 - epithelium|1 - urinary tract|2 - integument|2 - viscus|3 - whole body|4 - animal|5 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uroepithelium\", \n",
      "        \"name\": \"uroepithelium\", \n",
      "        \"cell_of_origin\": \"uroepithelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uroepithelium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine cervix|1 - uterus|2 - internal female genital organ|3 - female reproductive system|4 - reproductive system|5 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterine_cervix\", \n",
      "        \"name\": \"uterine_cervix\", \n",
      "        \"cell_of_origin\": \"uterine cervix\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterine_cervix_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine cervix|1 - uterus|2 - internal female genital organ|3 - female reproductive system|4 - reproductive system|5 - urogenital system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterine_cervix\", \n",
      "        \"name\": \"uterine_cervix\", \n",
      "        \"cell_of_origin\": \"uterine cervix\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterine_cervix_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine endometrium|1 - mucosa|1 - uterus|2 - integument|2 - internal female genital organ|3 - whole body|3 - female reproductive system|4 - animal|4 - reproductive system|5 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterine_endometrium\", \n",
      "        \"name\": \"uterine_endometrium\", \n",
      "        \"cell_of_origin\": \"uterine endometrium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterine_endometrium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterine endometrium|1 - mucosa|1 - uterus|2 - integument|2 - internal female genital organ|3 - whole body|3 - female reproductive system|4 - animal|4 - reproductive system|5 - tissues, cell types and enzyme sources|5 - urogenital system\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterine_endometrium\", \n",
      "        \"name\": \"uterine_endometrium\", \n",
      "        \"cell_of_origin\": \"uterine endometrium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterine_endometrium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterus|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterus\", \n",
      "        \"name\": \"uterus\", \n",
      "        \"cell_of_origin\": \"uterus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterus_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - uterus|1 - internal female genital organ|2 - female reproductive system|3 - reproductive system|4 - urogenital system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"uterus\", \n",
      "        \"name\": \"uterus\", \n",
      "        \"cell_of_origin\": \"uterus\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_uterus_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelial cell|1 - vascular endothelium|1 - endothelial cell|2 - endothelium|2 - blood vessel endothelium|2 - heart|3 - epithelium|3 - blood vessel|3 - cardiovascular system|4 - integument|4 - vascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vascular_endothelial_cell\", \n",
      "        \"name\": \"vascular_endothelial_cell\", \n",
      "        \"cell_of_origin\": \"vascular endothelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vascular_endothelial_cell_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelial cell|1 - vascular endothelium|1 - endothelial cell|2 - endothelium|2 - blood vessel endothelium|2 - heart|3 - epithelium|3 - blood vessel|3 - cardiovascular system|4 - integument|4 - vascular system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vascular_endothelial_cell\", \n",
      "        \"name\": \"vascular_endothelial_cell\", \n",
      "        \"cell_of_origin\": \"vascular endothelial cell\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vascular_endothelial_cell_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelium|1 - endothelium|2 - epithelium|3 - integument|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vascular_endothelium\", \n",
      "        \"name\": \"vascular_endothelium\", \n",
      "        \"cell_of_origin\": \"vascular endothelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vascular_endothelium_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vascular endothelium|1 - endothelium|2 - epithelium|3 - integument|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vascular_endothelium\", \n",
      "        \"name\": \"vascular_endothelium\", \n",
      "        \"cell_of_origin\": \"vascular endothelium\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vascular_endothelium_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Binary Call For Known Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vermiform appendix|1 - cecum|2 - large intestine|2 - ileocecum|3 - intestine|4 - gastrointestinal tract|5 - alimentary canal|6 - viscus|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vermiform_appendix\", \n",
      "        \"name\": \"vermiform_appendix\", \n",
      "        \"cell_of_origin\": \"vermiform appendix\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vermiform_appendix_KnownFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"FLOAT\", \n",
      "      \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": \"0 - vermiform appendix|1 - cecum|2 - large intestine|2 - ileocecum|3 - intestine|4 - gastrointestinal tract|5 - alimentary canal|6 - viscus|7 - whole body|8 - animal|9 - tissues, cell types and enzyme sources\", \n",
      "        \"study\": {\n",
      "          \"name\": \"GIANT\", \n",
      "          \"description\": \"Genetic Investigation of ANthropometric Traits\\n        The Genetic Investigation of ANthropometric Traits (GIANT) consortium is an international collaboration that seeks to identify genetic loci that modulate human body size and shape, including height and measures of obesity. The GIANT consortium is a collaboration between investigators from many different groups, institutions, countries, and studies, and the results represent their combined efforts. The primary approach has been meta-analysis of genome-wide association data and other large-scale genetic data sets. Anthropometric traits that have been studied by GIANT include body mass index (BMI), height, and traits related to waist circumference (such as waist-hip ratio adjusted for BMI, or WHRadjBMI). Thus far, the GIANT consortium has identified common genetic variants at hundreds of loci that are associated with anthropometric traits.\\n\\n        http://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium\\n        \"\n",
      "        }, \n",
      "        \"description\": \"vermiform_appendix\", \n",
      "        \"name\": \"vermiform_appendix\", \n",
      "        \"cell_of_origin\": \"vermiform appendix\"\n",
      "      }, \n",
      "      \"name\": \"GIANT_vermiform_appendix_ProbabilityOfFunctionalInteraction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"BOOLEAN\", \n",
      "      \"interactions_type\": \"Boolean Flag For Interaction\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": null, \n",
      "        \"study\": {\n",
      "          \"name\": \"BioGRID\", \n",
      "          \"description\": \" Biological General Repository for Interaction Datasets\\n        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\\n\\n        https://thebiogrid.org/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Default\", \n",
      "        \"name\": \"Default\", \n",
      "        \"cell_of_origin\": null\n",
      "      }, \n",
      "      \"name\": \"BioGRID_Interaction\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"STRING\", \n",
      "      \"interactions_type\": \"Experimental System Used\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": null, \n",
      "        \"study\": {\n",
      "          \"name\": \"BioGRID\", \n",
      "          \"description\": \" Biological General Repository for Interaction Datasets\\n        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\\n\\n        https://thebiogrid.org/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Default\", \n",
      "        \"name\": \"Default\", \n",
      "        \"cell_of_origin\": null\n",
      "      }, \n",
      "      \"name\": \"BioGRID_Experimental_System\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"STRING\", \n",
      "      \"interactions_type\": \"Experimental System Type\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": null, \n",
      "        \"study\": {\n",
      "          \"name\": \"BioGRID\", \n",
      "          \"description\": \" Biological General Repository for Interaction Datasets\\n        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\\n\\n        https://thebiogrid.org/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Default\", \n",
      "        \"name\": \"Default\", \n",
      "        \"cell_of_origin\": null\n",
      "      }, \n",
      "      \"name\": \"BioGRID_Experimental_System_Type\"\n",
      "    }, \n",
      "    {\n",
      "      \"datatype\": \"STRING\", \n",
      "      \"interactions_type\": \"High or Low Throughput Measurement\", \n",
      "      \"substudy\": {\n",
      "        \"tissue_hierarchy\": null, \n",
      "        \"study\": {\n",
      "          \"name\": \"BioGRID\", \n",
      "          \"description\": \" Biological General Repository for Interaction Datasets\\n        BioGRID is an interaction repository with data compiled through comprehensive curation efforts.\\n\\n        https://thebiogrid.org/\\n        \"\n",
      "        }, \n",
      "        \"description\": \"Default\", \n",
      "        \"name\": \"Default\", \n",
      "        \"cell_of_origin\": null\n",
      "      }, \n",
      "      \"name\": \"BioGRID_Throughput\"\n",
      "    }\n",
      "  ], \n",
      "  \"name\": \"BigGIM_70_v1\"\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    table = get('/metadata/table/%s' %  (default_table,))\n",
    "    jprint(table)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "### First 10 columns in default table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Column name: TCGA_ACC_Correlation\n",
      "Column interaction: Spearman Rank Correlation Coefficient\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_ACC_Pvalue\n",
      "Column interaction: P-value (-log10)\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_BLCA_Correlation\n",
      "Column interaction: Spearman Rank Correlation Coefficient\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_BLCA_Pvalue\n",
      "Column interaction: P-value (-log10)\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_BRCA_Correlation\n",
      "Column interaction: Spearman Rank Correlation Coefficient\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_BRCA_Pvalue\n",
      "Column interaction: P-value (-log10)\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_CESC_Correlation\n",
      "Column interaction: Spearman Rank Correlation Coefficient\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_CESC_Pvalue\n",
      "Column interaction: P-value (-log10)\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_CHOL_Correlation\n",
      "Column interaction: Spearman Rank Correlation Coefficient\n",
      "Column data type: FLOAT\n",
      "\n",
      "Column name: TCGA_CHOL_Pvalue\n",
      "Column interaction: P-value (-log10)\n",
      "Column data type: FLOAT\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for column in table['columns'][:10]:\n",
    "    print(\"Column name: %s\" % column['name'])\n",
    "    print(\"Column interaction: %s\" % column['interactions_type'])\n",
    "    print(\"Column data type: %s\" % column[\"datatype\"])\n",
    "    print"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_column\"></a>\n",
    "### Get a single column"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"datatype\": \"FLOAT\", \n",
      "  \"interactions_type\": \"Spearman Rank Correlation Coefficient\", \n",
      "  \"substudy\": {\n",
      "    \"tissue_hierarchy\": \"0 - adrenocortical carcinoma cell|1 - adrenal cortex|2 - adrenal gland|3 - viscus|3 - endocrine gland|4 - whole body|4 - gland|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "    \"study\": {\n",
      "      \"name\": \"TCGA\", \n",
      "      \"description\": \" The Cancer Genome Atlas\\n        The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. The TCGA dataset, comprising more than two petabytes of genomic data, has been made publically available, and this genomic information helps the cancer research community to improve the prevention, diagnosis, and treatment of cancer.\\n\\n        https://cancergenome.nih.gov/\\n        \"\n",
      "    }, \n",
      "    \"description\": \"Adrenocortical carcinoma\", \n",
      "    \"name\": \"ACC\", \n",
      "    \"cell_of_origin\": \"adrenocortical carcinoma cell\"\n",
      "  }, \n",
      "  \"name\": \"TCGA_ACC_Correlation\"\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "try:\n",
    "    column = get('/metadata/table/%s/column/%s' %  (default_table,'TCGA_ACC_Correlation'))\n",
    "    jprint(column)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_tissues\"></a>\n",
    "### List all tissues"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"tissues\": [\n",
      "    \"B_lymphocyte\", \n",
      "    \"B_lymphoma_cell_line\", \n",
      "    \"T_lymphocyte\", \n",
      "    \"acute_myeloid_leukemia_cell\", \n",
      "    \"adenocarcinoma_cell\", \n",
      "    \"adipose_tissue\", \n",
      "    \"adrenal_cortex\", \n",
      "    \"adrenal_gland\", \n",
      "    \"adrenal_gland_cancer_cell\", \n",
      "    \"adrenocortical_carcinoma_cell\", \n",
      "    \"adult_stem_cell\", \n",
      "    \"alimentary_canal\", \n",
      "    \"amygdala\", \n",
      "    \"animal\", \n",
      "    \"aorta\", \n",
      "    \"artery\", \n",
      "    \"astroblast\", \n",
      "    \"astrocyte\", \n",
      "    \"astrocytoma_cell\", \n",
      "    \"astroglia\", \n",
      "    \"astroglial_cell\", \n",
      "    \"avian_pallium\", \n",
      "    \"basal_ganglion\", \n",
      "    \"basophil\", \n",
      "    \"bile_duct\", \n",
      "    \"bile_duct_epithelium\", \n",
      "    \"biliary_epithelium\", \n",
      "    \"bladder\", \n",
      "    \"blast_cell\", \n",
      "    \"blastomere\", \n",
      "    \"blastula\", \n",
      "    \"blood\", \n",
      "    \"blood_cancer_cell\", \n",
      "    \"blood_plasma\", \n",
      "    \"blood_platelet\", \n",
      "    \"blood_vessel\", \n",
      "    \"blood_vessel_endothelium\", \n",
      "    \"bone\", \n",
      "    \"bone_cancer_cell\", \n",
      "    \"bone_marrow\", \n",
      "    \"bone_marrow_cell\", \n",
      "    \"brain\", \n",
      "    \"brain_stem\", \n",
      "    \"brain_ventricle\", \n",
      "    \"breast\", \n",
      "    \"breast_cancer_cell\", \n",
      "    \"breast_epithelium\", \n",
      "    \"bronchial_epithelial_cell\", \n",
      "    \"bronchial_epithelium\", \n",
      "    \"bronchoalveolar_system\", \n",
      "    \"bronchogenic_carcinoma_cell\", \n",
      "    \"bronchus\", \n",
      "    \"capsular_epithelium\", \n",
      "    \"carcinoid_cell\", \n",
      "    \"carcinoma_cell\", \n",
      "    \"cardiac_muscle\", \n",
      "    \"cardiovascular_system\", \n",
      "    \"cartilage\", \n",
      "    \"caudate_nucleus\", \n",
      "    \"caudate_putamen\", \n",
      "    \"cecum\", \n",
      "    \"cell_culture\", \n",
      "    \"central_nervous_system\", \n",
      "    \"cerebellar_cortex\", \n",
      "    \"cerebellum\", \n",
      "    \"cerebral_cortex\", \n",
      "    \"cerebral_hemisphere\", \n",
      "    \"cerebral_lobe\", \n",
      "    \"cervical_adenocarcinoma_cell\", \n",
      "    \"cervical_carcinoma_cell\", \n",
      "    \"cervical_squamous_cell_carcinoma_cell\", \n",
      "    \"cholangiocarcinoma_cell\", \n",
      "    \"chondrocyte\", \n",
      "    \"choroid\", \n",
      "    \"chromophobe_renal_cell_carcinoma_cell\", \n",
      "    \"cochlea\", \n",
      "    \"coelom\", \n",
      "    \"colon\", \n",
      "    \"colonic_cancer_cell\", \n",
      "    \"colorectal_adenocarcinoma_cell\", \n",
      "    \"colorectum\", \n",
      "    \"connecting_stalk\", \n",
      "    \"connective_tissue\", \n",
      "    \"cornea\", \n",
      "    \"corpus_callosum\", \n",
      "    \"corpus_luteum\", \n",
      "    \"corpus_striatum\", \n",
      "    \"culture_condition\", \n",
      "    \"culture_condition_CD8__cell\", \n",
      "    \"culture_condition_antigen_presenting_cell\", \n",
      "    \"dendritic_cell\", \n",
      "    \"dentate_gyrus\", \n",
      "    \"diencephalon\", \n",
      "    \"diffuse_large_B_cell_lymphoma_cell\", \n",
      "    \"digestive_gland\", \n",
      "    \"duodenum\", \n",
      "    \"ear\", \n",
      "    \"ectoderm\", \n",
      "    \"embryo\", \n",
      "    \"embryonic_structure\", \n",
      "    \"endocrine_gland\", \n",
      "    \"endocrine_pancreas\", \n",
      "    \"endometrioid_carcinoma_cell\", \n",
      "    \"endothelial_cell\", \n",
      "    \"endothelium\", \n",
      "    \"eosinophil\", \n",
      "    \"epidermal_cell\", \n",
      "    \"epidermis\", \n",
      "    \"epithelial_cell\", \n",
      "    \"epithelioma_cell\", \n",
      "    \"epithelium\", \n",
      "    \"esophageal_cancer_cell\", \n",
      "    \"esophagus\", \n",
      "    \"excretory_gland\", \n",
      "    \"exocrine_gland\", \n",
      "    \"exocrine_pancreas\", \n",
      "    \"eye\", \n",
      "    \"female_reproductive_gland\", \n",
      "    \"female_reproductive_system\", \n",
      "    \"fetus\", \n",
      "    \"fibroblast\", \n",
      "    \"forebrain\", \n",
      "    \"frontal_lobe\", \n",
      "    \"gametocyte\", \n",
      "    \"ganglion\", \n",
      "    \"gastric_adenocarcinoma_cell\", \n",
      "    \"gastric_cancer_cell\", \n",
      "    \"gastrointestinal_cancer_cell\", \n",
      "    \"gastrointestinal_tract\", \n",
      "    \"germ_cell\", \n",
      "    \"germ_layer\", \n",
      "    \"gland\", \n",
      "    \"glia\", \n",
      "    \"glial_cell\", \n",
      "    \"glioblastoma_cell\", \n",
      "    \"glioma_cell\", \n",
      "    \"glomerular_epithelium\", \n",
      "    \"gonad\", \n",
      "    \"granulocyte\", \n",
      "    \"hair_follicle\", \n",
      "    \"head\", \n",
      "    \"head_and_neck_cancer_cell\", \n",
      "    \"heart\", \n",
      "    \"hematopoietic_cell\", \n",
      "    \"hematopoietic_stem_cell\", \n",
      "    \"hematopoietic_system\", \n",
      "    \"hepatocyte\", \n",
      "    \"hindbrain\", \n",
      "    \"hippocampus\", \n",
      "    \"hypophysis\", \n",
      "    \"hypothalamus\", \n",
      "    \"ileocecum\", \n",
      "    \"ileum\", \n",
      "    \"immune_system\", \n",
      "    \"inner_ear\", \n",
      "    \"integument\", \n",
      "    \"internal_female_genital_organ\", \n",
      "    \"internal_male_genital_organ\", \n",
      "    \"intestinal_mucosa\", \n",
      "    \"intestine\", \n",
      "    \"jejunum\", \n",
      "    \"keratinocyte\", \n",
      "    \"kidney\", \n",
      "    \"kidney_cancer_cell\", \n",
      "    \"large_granular_lymphocyte\", \n",
      "    \"large_intestine\", \n",
      "    \"lateral_ventricle\", \n",
      "    \"lens\", \n",
      "    \"leukemia_cell\", \n",
      "    \"leukocyte\", \n",
      "    \"limbic_system\", \n",
      "    \"liver\", \n",
      "    \"liver_cancer_cell\", \n",
      "    \"locus_ceruleus\", \n",
      "    \"lung\", \n",
      "    \"lung_adenocarcinoma_cell\", \n",
      "    \"lung_cancer_cell\", \n",
      "    \"lung_squamous_cell_carcinoma_cell\", \n",
      "    \"lymph_node\", \n",
      "    \"lymphocyte\", \n",
      "    \"lymphoid_cell\", \n",
      "    \"lymphoid_cell_line\", \n",
      "    \"lymphoid_tissue\", \n",
      "    \"lymphoma_cell\", \n",
      "    \"lymphoma_cell_line\", \n",
      "    \"macroglia\", \n",
      "    \"macrophage\", \n",
      "    \"male_reproductive_gland\", \n",
      "    \"male_reproductive_system\", \n",
      "    \"mammary_epithelium\", \n",
      "    \"mammary_gland\", \n",
      "    \"marrow_cell\", \n",
      "    \"mast_cell\", \n",
      "    \"mature_ovarian_follicle\", \n",
      "    \"medulla_oblongata\", \n",
      "    \"megakaryoblast\", \n",
      "    \"megakaryocyte\", \n",
      "    \"melanocyte\", \n",
      "    \"melanoma_cell\", \n",
      "    \"mesoderm\", \n",
      "    \"mesothelioma_cell\", \n",
      "    \"mesothelium\", \n",
      "    \"metencephalon\", \n",
      "    \"midbrain\", \n",
      "    \"monocyte\", \n",
      "    \"mononuclear_cell\", \n",
      "    \"mononuclear_phagocyte\", \n",
      "    \"morula\", \n",
      "    \"mouth\", \n",
      "    \"mucosa\", \n",
      "    \"muscle\", \n",
      "    \"muscular_system\", \n",
      "    \"myelencephalon\", \n",
      "    \"myeloid_leukemia_cell\", \n",
      "    \"myeloid_progenitor_cell\", \n",
      "    \"myometrium\", \n",
      "    \"natural_killer_cell\", \n",
      "    \"neck\", \n",
      "    \"neostriatum\", \n",
      "    \"nephron\", \n",
      "    \"nerve\", \n",
      "    \"nervous_system\", \n",
      "    \"neural_crest\", \n",
      "    \"neural_crest_cell\", \n",
      "    \"neural_plate\", \n",
      "    \"neuroblast\", \n",
      "    \"neuroendocrine_cell\", \n",
      "    \"neuroendocrine_tumor_cell\", \n",
      "    \"neuron\", \n",
      "    \"neutrophil\", \n",
      "    \"non_small_cell_lung_cancer_cell\", \n",
      "    \"nucleus_accumbens\", \n",
      "    \"nucleus_lentiformis\", \n",
      "    \"null_cell\", \n",
      "    \"occipital_lobe\", \n",
      "    \"occipital_pole\", \n",
      "    \"organism_form\", \n",
      "    \"osteoblast\", \n",
      "    \"osteogenic_cell\", \n",
      "    \"other_source\", \n",
      "    \"ovarian_follicle\", \n",
      "    \"ovarian_serous_carcinoma_cell\", \n",
      "    \"ovary\", \n",
      "    \"ovary_cancer_cell\", \n",
      "    \"oviduct\", \n",
      "    \"pancreas\", \n",
      "    \"pancreatic_duct\", \n",
      "    \"pancreatic_ductal_adenocarcinoma_cell\", \n",
      "    \"pancreatic_ductal_carcinoma_cell\", \n",
      "    \"pancreatic_islet\", \n",
      "    \"papillary_renal_cell_carcinoma_cell\", \n",
      "    \"parietal_lobe\", \n",
      "    \"periosteum\", \n",
      "    \"peripheral_blood\", \n",
      "    \"peripheral_nervous_system\", \n",
      "    \"phagocyte\", \n",
      "    \"pheochromocytoma_cell\", \n",
      "    \"placenta\", \n",
      "    \"podocyte\", \n",
      "    \"pons\", \n",
      "    \"pre_T_lymphocyte\", \n",
      "    \"preosteoblast\", \n",
      "    \"primary_spermatocyte\", \n",
      "    \"prostate_adenocarcinoma_cell\", \n",
      "    \"prostate_gland\", \n",
      "    \"rectal_adenocarcinoma_cell\", \n",
      "    \"rectal_cancer_cell\", \n",
      "    \"rectum\", \n",
      "    \"renal_cell_carcinoma_cell\", \n",
      "    \"renal_clear_cell\", \n",
      "    \"renal_corpuscle\", \n",
      "    \"renal_glomerular_capsule\", \n",
      "    \"renal_glomerulus\", \n",
      "    \"renal_tubule\", \n",
      "    \"reproductive_system\", \n",
      "    \"respiratory_epithelium\", \n",
      "    \"respiratory_system\", \n",
      "    \"retina\", \n",
      "    \"salivary_gland\", \n",
      "    \"sarcoma_cell\", \n",
      "    \"secondary_spermatocyte\", \n",
      "    \"semen\", \n",
      "    \"sense_organ\", \n",
      "    \"serum\", \n",
      "    \"skeletal_muscle\", \n",
      "    \"skeletal_muscle_cancer_cell\", \n",
      "    \"skeletal_muscle_cell\", \n",
      "    \"skeletal_system\", \n",
      "    \"skin\", \n",
      "    \"skin_cancer_cell\", \n",
      "    \"skin_fibroblast\", \n",
      "    \"small_intestine\", \n",
      "    \"smooth_muscle\", \n",
      "    \"spermatid\", \n",
      "    \"spermatocyte\", \n",
      "    \"spermatogonium\", \n",
      "    \"spinal_cord\", \n",
      "    \"spleen\", \n",
      "    \"squamous_cell_carcinoma_cell\", \n",
      "    \"squamous_epithelium\", \n",
      "    \"stomach\", \n",
      "    \"stratum_spinosum\", \n",
      "    \"substantia_nigra\", \n",
      "    \"subthalamic_nucleus\", \n",
      "    \"tear_gland\", \n",
      "    \"telencephalon\", \n",
      "    \"temporal_lobe\", \n",
      "    \"testicular_cancer_cell\", \n",
      "    \"testis\", \n",
      "    \"thalamus\", \n",
      "    \"thorax\", \n",
      "    \"throat\", \n",
      "    \"thymic_cancer_cell\", \n",
      "    \"thymocyte\", \n",
      "    \"thymoma_cell\", \n",
      "    \"thymus\", \n",
      "    \"thyroid_cancer_cell\", \n",
      "    \"thyroid_gland\", \n",
      "    \"tissues__cell_types_and_enzyme_sources\", \n",
      "    \"tonsil\", \n",
      "    \"tooth\", \n",
      "    \"trachea\", \n",
      "    \"trophoblast\", \n",
      "    \"trunk\", \n",
      "    \"umbilical_cord\", \n",
      "    \"umbilical_vein\", \n",
      "    \"umbilical_vein_endothelial_cell\", \n",
      "    \"umbilical_vein_endothelium\", \n",
      "    \"urinary_bladder\", \n",
      "    \"urinary_bladder_cancer_cell\", \n",
      "    \"urinary_system\", \n",
      "    \"urinary_tract\", \n",
      "    \"uroepithelium\", \n",
      "    \"urogenital_system\", \n",
      "    \"uterine_cancer_cell\", \n",
      "    \"uterine_cervix\", \n",
      "    \"uterine_endometrium\", \n",
      "    \"uterine_sarcoma_cell\", \n",
      "    \"uterus\", \n",
      "    \"uveal_melanoma_cell\", \n",
      "    \"vascular_endothelial_cell\", \n",
      "    \"vascular_endothelium\", \n",
      "    \"vascular_system\", \n",
      "    \"vermiform_appendix\", \n",
      "    \"vertebrate_muscular_system\", \n",
      "    \"vestibular_labyrinth\", \n",
      "    \"vestibular_system\", \n",
      "    \"viscus\", \n",
      "    \"whole_body\", \n",
      "    \"zygote\"\n",
      "  ]\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "tissues = get('metadata/tissue')\n",
    "jprint(tissues)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"metadata_tissue\"></a>\n",
    "### Get single tissue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"tissue\": \"lymphoid_cell\", \n",
      "  \"substudies\": [\n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - natural killer cell|1 - null cell|1 - large granular lymphocyte|2 - lymphocyte|3 - leukocyte|3 - lymphoid cell|4 - hematopoietic cell|4 - lymphoid tissue|5 - blood|5 - immune system|5 - hematopoietic system|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"natural_killer_cell\", \n",
      "      \"name\": \"natural_killer_cell\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"natural killer cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - B-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"b_lymphocyte\", \n",
      "      \"name\": \"b_lymphocyte\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"B-lymphocyte\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - T-lymphocyte|1 - lymphocyte|2 - leukocyte|2 - lymphoid cell|3 - hematopoietic cell|3 - lymphoid tissue|4 - blood|4 - immune system|4 - hematopoietic system|5 - whole body|6 - animal|7 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"t_lymphocyte\", \n",
      "      \"name\": \"t_lymphocyte\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"T-lymphocyte\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - thymocyte|1 - thymus|1 - pre-T-lymphocyte|2 - immune system|2 - thorax|2 - endocrine gland|2 - lymphoid tissue|2 - T-lymphocyte|3 - whole body|3 - trunk|3 - gland|3 - hematopoietic system|3 - lymphocyte|4 - animal|4 - leukocyte|4 - lymphoid cell|5 - tissues, cell types and enzyme sources|5 - hematopoietic cell|6 - blood\", \n",
      "      \"description\": \"thymocyte\", \n",
      "      \"name\": \"thymocyte\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_thymocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"thymocyte\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - diffuse large B-cell lymphoma cell|1 - B-lymphoma cell line|2 - lymphoma cell line|3 - lymphoid cell line|3 - lymphoma cell|4 - lymphoid cell|4 - blood cancer cell|4 - lymphoid tissue|5 - hematopoietic cell|5 - immune system|5 - hematopoietic system|6 - blood|6 - whole body|7 - animal|8 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"Lymphoid Neoplasm Diffuse Large B-cell Lymphoma\", \n",
      "      \"name\": \"DLBC\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Correlation\", \n",
      "          \"interactions_type\": \"Spearman Rank Correlation Coefficient\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"TCGA_DLBC_Pvalue\", \n",
      "          \"interactions_type\": \"P-value (-log10)\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"diffuse large B-cell lymphoma cell\"\n",
      "    }, \n",
      "    {\n",
      "      \"tissue_hierarchy\": \"0 - lymphocyte|1 - leukocyte|1 - lymphoid cell|2 - hematopoietic cell|2 - lymphoid tissue|3 - blood|3 - immune system|3 - hematopoietic system|4 - whole body|5 - animal|6 - tissues, cell types and enzyme sources\", \n",
      "      \"description\": \"lymphocyte\", \n",
      "      \"name\": \"lymphocyte\", \n",
      "      \"columns\": [\n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"BigGIM_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"CRC_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_70_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_80_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_90_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_KnownFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Binary Call For Known Functional Interaction\"\n",
      "        }, \n",
      "        {\n",
      "          \"datatype\": \"FLOAT\", \n",
      "          \"table\": {\n",
      "            \"name\": \"FA_v1\"\n",
      "          }, \n",
      "          \"name\": \"GIANT_lymphocyte_ProbabilityOfFunctionalInteraction\", \n",
      "          \"interactions_type\": \"Bayesian Posterior Probability Of Functional Interaction\"\n",
      "        }\n",
      "      ], \n",
      "      \"cell_of_origin\": \"lymphocyte\"\n",
      "    }\n",
      "  ]\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "substudies = get('metadata/tissue/%s' % (\"lymphoid_cell\",))\n",
    "jprint(substudies)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"query_examples\"></a>\n",
    "## Query examples"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"query_simple\"></a>\n",
    "### Simple predefined query"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Query request:\n",
      "{\n",
      "  \"restriction_join\": \"union\", \n",
      "  \"limit\": 10000, \n",
      "  \"restriction_gt\": \"TCGA_GBM_Correlation,.2, GTEx_Brain_Correlation,.2\", \n",
      "  \"restriction_lt\": \"TCGA_GBM_Pvalue,.05, GTEx_Brain_Pvalue,.01\", \n",
      "  \"table\": \"BigGIM_70_v1\", \n",
      "  \"ids2\": \"5111,6996,57697,6815,889,7112,2176,1019,5888,5706,3333,1111,112,3333\", \n",
      "  \"columns\": \"TCGA_GBM_Correlation,TCGA_GBM_Pvalue,GTEx_Brain_Correlation,GTEx_Brain_Pvalue\", \n",
      "  \"ids1\": \"5111,6996,57697,6815,889,7112,2176,1019,5888,5706,5722,1111,112,3333\"\n",
      "}\n",
      "{\n",
      "  \"status\": \"submitted\", \n",
      "  \"request_id\": \"107b518b-6e7b-4024-a664-0b0638fc9028\"\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "example_query = {\n",
    "      # The table to select from.\n",
    "      \"table\": default_table, \n",
    "      # A comma delimited list of column names to return.\n",
    "      \"columns\": \"TCGA_GBM_Correlation,TCGA_GBM_Pvalue,GTEx_Brain_Correlation,GTEx_Brain_Pvalue\", \n",
    "      # A comma delimited list of Entrez gene ids to select.\n",
    "      \"ids1\": \"5111,6996,57697,6815,889,7112,2176,1019,5888,5706,5722,1111,112,3333\",\n",
    "      # Entrez gene ids to select. If not given, the query selects any gene related to a gene in ids 1. \n",
    "      # If given, the query only selects relations that contain a gene in ids1 and a gene in ids2.\n",
    "      \"ids2\": \"5111,6996,57697,6815,889,7112,2176,1019,5888,5706,3333,1111,112,3333\", \n",
    "      # The type of join made on restrictions. Either intersect or union\n",
    "      \"restriction_join\": \"union\", \n",
    "      # The maximum number of rows to return\n",
    "      \"restriction_gt\": \"TCGA_GBM_Correlation,.2, GTEx_Brain_Correlation,.2\", \n",
    "      # A list of pairs of values column name,value with which to restrict \n",
    "      # the results of the query to rows where the value of the column is greater than the given value.\n",
    "      \"restriction_lt\": \"TCGA_GBM_Pvalue,.05, GTEx_Brain_Pvalue,.01\",\n",
    "      # The maximum number of rows to return.\n",
    "      \"limit\": 10000 \n",
    "}\n",
    "print(\"Query request:\")\n",
    "jprint(example_query)\n",
    "try:\n",
    "    query_submit = get('interactions/query', data=example_query)\n",
    "    jprint(query_submit)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"query_status\"></a>\n",
    "### Check the status of the simple predefined query"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"107b518b-6e7b-4024-a664-0b0638fc9028\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"complete\", \n",
      "  \"rows\": 54, \n",
      "  \"processed_data\": \"0B\", \n",
      "  \"request_id\": \"107b518b-6e7b-4024-a664-0b0638fc9028\", \n",
      "  \"request_uri\": [\n",
      "    \"https://storage.googleapis.com/ncats_bigquery_results/107b518b-6e7b-4024-a664-0b0638fc9028000000000000.csv\"\n",
      "  ], \n",
      "  \"size\": \"2.72 KB\"\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "import time\n",
    "try:\n",
    "    while True:\n",
    "        query_status = get('interactions/query/status/%s'% (query_submit['request_id'],))\n",
    "        jprint(query_status)\n",
    "        if query_status['status'] !='running':\n",
    "            # query has finished\n",
    "            break\n",
    "        else:\n",
    "            time.sleep(1)\n",
    "            print \"Checking again\"\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"query_results\"></a>\n",
    "### Get the results as dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GPID</th>\n",
       "      <th>Gene1</th>\n",
       "      <th>Gene2</th>\n",
       "      <th>TCGA_GBM_Correlation</th>\n",
       "      <th>TCGA_GBM_Pvalue</th>\n",
       "      <th>GTEx_Brain_Correlation</th>\n",
       "      <th>GTEx_Brain_Pvalue</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58880000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>5888</td>\n",
       "      <td>0.4545</td>\n",
       "      <td>6.74</td>\n",
       "      <td>0.6227</td>\n",
       "      <td>135.34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>58880000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>5888</td>\n",
       "      <td>0.7331</td>\n",
       "      <td>20.76</td>\n",
       "      <td>0.5651</td>\n",
       "      <td>106.40</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>57060000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>5706</td>\n",
       "      <td>0.2880</td>\n",
       "      <td>2.85</td>\n",
       "      <td>0.7209</td>\n",
       "      <td>201.65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>57060000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>5706</td>\n",
       "      <td>0.4229</td>\n",
       "      <td>5.83</td>\n",
       "      <td>0.6719</td>\n",
       "      <td>165.37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>57060000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>5706</td>\n",
       "      <td>0.4009</td>\n",
       "      <td>5.24</td>\n",
       "      <td>0.4885</td>\n",
       "      <td>75.74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>57060000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>5706</td>\n",
       "      <td>0.1942</td>\n",
       "      <td>1.47</td>\n",
       "      <td>0.7647</td>\n",
       "      <td>241.41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>69960000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>6996</td>\n",
       "      <td>0.4991</td>\n",
       "      <td>8.18</td>\n",
       "      <td>0.6653</td>\n",
       "      <td>161.01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>69960000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>6996</td>\n",
       "      <td>0.3473</td>\n",
       "      <td>3.99</td>\n",
       "      <td>0.8152</td>\n",
       "      <td>299.70</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>11110000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>1111</td>\n",
       "      <td>-0.1197</td>\n",
       "      <td>0.72</td>\n",
       "      <td>0.2608</td>\n",
       "      <td>20.30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>11110000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>1111</td>\n",
       "      <td>0.7026</td>\n",
       "      <td>18.42</td>\n",
       "      <td>-0.0678</td>\n",
       "      <td>1.79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>11110000002176</td>\n",
       "      <td>2176</td>\n",
       "      <td>1111</td>\n",
       "      <td>0.7111</td>\n",
       "      <td>19.04</td>\n",
       "      <td>-0.0246</td>\n",
       "      <td>0.42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11110000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>1111</td>\n",
       "      <td>0.5158</td>\n",
       "      <td>8.78</td>\n",
       "      <td>0.1124</td>\n",
       "      <td>4.19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>11110000005888</td>\n",
       "      <td>5888</td>\n",
       "      <td>1111</td>\n",
       "      <td>0.6530</td>\n",
       "      <td>15.20</td>\n",
       "      <td>-0.0131</td>\n",
       "      <td>0.19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>11110000005111</td>\n",
       "      <td>5111</td>\n",
       "      <td>1111</td>\n",
       "      <td>0.6174</td>\n",
       "      <td>13.23</td>\n",
       "      <td>0.1711</td>\n",
       "      <td>9.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1120000002176</td>\n",
       "      <td>2176</td>\n",
       "      <td>112</td>\n",
       "      <td>0.2329</td>\n",
       "      <td>1.98</td>\n",
       "      <td>0.6387</td>\n",
       "      <td>144.51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>1120000000889</td>\n",
       "      <td>889</td>\n",
       "      <td>112</td>\n",
       "      <td>0.1300</td>\n",
       "      <td>0.80</td>\n",
       "      <td>0.7565</td>\n",
       "      <td>233.37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>1120000005111</td>\n",
       "      <td>5111</td>\n",
       "      <td>112</td>\n",
       "      <td>-0.2945</td>\n",
       "      <td>2.96</td>\n",
       "      <td>0.5295</td>\n",
       "      <td>91.17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1120000001111</td>\n",
       "      <td>1111</td>\n",
       "      <td>112</td>\n",
       "      <td>-0.0028</td>\n",
       "      <td>0.01</td>\n",
       "      <td>-0.2528</td>\n",
       "      <td>19.09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1120000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>112</td>\n",
       "      <td>-0.0171</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.7397</td>\n",
       "      <td>217.76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1120000005888</td>\n",
       "      <td>5888</td>\n",
       "      <td>112</td>\n",
       "      <td>0.0019</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.5560</td>\n",
       "      <td>102.35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>1120000001019</td>\n",
       "      <td>1019</td>\n",
       "      <td>112</td>\n",
       "      <td>-0.2191</td>\n",
       "      <td>1.79</td>\n",
       "      <td>0.4109</td>\n",
       "      <td>51.74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>8890000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>889</td>\n",
       "      <td>0.0360</td>\n",
       "      <td>0.16</td>\n",
       "      <td>0.8080</td>\n",
       "      <td>290.39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>8890000005111</td>\n",
       "      <td>5111</td>\n",
       "      <td>889</td>\n",
       "      <td>0.0933</td>\n",
       "      <td>0.51</td>\n",
       "      <td>0.7489</td>\n",
       "      <td>226.12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>8890000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>889</td>\n",
       "      <td>0.0584</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.8263</td>\n",
       "      <td>315.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>8890000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>889</td>\n",
       "      <td>0.0928</td>\n",
       "      <td>0.50</td>\n",
       "      <td>0.6010</td>\n",
       "      <td>123.74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>8890000005706</td>\n",
       "      <td>5706</td>\n",
       "      <td>889</td>\n",
       "      <td>-0.0382</td>\n",
       "      <td>0.17</td>\n",
       "      <td>0.8163</td>\n",
       "      <td>301.20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>8890000002176</td>\n",
       "      <td>2176</td>\n",
       "      <td>889</td>\n",
       "      <td>0.1706</td>\n",
       "      <td>1.20</td>\n",
       "      <td>0.7029</td>\n",
       "      <td>187.50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>8890000001019</td>\n",
       "      <td>1019</td>\n",
       "      <td>889</td>\n",
       "      <td>-0.0116</td>\n",
       "      <td>0.05</td>\n",
       "      <td>0.4816</td>\n",
       "      <td>73.38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>8890000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>889</td>\n",
       "      <td>0.2489</td>\n",
       "      <td>2.21</td>\n",
       "      <td>0.8262</td>\n",
       "      <td>314.88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>21760000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>2176</td>\n",
       "      <td>0.5989</td>\n",
       "      <td>12.30</td>\n",
       "      <td>0.7369</td>\n",
       "      <td>215.25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>21760000005888</td>\n",
       "      <td>5888</td>\n",
       "      <td>2176</td>\n",
       "      <td>0.5430</td>\n",
       "      <td>9.83</td>\n",
       "      <td>0.5403</td>\n",
       "      <td>95.62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>21760000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>2176</td>\n",
       "      <td>-0.2341</td>\n",
       "      <td>2.00</td>\n",
       "      <td>0.5937</td>\n",
       "      <td>120.03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>21760000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>2176</td>\n",
       "      <td>0.5478</td>\n",
       "      <td>10.02</td>\n",
       "      <td>0.5941</td>\n",
       "      <td>120.24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>21760000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>2176</td>\n",
       "      <td>0.0334</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.6541</td>\n",
       "      <td>153.85</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>21760000005111</td>\n",
       "      <td>5111</td>\n",
       "      <td>2176</td>\n",
       "      <td>0.3506</td>\n",
       "      <td>4.06</td>\n",
       "      <td>0.7075</td>\n",
       "      <td>191.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>68150000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>6815</td>\n",
       "      <td>0.2524</td>\n",
       "      <td>2.27</td>\n",
       "      <td>0.5279</td>\n",
       "      <td>90.55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>68150000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>6815</td>\n",
       "      <td>-0.1275</td>\n",
       "      <td>0.78</td>\n",
       "      <td>0.6142</td>\n",
       "      <td>130.73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>68150000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>6815</td>\n",
       "      <td>0.0446</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.5802</td>\n",
       "      <td>113.44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>71120000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>7112</td>\n",
       "      <td>0.6052</td>\n",
       "      <td>12.61</td>\n",
       "      <td>0.6201</td>\n",
       "      <td>133.93</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>51110000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.0325</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.7579</td>\n",
       "      <td>234.72</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>51110000005706</td>\n",
       "      <td>5706</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.3278</td>\n",
       "      <td>3.59</td>\n",
       "      <td>0.7406</td>\n",
       "      <td>218.54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>51110000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.6164</td>\n",
       "      <td>13.18</td>\n",
       "      <td>0.7789</td>\n",
       "      <td>256.21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>51110000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.3341</td>\n",
       "      <td>3.72</td>\n",
       "      <td>0.6808</td>\n",
       "      <td>171.41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>51110000005888</td>\n",
       "      <td>5888</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.5431</td>\n",
       "      <td>9.83</td>\n",
       "      <td>0.5510</td>\n",
       "      <td>100.17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>51110000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>5111</td>\n",
       "      <td>0.2376</td>\n",
       "      <td>2.05</td>\n",
       "      <td>0.6761</td>\n",
       "      <td>168.19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>10190000006996</td>\n",
       "      <td>6996</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.3294</td>\n",
       "      <td>3.62</td>\n",
       "      <td>0.4184</td>\n",
       "      <td>53.80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>10190000005706</td>\n",
       "      <td>5706</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.2200</td>\n",
       "      <td>1.80</td>\n",
       "      <td>0.4353</td>\n",
       "      <td>58.63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>10190000005111</td>\n",
       "      <td>5111</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.4983</td>\n",
       "      <td>8.16</td>\n",
       "      <td>0.6435</td>\n",
       "      <td>147.39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>10190000006815</td>\n",
       "      <td>6815</td>\n",
       "      <td>1019</td>\n",
       "      <td>-0.1305</td>\n",
       "      <td>0.81</td>\n",
       "      <td>0.5813</td>\n",
       "      <td>113.97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>10190000057697</td>\n",
       "      <td>57697</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.2391</td>\n",
       "      <td>2.07</td>\n",
       "      <td>0.3888</td>\n",
       "      <td>45.98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>10190000005888</td>\n",
       "      <td>5888</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.5021</td>\n",
       "      <td>8.29</td>\n",
       "      <td>0.3958</td>\n",
       "      <td>47.76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>10190000001111</td>\n",
       "      <td>1111</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.4050</td>\n",
       "      <td>5.35</td>\n",
       "      <td>0.2117</td>\n",
       "      <td>13.49</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>10190000007112</td>\n",
       "      <td>7112</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.5341</td>\n",
       "      <td>9.48</td>\n",
       "      <td>0.4414</td>\n",
       "      <td>60.44</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>10190000002176</td>\n",
       "      <td>2176</td>\n",
       "      <td>1019</td>\n",
       "      <td>0.3024</td>\n",
       "      <td>3.10</td>\n",
       "      <td>0.5253</td>\n",
       "      <td>89.52</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "              GPID  Gene1  Gene2  TCGA_GBM_Correlation  TCGA_GBM_Pvalue  \\\n",
       "0   58880000057697  57697   5888                0.4545             6.74   \n",
       "1   58880000007112   7112   5888                0.7331            20.76   \n",
       "2   57060000057697  57697   5706                0.2880             2.85   \n",
       "3   57060000006996   6996   5706                0.4229             5.83   \n",
       "4   57060000006815   6815   5706                0.4009             5.24   \n",
       "5   57060000007112   7112   5706                0.1942             1.47   \n",
       "6   69960000007112   7112   6996                0.4991             8.18   \n",
       "7   69960000057697  57697   6996                0.3473             3.99   \n",
       "8   11110000006815   6815   1111               -0.1197             0.72   \n",
       "9   11110000007112   7112   1111                0.7026            18.42   \n",
       "10  11110000002176   2176   1111                0.7111            19.04   \n",
       "11  11110000057697  57697   1111                0.5158             8.78   \n",
       "12  11110000005888   5888   1111                0.6530            15.20   \n",
       "13  11110000005111   5111   1111                0.6174            13.23   \n",
       "14   1120000002176   2176    112                0.2329             1.98   \n",
       "15   1120000000889    889    112                0.1300             0.80   \n",
       "16   1120000005111   5111    112               -0.2945             2.96   \n",
       "17   1120000001111   1111    112               -0.0028             0.01   \n",
       "18   1120000007112   7112    112               -0.0171             0.07   \n",
       "19   1120000005888   5888    112                0.0019             0.01   \n",
       "20   1120000001019   1019    112               -0.2191             1.79   \n",
       "21   8890000006996   6996    889                0.0360             0.16   \n",
       "22   8890000005111   5111    889                0.0933             0.51   \n",
       "23   8890000007112   7112    889                0.0584             0.28   \n",
       "24   8890000006815   6815    889                0.0928             0.50   \n",
       "25   8890000005706   5706    889               -0.0382             0.17   \n",
       "26   8890000002176   2176    889                0.1706             1.20   \n",
       "27   8890000001019   1019    889               -0.0116             0.05   \n",
       "28   8890000057697  57697    889                0.2489             2.21   \n",
       "29  21760000007112   7112   2176                0.5989            12.30   \n",
       "30  21760000005888   5888   2176                0.5430             9.83   \n",
       "31  21760000006815   6815   2176               -0.2341             2.00   \n",
       "32  21760000057697  57697   2176                0.5478            10.02   \n",
       "33  21760000006996   6996   2176                0.0334             0.14   \n",
       "34  21760000005111   5111   2176                0.3506             4.06   \n",
       "35  68150000057697  57697   6815                0.2524             2.27   \n",
       "36  68150000007112   7112   6815               -0.1275             0.78   \n",
       "37  68150000006996   6996   6815                0.0446             0.20   \n",
       "38  71120000057697  57697   7112                0.6052            12.61   \n",
       "39  51110000006815   6815   5111                0.0325             0.14   \n",
       "40  51110000005706   5706   5111                0.3278             3.59   \n",
       "41  51110000007112   7112   5111                0.6164            13.18   \n",
       "42  51110000057697  57697   5111                0.3341             3.72   \n",
       "43  51110000005888   5888   5111                0.5431             9.83   \n",
       "44  51110000006996   6996   5111                0.2376             2.05   \n",
       "45  10190000006996   6996   1019                0.3294             3.62   \n",
       "46  10190000005706   5706   1019                0.2200             1.80   \n",
       "47  10190000005111   5111   1019                0.4983             8.16   \n",
       "48  10190000006815   6815   1019               -0.1305             0.81   \n",
       "49  10190000057697  57697   1019                0.2391             2.07   \n",
       "50  10190000005888   5888   1019                0.5021             8.29   \n",
       "51  10190000001111   1111   1019                0.4050             5.35   \n",
       "52  10190000007112   7112   1019                0.5341             9.48   \n",
       "53  10190000002176   2176   1019                0.3024             3.10   \n",
       "\n",
       "    GTEx_Brain_Correlation  GTEx_Brain_Pvalue  \n",
       "0                   0.6227             135.34  \n",
       "1                   0.5651             106.40  \n",
       "2                   0.7209             201.65  \n",
       "3                   0.6719             165.37  \n",
       "4                   0.4885              75.74  \n",
       "5                   0.7647             241.41  \n",
       "6                   0.6653             161.01  \n",
       "7                   0.8152             299.70  \n",
       "8                   0.2608              20.30  \n",
       "9                  -0.0678               1.79  \n",
       "10                 -0.0246               0.42  \n",
       "11                  0.1124               4.19  \n",
       "12                 -0.0131               0.19  \n",
       "13                  0.1711               9.00  \n",
       "14                  0.6387             144.51  \n",
       "15                  0.7565             233.37  \n",
       "16                  0.5295              91.17  \n",
       "17                 -0.2528              19.09  \n",
       "18                  0.7397             217.76  \n",
       "19                  0.5560             102.35  \n",
       "20                  0.4109              51.74  \n",
       "21                  0.8080             290.39  \n",
       "22                  0.7489             226.12  \n",
       "23                  0.8263             315.00  \n",
       "24                  0.6010             123.74  \n",
       "25                  0.8163             301.20  \n",
       "26                  0.7029             187.50  \n",
       "27                  0.4816              73.38  \n",
       "28                  0.8262             314.88  \n",
       "29                  0.7369             215.25  \n",
       "30                  0.5403              95.62  \n",
       "31                  0.5937             120.03  \n",
       "32                  0.5941             120.24  \n",
       "33                  0.6541             153.85  \n",
       "34                  0.7075             191.00  \n",
       "35                  0.5279              90.55  \n",
       "36                  0.6142             130.73  \n",
       "37                  0.5802             113.44  \n",
       "38                  0.6201             133.93  \n",
       "39                  0.7579             234.72  \n",
       "40                  0.7406             218.54  \n",
       "41                  0.7789             256.21  \n",
       "42                  0.6808             171.41  \n",
       "43                  0.5510             100.17  \n",
       "44                  0.6761             168.19  \n",
       "45                  0.4184              53.80  \n",
       "46                  0.4353              58.63  \n",
       "47                  0.6435             147.39  \n",
       "48                  0.5813             113.97  \n",
       "49                  0.3888              45.98  \n",
       "50                  0.3958              47.76  \n",
       "51                  0.2117              13.49  \n",
       "52                  0.4414              60.44  \n",
       "53                  0.5253              89.52  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas\n",
    "result = pandas.concat(map(pandas.read_csv, query_status['request_uri']))\n",
    "result"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "<a id=\"full_example\"></a>\n",
    "# Full example\n",
    "\n",
    "Lets get relationships for a certain tissue where any tissue related correlation has a pvalue < .05\n",
    "\n",
    " Assume we are interested in issues in lymphoid tissues"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Using table BigGIM_70_v1\n",
      "Gene pairwise associations: correlation metrics from 33 TCGA tumor types, 21 GTEx tissues, functional interaction scores from 145 tissues (from GIANT), and BioGRID interactions. Containing only rows (gene pairs) where at least one of the TCGA or GTEx absolute correlations is higher than 0.7\n",
      "The constructed query.\n",
      "{\n",
      "  \"table\": \"BigGIM_70_v1\", \n",
      "  \"limit\": 1000000, \n",
      "  \"restriction_gt\": \"GTEx_Spleen_Pvalue,1.30102999566,TCGA_DLBC_Pvalue,1.30102999566,TCGA_THYM_Pvalue,1.30102999566\", \n",
      "  \"columns\": \"GIANT_b_lymphocyte_KnownFunctionalInteraction,GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction,GIANT_dendritic_cell_KnownFunctionalInteraction,GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction,GIANT_lymph_node_KnownFunctionalInteraction,GIANT_lymph_node_ProbabilityOfFunctionalInteraction,GIANT_lymphocyte_KnownFunctionalInteraction,GIANT_lymphocyte_ProbabilityOfFunctionalInteraction,GIANT_natural_killer_cell_KnownFunctionalInteraction,GIANT_natural_killer_cell_ProbabilityOfFunctionalInteraction,GIANT_spleen_KnownFunctionalInteraction,GIANT_spleen_ProbabilityOfFunctionalInteraction,GIANT_t_lymphocyte_KnownFunctionalInteraction,GIANT_t_lymphocyte_ProbabilityOfFunctionalInteraction,GIANT_thymocyte_KnownFunctionalInteraction,GIANT_thymocyte_ProbabilityOfFunctionalInteraction,GIANT_tonsil_KnownFunctionalInteraction,GIANT_tonsil_ProbabilityOfFunctionalInteraction,GTEx_Spleen_Correlation,GTEx_Spleen_Pvalue,TCGA_DLBC_Correlation,TCGA_DLBC_Pvalue,TCGA_THYM_Correlation,TCGA_THYM_Pvalue\", \n",
      "  \"restriction_join\": \"union\"\n",
      "}\n",
      "Submitting query request.\n",
      "Query request response.\n",
      "{\n",
      "  \"status\": \"submitted\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Check query status\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"running\", \n",
      "  \"message\": \"Extraction job is running.\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\"\n",
      "}\n",
      "Checking again\n",
      "{\n",
      "  \"status\": \"complete\", \n",
      "  \"rows\": 1000000, \n",
      "  \"processed_data\": \"0B\", \n",
      "  \"request_id\": \"04829dd7-e50b-4d70-9942-c5475ae76c2d\", \n",
      "  \"request_uri\": [\n",
      "    \"https://storage.googleapis.com/ncats_bigquery_results/04829dd7-e50b-4d70-9942-c5475ae76c2d000000000000.csv\"\n",
      "  ], \n",
      "  \"size\": \"154.41 MB\"\n",
      "}\n",
      "Query successful. Getting result.\n"
     ]
    }
   ],
   "source": [
    "# get lymphoid substudies\n",
    "import numpy as np\n",
    "query_status = None\n",
    "# get lymphoid substudies\n",
    "substudies = get('metadata/tissue/%s' % (\"lymphoid_tissue\",))\n",
    "\n",
    "# we only want things in a specific table, we'll use the default\n",
    "table = get('/metadata/table/%s' %  (default_table,))\n",
    "\n",
    "print(\"Using table %s\\n%s\" % (table['name'], table['description']))\n",
    "\n",
    "# grab pvalue columns\n",
    "pvalue_columns = []\n",
    "column_names = []\n",
    "for ss in substudies['substudies']:\n",
    "    for column in ss['columns']:\n",
    "        # only if column is from our table\n",
    "        if column['table']['name'] == table['name']:\n",
    "            # add column to select\n",
    "            column_names.append(column['name'])\n",
    "            if column['interactions_type'] == 'P-value (-log10)':\n",
    "                #add pvalue to where\n",
    "                pvalue_columns.append(column['name'])\n",
    " \n",
    "# grab columns with lower pvalues\n",
    "pv = []\n",
    "for p in pvalue_columns:\n",
    "    pv.append(p)\n",
    "    pv.append(str(abs(np.log10(.05))))\n",
    "query_arg = {}\n",
    "query_arg['table'] = table['name']\n",
    "query_arg['columns'] = ','.join(sorted(column_names))\n",
    "if len(pv):\n",
    "    query_arg['restriction_gt'] = ','.join(pv)\n",
    "# get if any of the lymphoid columns have \n",
    "query_arg['restriction_join'] = 'union'\n",
    "query_arg['limit'] = 1000000\n",
    "print \"The constructed query.\"\n",
    "jprint(query_arg)\n",
    "\n",
    "print(\"Submitting query request.\")\n",
    "try:\n",
    "    query_submit = get('interactions/query', data=query_arg)\n",
    "    print(\"Query request response.\")\n",
    "    jprint(query_submit)\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())\n",
    "\n",
    "print(\"Check query status\")\n",
    "try:\n",
    "    ctr = 1\n",
    "    while True:\n",
    "        \n",
    "        query_status = get('interactions/query/status/%s'% (query_submit['request_id'],))\n",
    "        jprint(query_status)\n",
    "        if query_status['status'] !='running':\n",
    "            # query has finished\n",
    "            break\n",
    "        else:\n",
    "            time.sleep(ctr)\n",
    "            ctr += 1\n",
    "            #linear backoff\n",
    "            print \"Checking again\"\n",
    "except requests.HTTPError as e:\n",
    "    print e\n",
    "    print #returned message\n",
    "    print jprint(e.response.json())\n",
    "\n",
    "\n",
    "if query_status and query_status['request_uri']:\n",
    "    print(\"Query successful. Getting result.\")\n",
    "    result = pandas.concat(map(pandas.read_csv, query_status['request_uri']))\n",
    "else:\n",
    "    print \"Error see above\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GPID</th>\n",
       "      <th>Gene1</th>\n",
       "      <th>Gene2</th>\n",
       "      <th>GIANT_b_lymphocyte_KnownFunctionalInteraction</th>\n",
       "      <th>GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction</th>\n",
       "      <th>GIANT_dendritic_cell_KnownFunctionalInteraction</th>\n",
       "      <th>GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction</th>\n",
       "      <th>GIANT_lymph_node_KnownFunctionalInteraction</th>\n",
       "      <th>GIANT_lymph_node_ProbabilityOfFunctionalInteraction</th>\n",
       "      <th>GIANT_lymphocyte_KnownFunctionalInteraction</th>\n",
       "      <th>...</th>\n",
       "      <th>GIANT_thymocyte_KnownFunctionalInteraction</th>\n",
       "      <th>GIANT_thymocyte_ProbabilityOfFunctionalInteraction</th>\n",
       "      <th>GIANT_tonsil_KnownFunctionalInteraction</th>\n",
       "      <th>GIANT_tonsil_ProbabilityOfFunctionalInteraction</th>\n",
       "      <th>GTEx_Spleen_Correlation</th>\n",
       "      <th>GTEx_Spleen_Pvalue</th>\n",
       "      <th>TCGA_DLBC_Correlation</th>\n",
       "      <th>TCGA_DLBC_Pvalue</th>\n",
       "      <th>TCGA_THYM_Correlation</th>\n",
       "      <th>TCGA_THYM_Pvalue</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>91170000171022</td>\n",
       "      <td>171022</td>\n",
       "      <td>9117</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-0.0303</td>\n",
       "      <td>0.12</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-0.3085</td>\n",
       "      <td>3.19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>842700101927402</td>\n",
       "      <td>101927402</td>\n",
       "      <td>84270</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-0.2865</td>\n",
       "      <td>2.50</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>46010000644759</td>\n",
       "      <td>644759</td>\n",
       "      <td>4601</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.2115</td>\n",
       "      <td>1.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>796400100506686</td>\n",
       "      <td>100506686</td>\n",
       "      <td>79640</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.2040</td>\n",
       "      <td>1.42</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>237740000729867</td>\n",
       "      <td>729867</td>\n",
       "      <td>23774</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.2811</td>\n",
       "      <td>2.42</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 27 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              GPID      Gene1  Gene2  \\\n",
       "0   91170000171022     171022   9117   \n",
       "1  842700101927402  101927402  84270   \n",
       "2   46010000644759     644759   4601   \n",
       "3  796400100506686  100506686  79640   \n",
       "4  237740000729867     729867  23774   \n",
       "\n",
       "   GIANT_b_lymphocyte_KnownFunctionalInteraction  \\\n",
       "0                                            NaN   \n",
       "1                                            NaN   \n",
       "2                                            NaN   \n",
       "3                                            NaN   \n",
       "4                                            NaN   \n",
       "\n",
       "   GIANT_b_lymphocyte_ProbabilityOfFunctionalInteraction  \\\n",
       "0                                                NaN       \n",
       "1                                                NaN       \n",
       "2                                                NaN       \n",
       "3                                                NaN       \n",
       "4                                                NaN       \n",
       "\n",
       "   GIANT_dendritic_cell_KnownFunctionalInteraction  \\\n",
       "0                                              NaN   \n",
       "1                                              NaN   \n",
       "2                                              NaN   \n",
       "3                                              NaN   \n",
       "4                                              NaN   \n",
       "\n",
       "   GIANT_dendritic_cell_ProbabilityOfFunctionalInteraction  \\\n",
       "0                                                NaN         \n",
       "1                                                NaN         \n",
       "2                                                NaN         \n",
       "3                                                NaN         \n",
       "4                                                NaN         \n",
       "\n",
       "   GIANT_lymph_node_KnownFunctionalInteraction  \\\n",
       "0                                          NaN   \n",
       "1                                          NaN   \n",
       "2                                          NaN   \n",
       "3                                          NaN   \n",
       "4                                          NaN   \n",
       "\n",
       "   GIANT_lymph_node_ProbabilityOfFunctionalInteraction  \\\n",
       "0                                                NaN     \n",
       "1                                                NaN     \n",
       "2                                                NaN     \n",
       "3                                                NaN     \n",
       "4                                                NaN     \n",
       "\n",
       "   GIANT_lymphocyte_KnownFunctionalInteraction        ...         \\\n",
       "0                                          NaN        ...          \n",
       "1                                          NaN        ...          \n",
       "2                                          NaN        ...          \n",
       "3                                          NaN        ...          \n",
       "4                                          NaN        ...          \n",
       "\n",
       "   GIANT_thymocyte_KnownFunctionalInteraction  \\\n",
       "0                                         NaN   \n",
       "1                                         NaN   \n",
       "2                                         NaN   \n",
       "3                                         NaN   \n",
       "4                                         NaN   \n",
       "\n",
       "   GIANT_thymocyte_ProbabilityOfFunctionalInteraction  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "2                                                NaN    \n",
       "3                                                NaN    \n",
       "4                                                NaN    \n",
       "\n",
       "   GIANT_tonsil_KnownFunctionalInteraction  \\\n",
       "0                                      NaN   \n",
       "1                                      NaN   \n",
       "2                                      NaN   \n",
       "3                                      NaN   \n",
       "4                                      NaN   \n",
       "\n",
       "   GIANT_tonsil_ProbabilityOfFunctionalInteraction  GTEx_Spleen_Correlation  \\\n",
       "0                                              NaN                  -0.0303   \n",
       "1                                              NaN                  -0.2865   \n",
       "2                                              NaN                   0.2115   \n",
       "3                                              NaN                   0.2040   \n",
       "4                                              NaN                   0.2811   \n",
       "\n",
       "   GTEx_Spleen_Pvalue  TCGA_DLBC_Correlation  TCGA_DLBC_Pvalue  \\\n",
       "0                0.12                    NaN               NaN   \n",
       "1                2.50                    NaN               NaN   \n",
       "2                1.51                    NaN               NaN   \n",
       "3                1.42                    NaN               NaN   \n",
       "4                2.42                    NaN               NaN   \n",
       "\n",
       "   TCGA_THYM_Correlation  TCGA_THYM_Pvalue  \n",
       "0                -0.3085              3.19  \n",
       "1                    NaN               NaN  \n",
       "2                    NaN               NaN  \n",
       "3                    NaN               NaN  \n",
       "4                    NaN               NaN  \n",
       "\n",
       "[5 rows x 27 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Full example result\n",
    "result.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
