{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## CQ1.7 - Gene-Gene Phenotype Similarity for FA Genes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Description: https://docs.google.com/document/d/10wvsoNLbbFOvHWVWRdYQdJ_PD0uO7SGGR-gIqCM8GWU/edit#"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "Approach 1: Human Pheno Cross-Species Analysis Approach: Single pooled set of input phenos used in a single cross-species phenosim analysis \n",
    "\n",
    "FA genes -> pheno profiles -> merge and de-dup phenos ->  single cross-species phenosim -> trim input genes -> select hits -> map to human ortholog\n",
    "\n",
    " Input: set of human all 27 FA genes\n",
    "Retrieve set of phenotypes linked to each of the input human FA gene \n",
    "\t-> 27 phenotype profiles (i.e. 27 sets of phenotypes)\n",
    "Pool phenotypes from all profiles into a single set and remove dups to give list of non-redundant phenotypes\n",
    "\t-> one set of non-redundant phenotypes \n",
    "Use this set to execute a single cross-species phenosim analysis to compare against pheno profiles of genes from 5 species (human, mouse, zebrafish, fly, worm)\n",
    "\t-> one list of phenosim results/scores containing genes from 5 species\n",
    "Remove FA genes from results (i.e. trim inputs)\n",
    "-> one list of scores with input genes removed\n",
    "Take top 25 results as gene ‘hits’\n",
    "-> trimmed list of 25 gene hits\n",
    "Retrieve human orthologs of any non-human gene hits\n",
    "\t-> set of ortholog-human gene pairs\n",
    "Output: Return list of 25 gene ‘hits’ - along with phenosim score, taxon of the gene, and its human ortholog"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# autogenerate biolink_client\n",
    "# curl --insecure -X POST -H \"content-type:application/json\" -d '{\"swaggerUrl\":\"https://api.monarchinitiative.org/api/swagger.json\"}' https://generator.swagger.io/api/gen/clients/python\n",
    "# and rename it to biolink_client"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os, sys\n",
    "# change this path\n",
    "sys.path.insert(0, \"/home/gstupp/projects/NCATS-Tangerine/biolink_client\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import biolink_client\n",
    "from biolink_client.api_client import ApiClient\n",
    "from biolink_client.rest import ApiException\n",
    "import requests\n",
    "from itertools import chain\n",
    "import pandas as pd\n",
    "from pprint import pprint\n",
    "from tqdm import tqdm, tqdm_notebook\n",
    "from collections import defaultdict\n",
    "\n",
    "pd.options.display.max_rows = 999\n",
    "\n",
    "MONARCH_API = \"https://api.monarchinitiative.org/api\"\n",
    "SIMSEARCH_API = \"https://monarchinitiative.org/simsearch/phenotype\"\n",
    "\n",
    "gene_list = \"https://raw.githubusercontent.com/NCATS-Tangerine/cq-notebooks/master/FA_gene_sets/FA_4_all_genes.txt\"\n",
    "\n",
    "client = ApiClient(host=MONARCH_API)\n",
    "client.set_default_header('Content-Type', 'text/plain')\n",
    "api_instance = biolink_client.BioentityApi(client)\n",
    "\n",
    "# Get the gene list from github\n",
    "dataframe = pd.read_csv(gene_list, sep='\\t', names=['gene_id', 'symbol'])\n",
    "df = dataframe.set_index('symbol')\n",
    "human_FA_genes = set(df.gene_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_obj(obj_id):\n",
    "    url = \"https://api.monarchinitiative.org/api/bioentity/{}\".format(obj_id)\n",
    "    res = requests.get(url)\n",
    "    d = res.json()\n",
    "    return d\n",
    "def get_taxon_from_gene(gene):\n",
    "    return get_obj(gene)['taxon']['label']\n",
    "def get_phenotype_from_gene(gene):\n",
    "    # https://monarchinitiative.org/gene/NCBIGene%3A2176/phenotype_list.json\n",
    "    url = \"https://monarchinitiative.org/gene/{}/phenotype_list.json\"\n",
    "    return [x['id'] for x in requests.get(url.format(gene)).json()['phenotype_list']]\n",
    "def get_phenotype_from_gene_verbose(gene):\n",
    "    # https://monarchinitiative.org/gene/NCBIGene%3A2176/phenotype_list.json\n",
    "    url = \"https://monarchinitiative.org/gene/{}/phenotype_list.json\"\n",
    "    return [(x['id'],x['label']) for x in requests.get(url.format(gene)).json()['phenotype_list']]\n",
    "def query_orthologs(gene_id, taxon=None):\n",
    "    \"\"\"Query Monarch to determine the orthologs of a gene.\"\"\"\n",
    "    url = \"https://api.monarchinitiative.org/api/bioentity/gene/{}/homologs/\".format(gene_id)\n",
    "    if taxon:\n",
    "        res = requests.get(url, params={'homolog_taxon': taxon})\n",
    "    else:        \n",
    "        res = requests.get(url)\n",
    "    d = res.json()\n",
    "    return [x['object']['id'] for x in d['associations']]\n",
    "#query_orthologs('MGI:88276', taxon=\"NCBITaxon:9606\")\n",
    "#get_phenotype_from_gene(\"NCBIGene:2176\")\n",
    "#get_taxon_from_gene('NCBIGene:2176')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_phenotypically_similar_genes(phenotypes, taxon, return_all=False):\n",
    "    headers = {\n",
    "        'Accept-Encoding': 'gzip, deflate, br',\n",
    "        'Accept-Language': 'en-US,en;q=0.8',\n",
    "        'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',\n",
    "        'Accept': 'application/json, text/javascript, */*; q=0.01',\n",
    "    }\n",
    "    data = {'input_items': \" \".join(phenotypes), \"target_species\": taxon}\n",
    "    r = requests.post(SIMSEARCH_API, data=data, headers=headers)\n",
    "    d = r.json()\n",
    "    if return_all:\n",
    "        return d\n",
    "    if 'b' in d:\n",
    "        scores = [(x['id'],x['score']['score'], x['label']) for x in d['b']]\n",
    "    else:\n",
    "        scores = []\n",
    "    return scores\n",
    "#get_phenotypically_similar_genes(phenotypes, \"10090\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The installed widget Javascript is the wrong version.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "# 1. Retrieve set of phenotypes linked to each of the input human FA gene\n",
    "gene_phenotypes = dict()\n",
    "for gene in tqdm_notebook(human_FA_genes):\n",
    "    phenotypes = get_phenotype_from_gene(gene)\n",
    "    gene_phenotypes[gene] = phenotypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['HP:0010302', 'HP:0000020', 'HP:0001903', 'HP:0011267', 'HP:0002916']\n",
      "449\n"
     ]
    }
   ],
   "source": [
    "# 2. Pool phenotypes from all profiles into a single set and remove dups to give list of non-redundant phenotypes\n",
    "phenotypes = set(chain(*gene_phenotypes.values()))\n",
    "print(list(phenotypes)[:5])\n",
    "print(len(phenotypes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# 3. Use this set to execute a single cross-species phenosim analysis to compare against pheno profiles of \n",
    "# genes from 5 species (human, mouse, zebrafish, fly, worm)\n",
    "\n",
    "# note: no human!!\n",
    "taxids = [10090, 7955, 7227, 6239]\n",
    "results = dict()\n",
    "for taxid in taxids:\n",
    "    response = get_phenotypically_similar_genes(phenotypes, taxid)\n",
    "    results[taxid] = [{\"id\": r[0], \"score\": r[1], \"symbol\": r[2]} for r in response]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# 4. Remove FA genes from results (i.e. trim inputs)\n",
    "# There's no human in here, so will do this after we get orthologs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7227\n",
      "[]\n",
      "10090\n",
      "[{'id': 'MGI:1330810', 'score': 66, 'symbol': 'Trp63'}, {'id': 'MGI:109583', 'score': 64, 'symbol': 'Pten'}, {'id': 'MGI:88276', 'score': 63, 'symbol': 'Ctnnb1'}, {'id': 'MGI:95523', 'score': 62, 'symbol': 'Fgfr2'}, {'id': 'MGI:88039', 'score': 62, 'symbol': 'Apc'}, {'id': 'MGI:98834', 'score': 61, 'symbol': 'Trp53'}, {'id': 'MGI:99484', 'score': 60, 'symbol': 'Chuk'}, {'id': 'MGI:98297', 'score': 60, 'symbol': 'Shh'}, {'id': 'MGI:96680', 'score': 60, 'symbol': 'Kras'}, {'id': 'MGI:1347466', 'score': 60, 'symbol': 'Foxc1'}]\n",
      "7955\n",
      "[{'id': 'ZFIN:ZDB-GENE-031112-7', 'score': 51, 'symbol': 'apc'}, {'id': 'ZFIN:ZDB-GENE-080405-1', 'score': 49, 'symbol': 'tert'}, {'id': 'ZFIN:ZDB-GENE-011026-1', 'score': 48, 'symbol': 'sox32'}, {'id': 'ZFIN:ZDB-GENE-060125-1', 'score': 48, 'symbol': 'med12'}, {'id': 'ZFIN:ZDB-GENE-030131-9685', 'score': 47, 'symbol': 'ddx18'}, {'id': 'ZFIN:ZDB-GENE-011010-3', 'score': 47, 'symbol': 'aldh1a2'}, {'id': 'ZFIN:ZDB-GENE-990615-5', 'score': 46, 'symbol': 'tbx16'}, {'id': 'ZFIN:ZDB-GENE-030404-2', 'score': 46, 'symbol': 'mib1'}, {'id': 'ZFIN:ZDB-GENE-030131-6378', 'score': 46, 'symbol': 'heatr1'}, {'id': 'ZFIN:ZDB-GENE-000626-1', 'score': 46, 'symbol': 'tnnt2a'}]\n",
      "6239\n",
      "[{'id': 'WormBase:WBGene00010556', 'score': 34, 'symbol': 'rack-1'}, {'id': 'WormBase:WBGene00004077', 'score': 29, 'symbol': 'pop-1'}]\n"
     ]
    }
   ],
   "source": [
    "# 5. Take top 25 results as gene ‘hits’\n",
    "for taxid in results:\n",
    "    print(taxid)\n",
    "    print(results[taxid][:10])\n",
    "# scores of each organism are very different, so I'll take the top X for each organism"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "results = {k:sorted(v, key=lambda x:x['score'], reverse=True)[:10] for k,v in results.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'id': 'ZFIN:ZDB-GENE-031112-7', 'score': 51, 'symbol': 'apc'},\n",
       " {'id': 'ZFIN:ZDB-GENE-080405-1', 'score': 49, 'symbol': 'tert'},\n",
       " {'id': 'ZFIN:ZDB-GENE-011026-1', 'score': 48, 'symbol': 'sox32'},\n",
       " {'id': 'ZFIN:ZDB-GENE-060125-1', 'score': 48, 'symbol': 'med12'},\n",
       " {'id': 'ZFIN:ZDB-GENE-030131-9685', 'score': 47, 'symbol': 'ddx18'},\n",
       " {'id': 'ZFIN:ZDB-GENE-011010-3', 'score': 47, 'symbol': 'aldh1a2'},\n",
       " {'id': 'ZFIN:ZDB-GENE-990615-5', 'score': 46, 'symbol': 'tbx16'},\n",
       " {'id': 'ZFIN:ZDB-GENE-030404-2', 'score': 46, 'symbol': 'mib1'},\n",
       " {'id': 'ZFIN:ZDB-GENE-030131-6378', 'score': 46, 'symbol': 'heatr1'},\n",
       " {'id': 'ZFIN:ZDB-GENE-000626-1', 'score': 46, 'symbol': 'tnnt2a'},\n",
       " {'id': 'MGI:1330810', 'score': 66, 'symbol': 'Trp63'},\n",
       " {'id': 'MGI:109583', 'score': 64, 'symbol': 'Pten'},\n",
       " {'id': 'MGI:88276', 'score': 63, 'symbol': 'Ctnnb1'},\n",
       " {'id': 'MGI:95523', 'score': 62, 'symbol': 'Fgfr2'},\n",
       " {'id': 'MGI:88039', 'score': 62, 'symbol': 'Apc'},\n",
       " {'id': 'MGI:98834', 'score': 61, 'symbol': 'Trp53'},\n",
       " {'id': 'MGI:99484', 'score': 60, 'symbol': 'Chuk'},\n",
       " {'id': 'MGI:98297', 'score': 60, 'symbol': 'Shh'},\n",
       " {'id': 'MGI:96680', 'score': 60, 'symbol': 'Kras'},\n",
       " {'id': 'MGI:1347466', 'score': 60, 'symbol': 'Foxc1'},\n",
       " {'id': 'WormBase:WBGene00010556', 'score': 34, 'symbol': 'rack-1'},\n",
       " {'id': 'WormBase:WBGene00004077', 'score': 29, 'symbol': 'pop-1'}]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results_list = list(chain(*results.values()))\n",
    "results_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The installed widget Javascript is the wrong version.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "# 6. Retrieve human orthologs of any non-human gene hits\n",
    "for r in tqdm_notebook(results_list):\n",
    "    r['orthologs'] = query_orthologs(r['id'], \"NCBITaxon:9606\")\n",
    "    s['ortholog_labels'] = [get_obj(x)['label'] for x in s['orthologs']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>ortholog_labels</th>\n",
       "      <th>orthologs</th>\n",
       "      <th>score</th>\n",
       "      <th>symbol</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MGI:1330810</td>\n",
       "      <td>[TP63]</td>\n",
       "      <td>[NCBIGene:8626]</td>\n",
       "      <td>66</td>\n",
       "      <td>Trp63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MGI:109583</td>\n",
       "      <td>[PTEN]</td>\n",
       "      <td>[NCBIGene:5728]</td>\n",
       "      <td>64</td>\n",
       "      <td>Pten</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MGI:88276</td>\n",
       "      <td>[CTNNB1]</td>\n",
       "      <td>[NCBIGene:1499]</td>\n",
       "      <td>63</td>\n",
       "      <td>Ctnnb1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MGI:95523</td>\n",
       "      <td>[FGFR2]</td>\n",
       "      <td>[NCBIGene:2263]</td>\n",
       "      <td>62</td>\n",
       "      <td>Fgfr2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MGI:88039</td>\n",
       "      <td>[APC]</td>\n",
       "      <td>[NCBIGene:324]</td>\n",
       "      <td>62</td>\n",
       "      <td>Apc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>MGI:98834</td>\n",
       "      <td>[TP53]</td>\n",
       "      <td>[NCBIGene:7157]</td>\n",
       "      <td>61</td>\n",
       "      <td>Trp53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>MGI:99484</td>\n",
       "      <td>[CHUK]</td>\n",
       "      <td>[NCBIGene:1147]</td>\n",
       "      <td>60</td>\n",
       "      <td>Chuk</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>MGI:98297</td>\n",
       "      <td>[SHH]</td>\n",
       "      <td>[NCBIGene:6469]</td>\n",
       "      <td>60</td>\n",
       "      <td>Shh</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>MGI:96680</td>\n",
       "      <td>[KRAS]</td>\n",
       "      <td>[NCBIGene:3845]</td>\n",
       "      <td>60</td>\n",
       "      <td>Kras</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>MGI:1347466</td>\n",
       "      <td>[FOXC1]</td>\n",
       "      <td>[NCBIGene:2296]</td>\n",
       "      <td>60</td>\n",
       "      <td>Foxc1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>ZFIN:ZDB-GENE-031112-7</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>51</td>\n",
       "      <td>apc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>ZFIN:ZDB-GENE-080405-1</td>\n",
       "      <td>[TERT]</td>\n",
       "      <td>[NCBIGene:7015]</td>\n",
       "      <td>49</td>\n",
       "      <td>tert</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>ZFIN:ZDB-GENE-011026-1</td>\n",
       "      <td>[SOX17]</td>\n",
       "      <td>[NCBIGene:64321]</td>\n",
       "      <td>48</td>\n",
       "      <td>sox32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>ZFIN:ZDB-GENE-060125-1</td>\n",
       "      <td>[MED12, NCOR1P1, MED12L]</td>\n",
       "      <td>[NCBIGene:9968, NCBIGene:149934, NCBIGene:116931]</td>\n",
       "      <td>48</td>\n",
       "      <td>med12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>ZFIN:ZDB-GENE-030131-9685</td>\n",
       "      <td>[DDX18]</td>\n",
       "      <td>[NCBIGene:8886]</td>\n",
       "      <td>47</td>\n",
       "      <td>ddx18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>ZFIN:ZDB-GENE-011010-3</td>\n",
       "      <td>[ALDH1A1, ALDH1A2, ALDH16A1]</td>\n",
       "      <td>[NCBIGene:216, NCBIGene:8854, NCBIGene:126133]</td>\n",
       "      <td>47</td>\n",
       "      <td>aldh1a2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>ZFIN:ZDB-GENE-990615-5</td>\n",
       "      <td>[MGA, TBX1, TBX10]</td>\n",
       "      <td>[NCBIGene:23269, NCBIGene:6899, NCBIGene:347853]</td>\n",
       "      <td>46</td>\n",
       "      <td>tbx16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>ZFIN:ZDB-GENE-030404-2</td>\n",
       "      <td>[MIB1]</td>\n",
       "      <td>[NCBIGene:57534]</td>\n",
       "      <td>46</td>\n",
       "      <td>mib1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>ZFIN:ZDB-GENE-030131-6378</td>\n",
       "      <td>[HEATR1]</td>\n",
       "      <td>[NCBIGene:55127]</td>\n",
       "      <td>46</td>\n",
       "      <td>heatr1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>ZFIN:ZDB-GENE-000626-1</td>\n",
       "      <td>[TNNT2]</td>\n",
       "      <td>[NCBIGene:7139]</td>\n",
       "      <td>46</td>\n",
       "      <td>tnnt2a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>WormBase:WBGene00010556</td>\n",
       "      <td>[RACK1]</td>\n",
       "      <td>[NCBIGene:10399]</td>\n",
       "      <td>34</td>\n",
       "      <td>rack-1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>WormBase:WBGene00004077</td>\n",
       "      <td>[TCF7, TCF7L1, TCF7L2, LEF1]</td>\n",
       "      <td>[NCBIGene:6932, NCBIGene:83439, NCBIGene:6934,...</td>\n",
       "      <td>29</td>\n",
       "      <td>pop-1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           id               ortholog_labels  \\\n",
       "0                 MGI:1330810                        [TP63]   \n",
       "1                  MGI:109583                        [PTEN]   \n",
       "2                   MGI:88276                      [CTNNB1]   \n",
       "3                   MGI:95523                       [FGFR2]   \n",
       "4                   MGI:88039                         [APC]   \n",
       "5                   MGI:98834                        [TP53]   \n",
       "6                   MGI:99484                        [CHUK]   \n",
       "7                   MGI:98297                         [SHH]   \n",
       "8                   MGI:96680                        [KRAS]   \n",
       "9                 MGI:1347466                       [FOXC1]   \n",
       "10     ZFIN:ZDB-GENE-031112-7                            []   \n",
       "11     ZFIN:ZDB-GENE-080405-1                        [TERT]   \n",
       "12     ZFIN:ZDB-GENE-011026-1                       [SOX17]   \n",
       "13     ZFIN:ZDB-GENE-060125-1      [MED12, NCOR1P1, MED12L]   \n",
       "14  ZFIN:ZDB-GENE-030131-9685                       [DDX18]   \n",
       "15     ZFIN:ZDB-GENE-011010-3  [ALDH1A1, ALDH1A2, ALDH16A1]   \n",
       "16     ZFIN:ZDB-GENE-990615-5            [MGA, TBX1, TBX10]   \n",
       "17     ZFIN:ZDB-GENE-030404-2                        [MIB1]   \n",
       "18  ZFIN:ZDB-GENE-030131-6378                      [HEATR1]   \n",
       "19     ZFIN:ZDB-GENE-000626-1                       [TNNT2]   \n",
       "20    WormBase:WBGene00010556                       [RACK1]   \n",
       "21    WormBase:WBGene00004077  [TCF7, TCF7L1, TCF7L2, LEF1]   \n",
       "\n",
       "                                            orthologs  score   symbol  \n",
       "0                                     [NCBIGene:8626]     66    Trp63  \n",
       "1                                     [NCBIGene:5728]     64     Pten  \n",
       "2                                     [NCBIGene:1499]     63   Ctnnb1  \n",
       "3                                     [NCBIGene:2263]     62    Fgfr2  \n",
       "4                                      [NCBIGene:324]     62      Apc  \n",
       "5                                     [NCBIGene:7157]     61    Trp53  \n",
       "6                                     [NCBIGene:1147]     60     Chuk  \n",
       "7                                     [NCBIGene:6469]     60      Shh  \n",
       "8                                     [NCBIGene:3845]     60     Kras  \n",
       "9                                     [NCBIGene:2296]     60    Foxc1  \n",
       "10                                                 []     51      apc  \n",
       "11                                    [NCBIGene:7015]     49     tert  \n",
       "12                                   [NCBIGene:64321]     48    sox32  \n",
       "13  [NCBIGene:9968, NCBIGene:149934, NCBIGene:116931]     48    med12  \n",
       "14                                    [NCBIGene:8886]     47    ddx18  \n",
       "15     [NCBIGene:216, NCBIGene:8854, NCBIGene:126133]     47  aldh1a2  \n",
       "16   [NCBIGene:23269, NCBIGene:6899, NCBIGene:347853]     46    tbx16  \n",
       "17                                   [NCBIGene:57534]     46     mib1  \n",
       "18                                   [NCBIGene:55127]     46   heatr1  \n",
       "19                                    [NCBIGene:7139]     46   tnnt2a  \n",
       "20                                   [NCBIGene:10399]     34   rack-1  \n",
       "21  [NCBIGene:6932, NCBIGene:83439, NCBIGene:6934,...     29    pop-1  "
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results_list = sorted(results_list, key=lambda x: x['score'], reverse=True)\n",
    "df = pd.DataFrame(results_list)\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# mark any rows where the ortholog is a FA gene\n",
    "df = pd.DataFrame(results_list)\n",
    "df = df[df.orthologs.map(len)>0]\n",
    "df1 = df.orthologs.apply(pd.Series).stack().rename('ortholog')\n",
    "df1 = df1.to_frame().reset_index(1, drop=True)\n",
    "df = df[['id', 'score', 'symbol']]\n",
    "df = df.join(df1).reset_index(drop=True)\n",
    "df['fa'] = df.ortholog.isin(human_FA_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "df['ortholog_label'] = df.ortholog.map(lambda x: get_obj(x)['label'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>score</th>\n",
       "      <th>symbol</th>\n",
       "      <th>ortholog</th>\n",
       "      <th>fa</th>\n",
       "      <th>ortholog_label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MGI:1330810</td>\n",
       "      <td>66</td>\n",
       "      <td>Trp63</td>\n",
       "      <td>NCBIGene:8626</td>\n",
       "      <td>False</td>\n",
       "      <td>TP63</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MGI:109583</td>\n",
       "      <td>64</td>\n",
       "      <td>Pten</td>\n",
       "      <td>NCBIGene:5728</td>\n",
       "      <td>False</td>\n",
       "      <td>PTEN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>MGI:88276</td>\n",
       "      <td>63</td>\n",
       "      <td>Ctnnb1</td>\n",
       "      <td>NCBIGene:1499</td>\n",
       "      <td>False</td>\n",
       "      <td>CTNNB1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>MGI:95523</td>\n",
       "      <td>62</td>\n",
       "      <td>Fgfr2</td>\n",
       "      <td>NCBIGene:2263</td>\n",
       "      <td>False</td>\n",
       "      <td>FGFR2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>MGI:88039</td>\n",
       "      <td>62</td>\n",
       "      <td>Apc</td>\n",
       "      <td>NCBIGene:324</td>\n",
       "      <td>False</td>\n",
       "      <td>APC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>MGI:98834</td>\n",
       "      <td>61</td>\n",
       "      <td>Trp53</td>\n",
       "      <td>NCBIGene:7157</td>\n",
       "      <td>False</td>\n",
       "      <td>TP53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>MGI:99484</td>\n",
       "      <td>60</td>\n",
       "      <td>Chuk</td>\n",
       "      <td>NCBIGene:1147</td>\n",
       "      <td>False</td>\n",
       "      <td>CHUK</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>MGI:98297</td>\n",
       "      <td>60</td>\n",
       "      <td>Shh</td>\n",
       "      <td>NCBIGene:6469</td>\n",
       "      <td>False</td>\n",
       "      <td>SHH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>MGI:96680</td>\n",
       "      <td>60</td>\n",
       "      <td>Kras</td>\n",
       "      <td>NCBIGene:3845</td>\n",
       "      <td>False</td>\n",
       "      <td>KRAS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>MGI:1347466</td>\n",
       "      <td>60</td>\n",
       "      <td>Foxc1</td>\n",
       "      <td>NCBIGene:2296</td>\n",
       "      <td>False</td>\n",
       "      <td>FOXC1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>ZFIN:ZDB-GENE-080405-1</td>\n",
       "      <td>49</td>\n",
       "      <td>tert</td>\n",
       "      <td>NCBIGene:7015</td>\n",
       "      <td>False</td>\n",
       "      <td>TERT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>ZFIN:ZDB-GENE-011026-1</td>\n",
       "      <td>48</td>\n",
       "      <td>sox32</td>\n",
       "      <td>NCBIGene:64321</td>\n",
       "      <td>False</td>\n",
       "      <td>SOX17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>ZFIN:ZDB-GENE-060125-1</td>\n",
       "      <td>48</td>\n",
       "      <td>med12</td>\n",
       "      <td>NCBIGene:9968</td>\n",
       "      <td>False</td>\n",
       "      <td>MED12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>ZFIN:ZDB-GENE-060125-1</td>\n",
       "      <td>48</td>\n",
       "      <td>med12</td>\n",
       "      <td>NCBIGene:149934</td>\n",
       "      <td>False</td>\n",
       "      <td>NCOR1P1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>ZFIN:ZDB-GENE-060125-1</td>\n",
       "      <td>48</td>\n",
       "      <td>med12</td>\n",
       "      <td>NCBIGene:116931</td>\n",
       "      <td>False</td>\n",
       "      <td>MED12L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>ZFIN:ZDB-GENE-030131-9685</td>\n",
       "      <td>47</td>\n",
       "      <td>ddx18</td>\n",
       "      <td>NCBIGene:8886</td>\n",
       "      <td>False</td>\n",
       "      <td>DDX18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>ZFIN:ZDB-GENE-011010-3</td>\n",
       "      <td>47</td>\n",
       "      <td>aldh1a2</td>\n",
       "      <td>NCBIGene:216</td>\n",
       "      <td>False</td>\n",
       "      <td>ALDH1A1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>ZFIN:ZDB-GENE-011010-3</td>\n",
       "      <td>47</td>\n",
       "      <td>aldh1a2</td>\n",
       "      <td>NCBIGene:8854</td>\n",
       "      <td>False</td>\n",
       "      <td>ALDH1A2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>ZFIN:ZDB-GENE-011010-3</td>\n",
       "      <td>47</td>\n",
       "      <td>aldh1a2</td>\n",
       "      <td>NCBIGene:126133</td>\n",
       "      <td>False</td>\n",
       "      <td>ALDH16A1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>ZFIN:ZDB-GENE-990615-5</td>\n",
       "      <td>46</td>\n",
       "      <td>tbx16</td>\n",
       "      <td>NCBIGene:23269</td>\n",
       "      <td>False</td>\n",
       "      <td>MGA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>ZFIN:ZDB-GENE-990615-5</td>\n",
       "      <td>46</td>\n",
       "      <td>tbx16</td>\n",
       "      <td>NCBIGene:6899</td>\n",
       "      <td>False</td>\n",
       "      <td>TBX1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>ZFIN:ZDB-GENE-990615-5</td>\n",
       "      <td>46</td>\n",
       "      <td>tbx16</td>\n",
       "      <td>NCBIGene:347853</td>\n",
       "      <td>False</td>\n",
       "      <td>TBX10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>ZFIN:ZDB-GENE-030404-2</td>\n",
       "      <td>46</td>\n",
       "      <td>mib1</td>\n",
       "      <td>NCBIGene:57534</td>\n",
       "      <td>False</td>\n",
       "      <td>MIB1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>ZFIN:ZDB-GENE-030131-6378</td>\n",
       "      <td>46</td>\n",
       "      <td>heatr1</td>\n",
       "      <td>NCBIGene:55127</td>\n",
       "      <td>False</td>\n",
       "      <td>HEATR1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>ZFIN:ZDB-GENE-000626-1</td>\n",
       "      <td>46</td>\n",
       "      <td>tnnt2a</td>\n",
       "      <td>NCBIGene:7139</td>\n",
       "      <td>False</td>\n",
       "      <td>TNNT2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>WormBase:WBGene00010556</td>\n",
       "      <td>34</td>\n",
       "      <td>rack-1</td>\n",
       "      <td>NCBIGene:10399</td>\n",
       "      <td>False</td>\n",
       "      <td>RACK1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>WormBase:WBGene00004077</td>\n",
       "      <td>29</td>\n",
       "      <td>pop-1</td>\n",
       "      <td>NCBIGene:6932</td>\n",
       "      <td>False</td>\n",
       "      <td>TCF7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>WormBase:WBGene00004077</td>\n",
       "      <td>29</td>\n",
       "      <td>pop-1</td>\n",
       "      <td>NCBIGene:83439</td>\n",
       "      <td>False</td>\n",
       "      <td>TCF7L1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>WormBase:WBGene00004077</td>\n",
       "      <td>29</td>\n",
       "      <td>pop-1</td>\n",
       "      <td>NCBIGene:6934</td>\n",
       "      <td>False</td>\n",
       "      <td>TCF7L2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>WormBase:WBGene00004077</td>\n",
       "      <td>29</td>\n",
       "      <td>pop-1</td>\n",
       "      <td>NCBIGene:51176</td>\n",
       "      <td>False</td>\n",
       "      <td>LEF1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           id  score   symbol         ortholog     fa  \\\n",
       "0                 MGI:1330810     66    Trp63    NCBIGene:8626  False   \n",
       "1                  MGI:109583     64     Pten    NCBIGene:5728  False   \n",
       "2                   MGI:88276     63   Ctnnb1    NCBIGene:1499  False   \n",
       "3                   MGI:95523     62    Fgfr2    NCBIGene:2263  False   \n",
       "4                   MGI:88039     62      Apc     NCBIGene:324  False   \n",
       "5                   MGI:98834     61    Trp53    NCBIGene:7157  False   \n",
       "6                   MGI:99484     60     Chuk    NCBIGene:1147  False   \n",
       "7                   MGI:98297     60      Shh    NCBIGene:6469  False   \n",
       "8                   MGI:96680     60     Kras    NCBIGene:3845  False   \n",
       "9                 MGI:1347466     60    Foxc1    NCBIGene:2296  False   \n",
       "10     ZFIN:ZDB-GENE-080405-1     49     tert    NCBIGene:7015  False   \n",
       "11     ZFIN:ZDB-GENE-011026-1     48    sox32   NCBIGene:64321  False   \n",
       "12     ZFIN:ZDB-GENE-060125-1     48    med12    NCBIGene:9968  False   \n",
       "13     ZFIN:ZDB-GENE-060125-1     48    med12  NCBIGene:149934  False   \n",
       "14     ZFIN:ZDB-GENE-060125-1     48    med12  NCBIGene:116931  False   \n",
       "15  ZFIN:ZDB-GENE-030131-9685     47    ddx18    NCBIGene:8886  False   \n",
       "16     ZFIN:ZDB-GENE-011010-3     47  aldh1a2     NCBIGene:216  False   \n",
       "17     ZFIN:ZDB-GENE-011010-3     47  aldh1a2    NCBIGene:8854  False   \n",
       "18     ZFIN:ZDB-GENE-011010-3     47  aldh1a2  NCBIGene:126133  False   \n",
       "19     ZFIN:ZDB-GENE-990615-5     46    tbx16   NCBIGene:23269  False   \n",
       "20     ZFIN:ZDB-GENE-990615-5     46    tbx16    NCBIGene:6899  False   \n",
       "21     ZFIN:ZDB-GENE-990615-5     46    tbx16  NCBIGene:347853  False   \n",
       "22     ZFIN:ZDB-GENE-030404-2     46     mib1   NCBIGene:57534  False   \n",
       "23  ZFIN:ZDB-GENE-030131-6378     46   heatr1   NCBIGene:55127  False   \n",
       "24     ZFIN:ZDB-GENE-000626-1     46   tnnt2a    NCBIGene:7139  False   \n",
       "25    WormBase:WBGene00010556     34   rack-1   NCBIGene:10399  False   \n",
       "26    WormBase:WBGene00004077     29    pop-1    NCBIGene:6932  False   \n",
       "27    WormBase:WBGene00004077     29    pop-1   NCBIGene:83439  False   \n",
       "28    WormBase:WBGene00004077     29    pop-1    NCBIGene:6934  False   \n",
       "29    WormBase:WBGene00004077     29    pop-1   NCBIGene:51176  False   \n",
       "\n",
       "   ortholog_label  \n",
       "0            TP63  \n",
       "1            PTEN  \n",
       "2          CTNNB1  \n",
       "3           FGFR2  \n",
       "4             APC  \n",
       "5            TP53  \n",
       "6            CHUK  \n",
       "7             SHH  \n",
       "8            KRAS  \n",
       "9           FOXC1  \n",
       "10           TERT  \n",
       "11          SOX17  \n",
       "12          MED12  \n",
       "13        NCOR1P1  \n",
       "14         MED12L  \n",
       "15          DDX18  \n",
       "16        ALDH1A1  \n",
       "17        ALDH1A2  \n",
       "18       ALDH16A1  \n",
       "19            MGA  \n",
       "20           TBX1  \n",
       "21          TBX10  \n",
       "22           MIB1  \n",
       "23         HEATR1  \n",
       "24          TNNT2  \n",
       "25          RACK1  \n",
       "26           TCF7  \n",
       "27         TCF7L1  \n",
       "28         TCF7L2  \n",
       "29           LEF1  "
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
