{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Query:\n",
    "What genes encode proteins that physically interact with proteins encoded by the 11 Fanconi Anemia core complex genes (set FA-core)? \n",
    "\n",
    "### Input:\n",
    "\n",
    "Hardcoded tsv file from:\n",
    "\n",
    "https://raw.githubusercontent.com/NCATS-Tangerine/cq-notebooks/master/FA_gene_sets/FA_1_core_complex.txt\n",
    "\n",
    "### Goal:\n",
    "This simple query aims to expand the FA-core gene set based PPI network membership.\n",
    "\n",
    "### Route:\n",
    "1. Protein-protein interactions from Monarch Solr index, either direct or inferred through orthology.  Sources: [BioGRID](https://thebiogrid.org/) and [STRING](http://string-db.org/)\n",
    "2. Gene-ortholog associations from Monarch Solr index.  Sources: [Panther](http://www.pantherdb.org/)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene</th>\n",
       "      <th>interactor_id</th>\n",
       "      <th>interactor_symbol</th>\n",
       "      <th>qualifier</th>\n",
       "      <th>inferred_gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>NCBIGene:4851</td>\n",
       "      <td>NOTCH1</td>\n",
       "      <td>direct</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>NCBIGene:6117</td>\n",
       "      <td>RPA1</td>\n",
       "      <td>direct</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>MGI:2384790</td>\n",
       "      <td>Fanci</td>\n",
       "      <td>homology</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>NCBIGene:2189</td>\n",
       "      <td>FANCG</td>\n",
       "      <td>direct</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>MGI:1914446</td>\n",
       "      <td>Ube2t</td>\n",
       "      <td>homology</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>MGI:3648788</td>\n",
       "      <td>Gm5239</td>\n",
       "      <td>homology</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>NCBIGene:10915</td>\n",
       "      <td>TCERG1</td>\n",
       "      <td>direct</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>NCBIGene:2783</td>\n",
       "      <td>GNB2</td>\n",
       "      <td>direct</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>MGI:1917178</td>\n",
       "      <td>Cenps</td>\n",
       "      <td>homology</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>FANCA</td>\n",
       "      <td>MGI:3643360</td>\n",
       "      <td>Gm7808</td>\n",
       "      <td>homology</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    gene   interactor_id interactor_symbol qualifier inferred_gene\n",
       "0  FANCA   NCBIGene:4851            NOTCH1    direct           NaN\n",
       "1  FANCA   NCBIGene:6117              RPA1    direct           NaN\n",
       "2  FANCA     MGI:2384790             Fanci  homology           NaN\n",
       "3  FANCA   NCBIGene:2189             FANCG    direct           NaN\n",
       "4  FANCA     MGI:1914446             Ube2t  homology           NaN\n",
       "5  FANCA     MGI:3648788            Gm5239  homology           NaN\n",
       "6  FANCA  NCBIGene:10915            TCERG1    direct           NaN\n",
       "7  FANCA   NCBIGene:2783              GNB2    direct           NaN\n",
       "8  FANCA     MGI:1917178             Cenps  homology           NaN\n",
       "9  FANCA     MGI:3643360            Gm7808  homology           NaN"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import requests\n",
    "import pandas as pd\n",
    "import copy\n",
    "\n",
    "solr_url = 'https://solr-dev.monarchinitiative.org/solr/golr/select'\n",
    "core_set = 'https://raw.githubusercontent.com/NCATS-Tangerine/cq-notebooks/master/FA_gene_sets/FA_1_core_complex.txt'\n",
    "\n",
    "columns = ['gene', 'interactor_id', 'interactor_symbol', 'qualifier', 'inferred_gene']\n",
    "dataframe = pd.read_csv(core_set, sep='\\t', names=['gene', 'symbol'])\n",
    "\n",
    "def get_solr_results(solr, params):\n",
    "    resultCount = params['rows']\n",
    "    while params['start'] < resultCount:\n",
    "        solr_request = requests.get(solr, params=params)\n",
    "        response = solr_request.json()\n",
    "        resultCount = response['response']['numFound']\n",
    "        params['start'] += params['rows']\n",
    "        for doc in response['response']['docs']:\n",
    "            yield doc\n",
    "\n",
    "interaction_params = {\n",
    "    'wt': 'json',\n",
    "    'rows': 100,\n",
    "    'start': 0,\n",
    "    'q': '*:*',\n",
    "    'fl': 'subject, subject_label, subject_closure, \\\n",
    "           object, object_label, object_taxon',\n",
    "    'fq': ['relation_closure: \"RO:0002434\"']\n",
    "}\n",
    "\n",
    "# Make new dataframe for results\n",
    "interact_table = pd.DataFrame(columns=columns)\n",
    "\n",
    "\n",
    "# Get interactions, both direct and inferred\n",
    "for index, row in dataframe.iterrows():\n",
    "    params = copy.deepcopy(interaction_params)\n",
    "    params['fq'].append('subject_closure: \"{0}\" \\\n",
    "                        OR subject_ortholog_closure: \"{0}\"'\n",
    "                        .format(row['gene']))\n",
    "    for doc in get_solr_results(solr_url, params):\n",
    "        result = {}\n",
    "        result['gene'] = row['symbol']\n",
    "        result['interactor_id'] = doc['object']\n",
    "        result['interactor_symbol'] = doc['object_label']\n",
    "        if row['gene'] in doc['subject_closure']:\n",
    "            result['qualifier'] = \"direct\"\n",
    "        else:\n",
    "            result['qualifier'] = \"homology\"    \n",
    "        interact_table = interact_table.append(result, ignore_index=True)\n",
    "            \n",
    "interact_table.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene</th>\n",
       "      <th>interactor_id</th>\n",
       "      <th>interactor_symbol</th>\n",
       "      <th>qualifier</th>\n",
       "      <th>inferred_gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3633</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:2187</td>\n",
       "      <td>FANCB</td>\n",
       "      <td>homology</td>\n",
       "      <td>Fancb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3634</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:548593</td>\n",
       "      <td>SLX1A</td>\n",
       "      <td>homology</td>\n",
       "      <td>Slx1b</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3635</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:983</td>\n",
       "      <td>CDK1</td>\n",
       "      <td>homology</td>\n",
       "      <td>cdk1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3636</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:57531</td>\n",
       "      <td>HACE1</td>\n",
       "      <td>homology</td>\n",
       "      <td>Hace1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3637</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:6047</td>\n",
       "      <td>RNF4</td>\n",
       "      <td>homology</td>\n",
       "      <td>rnf4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3638</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:643904</td>\n",
       "      <td>RNF222</td>\n",
       "      <td>homology</td>\n",
       "      <td>rnf4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3639</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:26091</td>\n",
       "      <td>HERC4</td>\n",
       "      <td>homology</td>\n",
       "      <td>herc4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3640</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:55215</td>\n",
       "      <td>FANCI</td>\n",
       "      <td>homology</td>\n",
       "      <td>Fanci</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3641</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:643904</td>\n",
       "      <td>RNF222</td>\n",
       "      <td>homology</td>\n",
       "      <td>Rnf4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3642</th>\n",
       "      <td>UBE2T</td>\n",
       "      <td>NCBIGene:6047</td>\n",
       "      <td>RNF4</td>\n",
       "      <td>homology</td>\n",
       "      <td>Rnf4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       gene    interactor_id interactor_symbol qualifier inferred_gene\n",
       "3633  UBE2T    NCBIGene:2187             FANCB  homology         Fancb\n",
       "3634  UBE2T  NCBIGene:548593             SLX1A  homology         Slx1b\n",
       "3635  UBE2T     NCBIGene:983              CDK1  homology          cdk1\n",
       "3636  UBE2T   NCBIGene:57531             HACE1  homology         Hace1\n",
       "3637  UBE2T    NCBIGene:6047              RNF4  homology          rnf4\n",
       "3638  UBE2T  NCBIGene:643904            RNF222  homology          rnf4\n",
       "3639  UBE2T   NCBIGene:26091             HERC4  homology         herc4\n",
       "3640  UBE2T   NCBIGene:55215             FANCI  homology         Fanci\n",
       "3641  UBE2T  NCBIGene:643904            RNF222  homology          Rnf4\n",
       "3642  UBE2T    NCBIGene:6047              RNF4  homology          Rnf4"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Define function to fetch orthologs given a gene ID\n",
    "def get_human_ortholog(solr, gene):\n",
    "    params = {\n",
    "            'wt': 'json',\n",
    "            'rows': 100,\n",
    "            'start': 0,\n",
    "            'q': '*:*',\n",
    "            'fl': 'subject, subject_label,'\n",
    "                  'object, object_label',\n",
    "            'fq': ['subject_closure: \"{0}\"'.format(gene),\n",
    "                   'relation_closure: \"RO:HOM0000017\"',\n",
    "                   'object_taxon: \"NCBITaxon:9606\"'\n",
    "            ]\n",
    "    }\n",
    "    for doc in get_solr_results(solr, params):\n",
    "        yield doc\n",
    "\n",
    "# Get interactions, both direct and inferred\n",
    "for index, row in interact_table.iterrows():\n",
    "    if row['qualifier'] == 'homology':\n",
    "        for doc in get_human_ortholog(solr_url, row['interactor_id']):\n",
    "            result = {}\n",
    "            result['gene'] = row['gene']\n",
    "            result['interactor_id'] = doc['object']\n",
    "            result['interactor_symbol'] = doc['object_label']\n",
    "            result['qualifier'] = \"homology\"    \n",
    "            result['inferred_gene'] = row['interactor_symbol']\n",
    "            interact_table = interact_table.append(result, ignore_index=True)\n",
    "        \n",
    "interact_table.tail(10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "FANCI      51\n",
       "HES1       49\n",
       "FANCD2     48\n",
       "FANCL      48\n",
       "FANCM      47\n",
       "FAAP24     47\n",
       "CENPX      47\n",
       "RPA1       46\n",
       "CENPS      46\n",
       "UBE2T      45\n",
       "ERCC1      44\n",
       "FANCE      43\n",
       "ERCC4      43\n",
       "ATRIP      42\n",
       "FANCB      42\n",
       "RPA3       41\n",
       "FANCG      41\n",
       "FANCF      41\n",
       "SLX4       41\n",
       "FAN1       41\n",
       "FANCC      40\n",
       "EME1       40\n",
       "ATR        39\n",
       "USP1       38\n",
       "ZBTB32     38\n",
       "BRIP1      33\n",
       "MUS81      32\n",
       "FAAP100    32\n",
       "RPA2       28\n",
       "TOP3A      28\n",
       "Name: interactor_symbol, dtype: int64"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Across the list of gene pairs, which genes show up the most?\n",
    "\n",
    "df = interact_table['interactor_symbol'].value_counts()\n",
    "df.head(30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "HES1          49\n",
       "RPA1          46\n",
       "ERCC1         44\n",
       "ATRIP         42\n",
       "RPA3          41\n",
       "FAN1          41\n",
       "EME1          40\n",
       "ATR           39\n",
       "ZBTB32        38\n",
       "USP1          38\n",
       "MUS81         32\n",
       "TOP3A         28\n",
       "POLN          27\n",
       "UBA52         27\n",
       "RPS27A        26\n",
       "EME2          26\n",
       "CHEK1         25\n",
       "BLM           25\n",
       "HES4          19\n",
       "HES3          19\n",
       "TOPBP1        18\n",
       "SLX1A         17\n",
       "RFC4          17\n",
       "RAD9A         16\n",
       "CORT          16\n",
       "RPA4          16\n",
       "WDR48         16\n",
       "RAD17         15\n",
       "ZBTB16        15\n",
       "RFC5          14\n",
       "RAD1          14\n",
       "PCNA          14\n",
       "RFC3          14\n",
       "XRCC3         13\n",
       "DCLRE1A       12\n",
       "UBC           12\n",
       "DCLRE1B       11\n",
       "RMI1          11\n",
       "HUS1          11\n",
       "CENPS-CORT    11\n",
       "Name: interactor_symbol, dtype: int64"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Filter out genes from FA set\n",
    "fa_all = 'https://raw.githubusercontent.com/NCATS-Tangerine/cq-notebooks/master/FA_gene_sets/FA_4_all_genes.txt'\n",
    "\n",
    "all_genes = pd.read_csv(fa_all, sep='\\t', names=['gene', 'symbol'])\n",
    "\n",
    "filtered_frame = interact_table[~interact_table['interactor_id'].isin(all_genes['gene'].tolist())]\n",
    "filtered_frame = filtered_frame[~filtered_frame['interactor_symbol'].isin(interact_table['inferred_gene'].tolist())]\n",
    "\n",
    "\n",
    "df = filtered_frame['interactor_symbol'].value_counts()\n",
    "df.head(40)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
